<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>We Are The University</title><link rel="stylesheet" href="/styles.css"></head><body><header><h1 style="color: #fff;font-family: 'Arial Black', Gadget, sans-serif;font-style: italic;font-weight: 900;text-transform: uppercase;">We Are The University    </h1><nav><ul><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li><li><a href="/blog">Blog</a></li><li><a href="/videos">Videos</a></li><li><a href="/authors">Authors</a></li></ul></nav></header><main><h2 style="text-align: center;">Daniel MacArthur Public Lecture 4 July 2017 [1:11:48]</h2><p style="text-align: center;"><a href="https://www.youtube.com/watch?v=xuevQiYsjOc" target="_blank">Watch on Youtube</a></p><p style="text-align: center;"><a href="https://www.youtube.com/channel/UCPVjZXotZ5oX9jyXlFFRj2w" target="_blank">University of Otago - Ōtākou Whakaihu Waka</a></p><img src="https://i.ytimg.com/vi_webp/xuevQiYsjOc/maxresdefault.webp" alt="Thumbnail for video titled: Daniel MacArthur Public Lecture 4 July 2017" style="width: 100%;"><div class="tags"><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#genetics</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#otago</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#harvard</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#research</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#university of otago</span></div><h2>Description</h2><p>Using large-scale genomic databases to understand human genetic variation.<br><br>Daniel MacArthur - Broad Institute of MIT and Harvard Massachusetts General Hosptal, Harvard Medical School.<br><br>@dgmacarthur</p><h2>Transcript</h2><p style="opacity: 0.9; font-size: 0.8em">Transcripts may be automatically generated and may not be 100% accurate.</p><p>well welcome everybody my name's Steven<br>Robertson<br>a clinical<br>geneticists and I like to sustain a warm<br>welcome to all of you for this special<br>public lecture guest lecture as part of<br>the GSA conference it gives me great<br>pleasure to be able to introduce one of<br>Australia's own a world leader in his<br>field to address us tonight and our very<br>rapidly an exciting evolving area of<br>human genetics Daniel McArthur has<br>multiple appointments in Boston at the<br>Broad Institute MIT Harvard and the Mass<br>General Hospital he's made his name in<br>establishing and then analyzing very<br>large and ever-expanding datasets of<br>human sequence data to further<br>understanding about human genetic<br>disease<br>Daniel completed his PhD in Sydney<br>before moving to Cambridge for his first<br>postdoc at the Sanger Center at Engst<br>inand Cambridge sure before moving to<br>the Broad Institute in Boston where he<br>currently resides so it's with great<br>pleasure I introduced Daniel I think<br>we're in for a treat and there'll be an<br>opportunity for a bit of Q&amp;A afterwards<br>Daniel<br>very much Steven can people hear me at<br>the back okay where's that too loud<br>feels a bit too bad perhaps not too loud<br>okay all right thank you very much<br>all right well thank you so much to<br>Steven for that very kind introduction<br>and of course to the organizers for the<br>chance to come out it is always a<br>delight to be back in the southern<br>hemisphere even in winter so as Steven<br>said my my group is pretty focused on<br>different ways of taking very<br>large-scale databases of human genetic<br>variation and using those to improve our<br>interpretation of the types of genetic<br>changes that we find in patients with<br>rare disease and what that means in<br>practice is that my group basically<br>spends about half its time thinking<br>about biology and in particular the<br>biology underlying specific rare<br>diseases like muscular dystrophy and the<br>other 50% of our time we actually spend<br>thinking about things like software<br>engineering and data Sciences and a lot<br>of the work that I'll be presenting<br>today is really a kind of fusion between<br>those two different areas thinking about<br>how we can leverage the just enormous<br>amounts of data that are now being<br>generated in the context of human<br>genetics and turn those into things that<br>are actually useful to us for<br>interpreting human genetic variation but<br>they'd also become useful for the rest<br>of the community as well and so in the<br>in the talk this evening I wanted to<br>start by I guess describing why I think<br>this particular point of time in time in<br>human genetics is an absolutely amazing<br>time to be doing human genetics I would<br>say I would argue the best time in the<br>history of the field I'll then go<br>through a relatively brief description<br>of two of the datasets that we've been<br>working on now I guess for the last four<br>years the x AK and nomad datasets very<br>large collections of human genetic<br>variation and I'll talk a little bit<br>about how we built those and some of the<br>politics behind them and then also how<br>we can use them to understand more about<br>human genes and then I wanted to tell<br>two stories one about that thing we call<br>the human knockout project which is an<br>exercise in trying to identify very rare<br>individuals who are completely lacking<br>the function of particular human genes<br>the human equivalents of knockout mice<br>and using them to understand more about<br>genes that we haven't where we don't<br>fully know the biology yet and then the<br>final story which also to the end is a<br>story actually a pretty personal story<br>about the ways in which a person with a<br>very strong incentive to learn more<br>about a specific disease can leverage<br>these large collections of human very<br>to understand more about how individual<br>genetic variants actually influence the<br>risk of disease so let me start by<br>talking about why human genetics is so<br>amazing at this particular point in time<br>and I like to think of human genetics as<br>important in part because it provides us<br>with a lever for moving behind between<br>two very different kinds of information<br>that we as a community have become<br>extremely good at generating and the<br>first of those kinds of information<br>which is encapsulated in this relatively<br>random screenshot from the UCSC genome<br>browser to suspect many people in the<br>room of familiar with but basically what<br>this browser allows us to do is to look<br>at a section of the human genome some<br>some chunk of the 3 billion bases of the<br>human genome and learn more about what<br>each of those bases Dutton's so that if<br>that base is found within a gene it will<br>tell us that it is present in a gene we<br>can look up which tissues that<br>particular gene is expressed in if it's<br>found outside genes we can learn more<br>about whether that particular region is<br>involved in the regulation of gene<br>expression and and across again across<br>what tissues and as efforts like the<br>encode project and epigenomics roadmap<br>have gotten up and running we can learn<br>more and more about the functional layer<br>this layer on this functional annotation<br>on each of the different points in the<br>human genome a second type of<br>information that we have been developing<br>really I guess over over many decades is<br>information about human phenotypes that<br>is the collection of all of the<br>different pieces of information that<br>make up how a person appears and behaves<br>and feels and the types of diseases that<br>they have so national bio banks now<br>contain many hundreds of thousands in<br>fact millions of humans and for those<br>people we often have information about<br>all sorts of things so we can look up<br>their disease diagnosis in electronic<br>medical records they have information<br>about prescription registers what drugs<br>they're taking whether or not they have<br>about cancer at particular points in<br>time increasingly as we develop<br>functional genomic technologies like<br>metabolomics and transcriptomics we can<br>look at that data as well for all of<br>these individuals and that means we can<br>really start to build up a picture of<br>the biology of those of those humans<br>those individual humans and then the<br>nice thing about human genetics is it<br>allows us with ways of moving between<br>these two worlds and we can do this we<br>can actually move in both directions<br>here<br>sorry in that in the context of standard<br>human genetics what we would typically<br>call forward human genetics what we tend<br>to do is to take people who have a<br>particular disease so this might be a<br>set of families with the specific rare<br>disease like muscular dystrophy or it<br>might be a set of families with a common<br>set of individuals with the common<br>complex disease like type 2 diabetes or<br>rheumatoid arthritis we take those<br>people and we look at their DNA using<br>one technology or another and we can use<br>that to then move towards particular<br>genes that are implicated in that<br>specific disease and that approach this<br>for genetics approach in humans has been<br>extremely successful it has allowed us<br>to identify thousands nearly 4,000 genes<br>now associated with rare diseases we've<br>also identified many thousands of<br>regions of the genome that are<br>associated with common complex diseases<br>through genome-wide Association studies<br>and so that approach really has allowed<br>us to move from human phenotypes right<br>the way through to some understanding of<br>the underlying biology now as as our<br>data sets of human genetics get bigger<br>it becomes feasible to think about<br>moving in the opposite direction that is<br>to start with individual genes to find<br>people who have genetic changes that<br>perturb the expression or the function<br>of those particular genes and then move<br>and then collect those individuals and<br>learn more about what their phenotype is<br>this is called reverse genetics of<br>course this approaches extreme is is<br>well-established in the context of model<br>organisms where we have the luxury of<br>being able to actually directly<br>interfere in an organism's genome and<br>make changes to a particular gene in the<br>context of humans we can't do direct<br>genetic engineering for a combination of<br>logistical and ethical reasons but what<br>we can do is take advantage of naturally<br>occurring human variation to find<br>exactly the people who actually have<br>changes in a particular gene and then<br>again collect them and learn more about<br>their function and the reason this is so<br>powerful is because there is so much<br>variation that exists in the human<br>population so there are seven more than<br>seven billion humans now currently alive<br>every single one of this carries<br>something like somewhere between 60 and<br>80 mutations that are present in us but<br>we're not present in our parents these<br>are de novo mutations so if you if you<br>add up those numbers seven million<br>humans and sixty to eighty new mutations<br>and only 3 billion bases that exist<br>in the human genome it's actually very<br>clear that the vast the vast majority in<br>fact probably all of the single based<br>changes that can exist in the human<br>genome that are compatible with life do<br>exist somewhere out there in the human<br>population so there is somewhere out<br>there there is at least one and often<br>hundreds or thousands of people who<br>carry any specific based range that you<br>could care to be interested in<br>engineering into the human genome so<br>that means that basically the human the<br>human population consists of a whole<br>collection of individuals with very<br>specific genetic perturbations each of<br>what each of us effectively is an<br>experiment of nature and all we have to<br>do is find those individuals who carry<br>that particular change in a specific<br>gene and look at those individuals to<br>learn more about what that change adds<br>so that of course is slightly ridiculous<br>when we talk about 7 billion people most<br>of whom we don't actually have sequenced<br>but it starts to become a little bit<br>more it start to become a little bit<br>more feasible as we start to generate<br>the scale of data that I'll be talking<br>about in acting a couple of slides time<br>so before I get to that I just wanted to<br>emphasize some of the pros and cons of<br>think about humans as model organisms<br>and I know there's many people here who<br>work on various other model organisms<br>many of which are much simpler to deal<br>with in humans and so I thought it was<br>worth actually articulating some of the<br>challenges and the benefits of humans in<br>this space so the challenges include the<br>fact that humans have a very long<br>breeding cycle they have very few<br>disappointingly few offspring commit to<br>mice certainly compared to worms or<br>flies and also they have this terrible<br>habit of random mating or at least<br>random with respect to what geneticists<br>want them to do so and so as a result we<br>can't set up we can't set up the<br>population in the way that we would in<br>the context of a particular model<br>organism although as I'll show later on<br>in the talk there are particular things<br>that we there are particular types of<br>population that we can look at to make<br>it more likely we will find out the<br>model organisms that we're going after a<br>second big problem with humans is that<br>we can't engineer their genomes and that<br>is a problem for in terms of generating<br>particular types of alleles that<br>obviously are never going to exist in<br>nature so things like conditional<br>knockouts its nature is not kind in us<br>enough to ever give us a mutation that<br>switches off a particular gene in<br>particular tissues or a particular<br>points in time so we don't have that<br>flexibility that we have in model<br>systems there are major limits to<br>interventional studies from any ethical<br>reasons there are there were certain<br>types of experiments that just can't be<br>done thankfully in humans and they're<br>incredibly expensive to housing<br>although many of them do actually manage<br>to pay for themselves so that makes that<br>makes life a little bit easier so those<br>are some of the challenges but there are<br>also considerable benefits to using<br>humans in this setting the first is that<br>they're biologically is unambiguously<br>relevant to humans we don't know when<br>we're making an argument in our grants<br>about the fact that the biology we study<br>is relevant to the human situation<br>that's much easier than if we're<br>studying mice I should say I did my PhD<br>studying mice so that makes our life a<br>lot easier there are enormous lis large<br>numbers of humans available that we can<br>actually study there are very well<br>established systems for phenotyping them<br>they're called medicine and we also have<br>these large well-established phenotyping<br>centers called hospitals that collect a<br>long term observational data from these<br>organisms over long period of time and<br>although it's not it's not it's often<br>not collected in the perfect way for<br>research studies it does mean that we<br>can actually get if we have enough<br>sequence individuals we do have lots and<br>lots of data that we could theoretically<br>use to connect to human genetics to<br>learn more about the functions of genes<br>and we are now starting to live in a<br>world where the amount of sequence<br>humans is growing absolutely<br>exponentially and I mean that literally<br>exponentially so it is the case that now<br>around the world over 2 million humans<br>have had sequencing done on a on a<br>genome-wide scale and by genome-wide I<br>typically mean two different to one of<br>two different technologies has been<br>applied either whole genome sequencing<br>which looks at all of the bases roughly<br>all of the 3 billion bases in the human<br>genome or more commonly exome sequencing<br>and you'll hear that term come up a<br>little bit later on for those of you who<br>don't know what an exome is an exome is<br>a very cost effective way of looking at<br>the roughly 2 percent of the human<br>genome that codes for proteins so the<br>bit of the genome that we understand the<br>best and it's also the bit of the genome<br>where most of the changes that underlie<br>rare disease are found so so exome<br>sequencing is a really good way of<br>looking at that so we have over 2<br>million people who've now been sequence<br>and they've been sequenced in a whole<br>range of different settings from<br>large-scale academic sequencing centers<br>right the way through to to big medical<br>centers as well as a bunch of projects<br>currently underway that are being<br>financed by pharmaceutical companies as<br>well just to give you a sense about how<br>rapidly that the scale of sequencing has<br>gone up these are numbers from just one<br>place in the world this is the place<br>that I work the Broad Institute of<br>Harvard and MIT the bone has the largest<br>genome sequencing facility in the US<br>over this what this plot shows for those<br>of you can't read the numbers is from<br>2009 up to 2000 and the end of 2016<br>how many genomes we've sequenced on the<br>left-hand side and how many exomes we've<br>sequenced on the right-hand side and the<br>numbers of those we can't read are we<br>now we currently have in our in our<br>hands just over 70,000 this numbers a<br>little bit out of date just over 70,000<br>whole human genomes that have been<br>sequenced and somewhere north of a<br>quarter of a million exomes that have<br>been sequenced just in one place in the<br>world and so that's those numbers are<br>impressive but all the growth of these<br>numbers is perhaps the most important<br>thing and that's particularly true for<br>whole genome sequencing as that<br>technology has begun to explode we are<br>now generally generating more data in<br>any given year typically than we have<br>generated in all previous years combined<br>so this data is so a number the amount<br>of data is increasing at a truly<br>ferocious rate and this is really a lot<br>of data and I don't just mean that by<br>biology standards I mean that by<br>anyone's standards so one genome the<br>amount of data that goes into one genome<br>is about 120 gigabytes of information<br>the average net weekly Netflix user at<br>least in the u.s. I have no idea about<br>in Australia or New Zealand the average<br>Netflix user stream is about forty five<br>gigabytes of data per month so basically<br>it would take you about 3 months to<br>stream enough data through Netflix under<br>average consumption to get the<br>approximately the same amount of<br>information that you have existing in<br>your genome and all the cells in your<br>body the broad currently sequences about<br>one human genome every 10 minutes so it<br>obviously takes a lot longer than 10<br>minutes to sequence a human genome but<br>that's how often they roll off the<br>machines and that means that we build up<br>the equivalent of the entire Netflix<br>library every two months there's a lot<br>of crap in the entire Netflix library<br>but most of this information is actually<br>is actually pretty pretty valuable and<br>in total in the road data center we have<br>about 10 petabytes of data and that is a<br>truly staggering amount of data so to<br>put that in perspective if you were to<br>take all 10 billion photos that exist in<br>the facebook on the Facebook servers<br>that only takes up 1.5 petabytes so<br>that's about 1/7 of the total amount of<br>data that we have on our servers so here<br>we are talking about so biology at least<br>human genomics is very rapid rapidly<br>becoming a data science and involves a<br>scale that is truly impressive not just<br>to us but to other people as well and I<br>should say actually one of the things we<br>like to brag about of the road is that<br>twice over the last year we have managed<br>to break the Google glass<br>which is actually was actually pretty<br>hard Google is very good at engineering<br>clouds that are robust but we've been<br>generating and computing data at a scale<br>that has actually made it very difficult<br>for them to keep up so in addition to<br>the war challenges of actually des<br>storing and processing that amount of<br>data there are some real issues with<br>this scale of genomic data generation<br>one of those is it's actually quite hard<br>to just move this amount of data from<br>one place to another 10 petabytes of<br>data will exceed the bandwidth of<br>basically any network connection between<br>two places and in fact for sufficiently<br>large amounts of data it is often the<br>case that the most efficient way of<br>moving data from one place to another is<br>to download it all onto individual hard<br>drives load those up into a truck drive<br>the truck to another place and then<br>download it at the other end which is<br>completely ridiculous in the internet<br>era<br>this is becoming somewhat easier in the<br>era of the cloud where we can just store<br>data in one centralized place but simple<br>storage and movement logistics is so<br>hard<br>a real non-technical challenge to to<br>analyzing the scale of data is the fact<br>that consent has been collected over<br>many decades and what that means is that<br>people who participate in these<br>large-scale genomic studies were often<br>consented at a time when genomics didn't<br>exist and what we didn't really think<br>carefully about the prospect of global<br>data sharing of data so that makes how<br>large much more complicated there are<br>objections to data sharing by the people<br>who generator so many industry-funded<br>collaborations are not interested in<br>making the data available to the world<br>because they see it as something that<br>will benefit their corporation but by<br>keeping it secret some of you may be<br>shocked to hear that there are even some<br>academics that don't believe in sharing<br>data and so all of this makes our life a<br>little bit harder as well but then one<br>of the big challenges that we've had to<br>overcome in our work is the fact that<br>every single person who sequences<br>genomes tend tends to use their own<br>slightly idiosyncratic pipeline for<br>processing and analyzing and looking at<br>genetic changes in in that data and so<br>that means if we which and I Eve ly take<br>the output of each of those different<br>pipelines and just try to add munjal<br>that together into one collection what<br>we would end up with is a data set where<br>the interesting differences were<br>actually dominated by technical errors<br>rather than by real biology and so what<br>we need to be able to do is to take as<br>much data as we can get our hands on<br>process that through exactly the same<br>pipeline and call genetic variants in<br>exactly the same way across large<br>amounts of data and then make that data<br>available to the world and that's that's<br>what we've been trying to do over the<br>last for you<br>so when we when I started my lab back in<br>2012 one of the first projects we worked<br>on was an analysis of exome sequencing<br>data from patients with various rare<br>forms of muscular dystrophy and one of<br>the things we realized very quickly is<br>the the existing databases of human<br>variation we're actually inadequate for<br>analyzing the data we were finding in<br>our patients what we wanted to do of<br>course in the context of a patient is to<br>take every genetic change that we<br>observed in that person's genome or<br>exome and figure out if that if that had<br>ever been seen before if so how common<br>it was in the human population and if<br>any of the other people who carry that<br>particular variant also have the same<br>disease and so the two datasets that we<br>had available for asking and answering<br>those kinds of questions at the time<br>were the thousand genomes project which<br>consisted despite its name consisted of<br>about two and a half thousand genomes<br>from people of diverse different diverse<br>ancestry we also have the NHLBI exome<br>sequencing project or ESP which<br>contained about six and a half thousand<br>individuals over the European or African<br>American ancestry and then through a<br>process that took about 18 months and<br>I'm not gonna describe in any detail<br>role at all but was was fairly painful<br>both politically and technically we were<br>able to release in October 2014 a<br>brand-new collection of sequencing data<br>consisting of exome exomes across nearly<br>61 thousand people so this dataset which<br>is called the exome aggregation<br>consortium or X Act data set was by far<br>the largest collection of human genetic<br>variation that existed at the time<br>actually by almost an order of magnitude<br>and that has been used very heavily over<br>the last couple of years and then in<br>October last year we were able to<br>release a new version of this data set<br>this is now called the genome<br>aggregation database or nomad we like as<br>as Eleanor pointed out last night we<br>like our our<br>summaries of our projects to contain<br>idiosyncratic capitalization and silent<br>letters just to be confusing so Nommo<br>contains about twice the number of<br>exomes that are present in x AK and also<br>for the first time incorporates a large<br>number of whole human genome so that<br>means we can look at both the protein<br>coding bits of the genome as well as the<br>non-coding bits of the genome and<br>importantly all of this data was these<br>data were aggregated not sequenced by us<br>but were collected by aggregating them<br>together from many other sequencing<br>studies I'll talk about those studies in<br>a moment<br>they were all processed through the same<br>pipeline pipeline and then we called<br>genetic variants jointly across all of<br>those samples simultaneously and this<br>has required a huge amount of effort to<br>get it done but the end result has<br>really been worth it so the reason we<br>built the exact database in the first<br>place is because we wanted to improve<br>our ability to filter the genetic<br>changes that we found in rare disease<br>patients and x-act is much more powerful<br>than previous databases for doing that<br>so just to illustrate that here we have<br>on the left-hand side but not the number<br>of variants that remain in an average<br>person's exome from five different<br>continental groups the different color<br>represents different ancestry groups for<br>for analysis that was done using the ESP<br>project which contained about six and a<br>half thousand exomes and so if you apply<br>a frequency filter of about 0.1% so<br>that's something you might imagine<br>applying in the context of a dominant<br>and a rare dominant disease you end up<br>with up to a thousand almost a thousand<br>candidate variants that remain in a<br>person's exome if you do exactly the<br>same analysis but now you're using X AK<br>which is ten times as large and much<br>more diverse you get you get an order of<br>magnitude reduction in the number of<br>candidate variants that show up in in<br>that person's genome so X AK is a much<br>more powerful way of filtering the<br>genetic changes that are found in rare<br>disease patients which is great because<br>that's one of the key things we built it<br>for the Cullens okay all right so it's<br>worth it's worth being a tiny bit of<br>time before I move on to the science we<br>can do with this project just<br>emphasizing who exactly has ended up in<br>these databases so again we didn't<br>sequence these people we aggregated them<br>where we are<br>data parasites we're opportunistic<br>collectors of data that was sequence for<br>other purposes and what we do is just<br>take that data and reprocess it and put<br>it together in the same project and make<br>it available to people so the people who<br>have been sequencing this project mostly<br>consist of people from case control<br>studies of complex adult onset diseases<br>such as type 2 diabetes neuropsychiatric<br>conditions such as schizophrenia and<br>bipolar and cardiovascular disease<br>particularly early onset heart attack so<br>we have so we that's that's where most<br>people come from we have as best we can<br>depleted these collections of people who<br>either have very severe early onset<br>diseases like muscular dystrophy or<br>autism or pediatric cancer where some of<br>them may well remain and we've also<br>removed their first-degree relatives and<br>worth noting though that some severely<br>ill people do actually remain in this<br>data set so we can't get rid of all of<br>them because we don't have perfect<br>health information and also some of the<br>people may not have disease right now<br>but they will go on to get disease later<br>and that's important for the project<br>that I'll talk about at the end of this<br>at the end of this talk and then finally<br>although these samples often weren't<br>collected for the purpose of doing<br>large-scale genetic studies across many<br>different phenotypes about two fifths or<br>about two fifths of them we do actually<br>have the ability to go back and get more<br>clinical or phenotype data this in some<br>cases consists of really deep clinical<br>data so for some of the people who were<br>in for instance the bio banks from<br>Finland in this collection we can<br>actually go back and look at every<br>diagnosis since ever been made for these<br>people over the course of the 25 to 50<br>so this slightly hypnotic rotating gif<br>on the left-hand side here gives you a<br>snapshot there's actually one of the<br>first analyses we did on the the new<br>nomad dataset and what this shows you is<br>the ancestry of the individuals who<br>actually came into this this collection<br>this is using a analysis method called<br>principal component analysis or PCA it's<br>very simple basically what it does is it<br>takes all the different all the<br>information that's present in the<br>genetics of these genetic data of these<br>individuals and condenses it down so<br>that we can view it along three three<br>dimensions and that those three<br>dimensions what we're basically rotating<br>around here and so the nice thing about<br>that is it allows us to cluster these<br>individuals by their by how closely<br>genetically related to it they are to<br>one another which corresponds to their<br>genetic ancestry so you can see here<br>very clear although somewhat diffuse<br>clusters corresponding to European<br>ancestry here in orange and there's<br>these little jerk-offs here which<br>correspond to individuals of Finnish<br>ancestry and Ashkenazi Jewish ancestry<br>it's a big cluster of Latino samples<br>down here we have South Asian samples up<br>here African ancestry over here this<br>little cluster of East Asian individuals<br>over here so this is absolutely by no<br>means representative of the entire<br>spectrum of human genetic variation we<br>are missing for instance the vast<br>majority of variation that exists in the<br>African population we only have a couple<br>of populations in here we're also<br>missing the whole of the Middle East and<br>importantly for this audience pretty<br>much all of Oceania so Australian New<br>Zealand populations Polynesian<br>populations are also completely missing<br>here but what it does mean is that for<br>many of these continental populations we<br>have often thousands you can see the<br>numbers over here<br>thousands of individuals from these<br>continental populations that we can use<br>to look up any any variant that we find<br>in a patient of that ancestry who's<br>turned up in a rare disease clinic we<br>can look up to see how common that<br>variant is in in that population so<br>still a lot of work to do but we're<br>getting there in terms of collecting<br>information across a large chunk of the<br>human of the human population<br>now our purpose in doing the in<br>generating this project was not just to<br>benefit our own rare disease research<br>but to make sure that anyone around the<br>world could take advantage of the same<br>datasets and so pretty much as soon as<br>we had the data set to the point that we<br>were comfortable that it was actually<br>working we had gone through quality<br>control and it seems to not be a total<br>disaster<br>we made that data available through a<br>series of websites there's x-acto Pratt<br>Institute org and more recently Nomad<br>Broad Institute org if you go to those<br>browsers which were built by people in<br>my group what you can find is<br>information for your favorite gene is<br>one of mine the blue Peaks here<br>represents the coverage that is the<br>number of reads that we have on average<br>in across our 60,000 or 120,000<br>individuals for exomes shown here in<br>blue and for genomes shown here in green<br>and then below that we have a list of<br>every variant that we've observed across<br>our entire collection of individuals and<br>you can click on those to learn more<br>information about how common that<br>variant is across different populations<br>and how likely we are how likely we<br>think it is that it's actually a real<br>variant there's a lot more information<br>there and anyone who's interested in<br>human genetics of course can look that<br>up and there's no absolutely no<br>restrictions whatsoever on non analysis<br>or publication from the data set that's<br>available here and you can download all<br>of the information here in VCF form as<br>well so there's lots of things you can<br>do with this data set I wanted to I<br>wanted to just briefly mention one of<br>them and that is work that's been led by<br>Caitlin smoker a graduate student in<br>martelli's lab has been working with my<br>lab for this project what Caitlin was<br>interested in is not just the variants<br>that were actually present in the data<br>set but the variants that were missing<br>from the data set and to and to study<br>those what she did is to build a<br>statistical model that allows her to<br>predict for any given gene in the genome<br>how many variants of a particular<br>functional class you would expect to see<br>by chance so that means if the only<br>process that was acting on that gene is<br>random mutation and natural selection<br>had been removed how many such variants<br>we would expect to see pop up you know<br>in a collection of 60,000 people<br>and you can see here on this plot here<br>we're looking at synonymous variants<br>which are variants that silent that is<br>they don't disrupt the protein coding<br>region of the genome and on the x-axis<br>we have the number of variants per gene<br>predicted by Kaitlyn's model on the<br>y-axis we have the number that were<br>actually observed in 60,000 people and<br>you can see the fit is exquisitely good<br>so when in classes of variation where<br>there is no selection Kaitlyn's model<br>predicts almost perfectly<br>how many variants we should expect to<br>see in in each of the genes in the human<br>genome but as we move towards classes of<br>variation that do affect gene function<br>and where we expect the natural<br>selection will be acting that foo<br>becomes much worse and the reason for<br>that is very simple so here we're<br>looking at loss of function variants or<br>protein truncating variants variants<br>that we expect to completely break the<br>genes that they're found in and you can<br>see here that the vast majority of genes<br>fall below the line and the reason for<br>that is is that for most genes in the<br>genome<br>lots of function variants are bad they<br>disrupt a gene that's actually doing<br>something useful in the cell and as a<br>result most of those variants tend to<br>get removed from the population by<br>natural selection so there are fewer of<br>them than we would expect to see by<br>chance so that's no surprise at all<br>we're not going to win any Nobel prizes<br>by proving that natural selection<br>actually exists but what we can do with<br>60,000 people in this in this picture is<br>for the first time calculate how far<br>below the line each each gene Falls and<br>that gives us a quantitative estimate of<br>how strong strongly natural selection is<br>actually acting on the particular class<br>of variation so just give you one<br>example here's a gene called dink one h1<br>we know exactly what happens when you<br>break this gene break a single copy of<br>this gene to know very missense and<br>loss-of-function variants in this gene<br>associated with a whole host of horrible<br>early onset neurodevelopmental<br>phenotypes like intellectual disability<br>seizures seizures and and other<br>phenotypes as well and what we see in<br>this gene is that Kaitlyn's model<br>predicts almost perfectly how many<br>synonymous variants we would expect to<br>see but we're missing about two-thirds<br>of the expected missense variation and<br>almost all of the expected loss of<br>function variation so we actually we<br>expect to see 160 loss of function<br>variants in this gene and we are<br>actually only observed for and with of<br>the four that we observed two of them<br>are artifacts one is a sequencing error<br>one of them is an annotation error and<br>there are two people carrying these<br>variants in the Exide population who<br>probably actually do have one of these<br>diseases so nonetheless we could tell<br>even without knowing that this was<br>associated with disease we could tell<br>just from this<br>very functional variation in this<br>particular gene that something<br>catastrophic happens when you break this<br>gene even a single copy of this gene so<br>that's a case where we know what the<br>disease is here's the case where we<br>don't this is a gene called u br5 where<br>we know that it falls in the ubiquitin<br>pathway we know if you knock it out in<br>mice that kills them during<br>embryogenesis very little else is known<br>but we had we sit here we see a picture<br>that's very similar to what we see in<br>Dean connect one so that is you know<br>well calibrated model for silent<br>variants a substantial reduction in<br>missense variation that is variants that<br>change amino acid sequence and a near<br>complete absence of loss of function<br>variation so this tells us although we<br>don't know what the disease phenotype is<br>that there likely is some relatively<br>severe phenotype associated with the<br>loss of one copy of the UBR five gene<br>and we can apply this of course to any<br>of the genes in the human genome what we<br>when we do that what we find is that<br>about there are just over 3,000 genes<br>with a near complete absence of loss of<br>function variants or protein truncating<br>variants in the in the population what<br>that tells us is that those genes are<br>almost certainly again<br>disease-associated genes and indeed that<br>collection that set of genes contains<br>almost all of the null and haploid and<br>sufficient disease genes where haplin's<br>efficient here means genes where if you<br>knock out one copy of the gene it<br>results in a severe disease so that as<br>we expect contains almost all of those<br>but importantly almost three-quarters of<br>these genes we really have no idea what<br>they are<br>there's no phenotype associated with<br>mutations in those genes for in fact for<br>most of these genes we we often have<br>absolutely no biochemical information as<br>well that's really interesting because<br>we're pretty it's the genetics tells us<br>that these are interesting we don't know<br>exactly what they do and these are now<br>being heavily prioritized in studies of<br>follow-up studies of a whole range of<br>different diseases including work that<br>marked daily is now leading in autism<br>and schizophrenia where he has shown<br>that in fact this collection of 3,000<br>genes contains almost all of the signal<br>for de novo protein truncating variants<br>in in autism cases so there's lots of<br>work ahead first in nomad and i just<br>wanted to finish this out before i go<br>into two stories that explains how we<br>can use nomad to learn more about our<br>gene biology so we have a we're working<br>on two papers at the moment resulting<br>from the coming out of the nomad work<br>one of them is on protein truncating<br>variants and their effect on genes<br>across the human genome a second of them<br>is looking at the depletion of variation<br>in particular chunks of the non-coding<br>region so so applying the type of<br>constraint model that we've done<br>across genes and now applying that to<br>regulatory regions we are calling<br>structural variation our our analysis<br>pipeline to date has really focused on<br>very small types of variation so single<br>nucleotide variants and in domes we're<br>now expanding it up to include things<br>like deletions insertions and inversions<br>and we're moving all of our analysis by<br>plane over to the Google clouds that we<br>can stop being held back by not being<br>able to scale based on the physical<br>hardware that we have currently existing<br>at the road and then finally our mission<br>this year by the end of this year is to<br>apply and complete a variant call set<br>across just over 50 thousand human<br>genomes and so we hope to be able to<br>release publicly by the end of this year<br>a call set that will actually likely be<br>over 60,000 human genomes so where we<br>still have a lot of what to do to get<br>there but we're pretty confident we will<br>hit that number okay so I wanted to<br>spend the rest of my time<br>let me just how I'm going for time the<br>rest of my time today talking through<br>two ways in which we can use the<br>variants that we collect from these<br>large repositories of human genetic<br>variation to understand the ways in<br>which genetic variation can impact gene<br>function and so the first of those will<br>be the human knockout project so I'll<br>talk about actually using leveraging<br>naturally-occurring genetic variation to<br>identify it rare individuals who are<br>completely lacking the function of a<br>particular gene and then using that to<br>learn more about the function of that<br>gene and then secondly I wanted to talk<br>through the use of the example of prion<br>disease to talk through how we can<br>actually leverage these datasets to to<br>understand the penetrance that is the<br>risk that someone who carries a<br>particular genetic variant will actually<br>go on to get the disease to get a<br>particularly severe rare disease okay so<br>let's start with a human knockout<br>project so so again going back to this<br>slide revealing talking about reverse<br>genetics in humans so again that is<br>starting with people who we know have<br>disrupted mutations in a particular gene<br>and then and then going through using<br>this this matrix of human genetic<br>variation to identify those people and<br>then learn more about their phenotype<br>and again just a reminder that about 40%<br>of the people who no matter actually<br>recontact able they have a whole variety<br>of information about their phenotype<br>that may be available so we can we can<br>use this in theory to learn more about<br>where each of the 20,000 or so genes in<br>the human genome sits along this<br>spectrum of tolerance to loss of<br>function variation<br>so this spectrum is pretty theoretical<br>we don't actually know where these<br>boundaries are or where these particular<br>numbers are but we would expect if we<br>would arrange all of the genes in the<br>human genome in some in some reasonable<br>way what we would see is that there are<br>some genes over on the left hand side<br>here where lots of function variation is<br>extremely disruptive to human fitness so<br>there's sorry this might be cases for<br>instance where knocking out a single<br>copy of the gene results in very severe<br>disease and on the right hand side we<br>have genes which are what we were called<br>non-essential so these may be cases<br>where completely an activation of a gene<br>that is homozygous inactivation of a<br>gene could actually result in a human<br>being who is perfectly healthy or at<br>least doesn't doesn't have any disease<br>that is severe enough that they would<br>end up in a clinical genetics clinic and<br>we can use lots of different ways to get<br>it wherever gene fits along the spectrum<br>so we've become extremely good at<br>looking at variants in the middle here<br>so these are variants where either<br>heterozygous or homozygous inactivation<br>results in a severe disease if they're<br>dominant or a recessive severe disease<br>and as I mentioned at the beginning of<br>the talk we found now about 4,000 genes<br>associated with those very rare diseases<br>to mostly fall in this middle part of<br>the spectrum here now we don't know a<br>huge amount about those stuff on the<br>left-hand side here these are often<br>variant genes we're messing with them<br>results in embryonic lethality and so as<br>a result weak so we know that roughly<br>30% of genes in mice if they're knocked<br>out result in a loss of the embryos of<br>those mice we don't know many of those<br>genes in humans and most most of the<br>corresponding genes in humans are<br>actually not associated with disease<br>suggesting that again messing with them<br>in the human situation results in a loss<br>during embryogenesis and we just never<br>see humans who carry those mutations but<br>we can learn more about those genes by<br>looking at that constraint so the study<br>that I the analysis that I showed you<br>earlier<br>where we look at genes that are actually<br>missing variation in the population we<br>think points us pretty heavily towards<br>genes that fall on this end of the<br>spectrum but what we also actually don't<br>have a lot of information about genes on<br>the other end of the spectrum that is<br>the spectrum we're messing with those<br>dreams results in some kind of phenotype<br>but often not a phenotype the results in<br>people going to a clinical geneticist so<br>to study those people you can imagine<br>looking at lots of function variation in<br>very large numbers of individuals from<br>the general population is the right<br>approach this is the these are the types<br>of genes where human knockouts may<br>actually be found and they may be really<br>informative about what those genes<br>actually do so there are a couple of<br>tricks we can use to enrich for<br>populations that are actually likely to<br>have true not<br>that is homozygous inactivation of both<br>copies of a particular gene and these<br>have very different effects on the types<br>of the types of genes that were and the<br>types in the prevalence of genes that<br>were able to see knocked out in these in<br>these populations so one of the things<br>we can do is to look at bottlenecks<br>populations these are populations that<br>have gone through a relatively recent<br>small founder population so this might<br>be the case for instance in Finland and<br>where those that found a population has<br>then expanded up to grow to a very large<br>number of people so five million people<br>in the case of Finland<br>so these bottlenecks populations because<br>they went through a very small number of<br>founders and then expanded up often<br>actually contain carry relatively<br>deleterious mutations at relatively high<br>frequency within those within within the<br>population and that's something we can<br>leverage to learn more about variants<br>that actually do cause disease like<br>recessive disease variants for instance<br>but we would also we would also be<br>expect that we will see some true human<br>knockout genes that come through that<br>particular bottleneck as well and<br>because of the nature of this bottleneck<br>what we will find is that relatively few<br>genes are knocked out in these<br>populations but the genes that are<br>knocked out will be knocked out often in<br>many people so it gives us more<br>statistical power to look at the exact<br>impact of human knockouts on particular<br>phenotypes as a second approach that we<br>can imagine doing is to look at concern<br>when its populations I can tell you what<br>it's just means populations where there<br>is a high rate of parental relatedness<br>that is people who may together who are<br>related to each other<br>and this exists typically due to<br>cultural practices that result in high<br>levels of cousin cousin marriage or<br>uncle niece marriage often so when these<br>concern when its populations again we<br>will see a large number of individuals<br>who carry homozygous loss of function<br>variants but here these will be<br>scattered across many of the genes in<br>the human genome<br>and the reason for that is that concern<br>witness marriages basically can take any<br>rare variant that exists within the<br>family and happens to be carried by both<br>of those parents and then what we call<br>hunter and then what we would call<br>homozygotes that that is take that and<br>make make it make it that the offspring<br>of those individuals carries two copies<br>of that Burk and Jane and so the end<br>result will be a large number of genes<br>that we find knocked out but typically a<br>relatively small number of individuals<br>who are knocked out for any one gene so<br>we've been taking both of these<br>approaches to look for human knockouts<br>across a number now of different<br>populations and<br>in two cases we've been able to study<br>you can say - individuals these are both<br>published now so we've been working with<br>the East London genes and health cohort<br>led by David Byrne Hill and Richard<br>Trenberth this is a population of mostly<br>Pakistani descent living in the UK we've<br>now sequence well over 3,000 individuals<br>from that population and then secondly<br>working with Dinesh salihin and say hi<br>Katherine we've also studied a set of<br>pakistani individuals living in pakistan<br>we've now sequenced about 10,000<br>individuals from that population about a<br>third of whom american sanguinis so in<br>the in these two studies we really get a<br>good handle on the distribution of<br>homozygous knockout mutations across<br>these concern - populations which I'll<br>talk about on the next slide another<br>very large scale analysis that we've<br>been pursuing and this has been funded<br>by a full way pharmaceutical company<br>collaboration between Merck SI Biogen<br>and Pfizer<br>is to is to pull together as much<br>information as we can care about the<br>genetic the genetics of the Finnish<br>population this is work has been led by<br>an employer in Finland and Martelli at<br>brode what we now have in Finland is a<br>collection of incredibly well phenotyped<br>individuals these are there's about half<br>a million people in a whole variety of<br>different finished bio banks about<br>400,000 DNA samples that have already<br>been collected 20,000 of those we've we<br>have now sequence diva exomes or genomes<br>and a much larger number now for now we<br>have genotyping arrays that have been<br>run and so that gives us an enormous<br>amount of information about the<br>distribution of even relatively rare<br>variants across the Finnish population<br>that we can collect with this glorious<br>electronic medical record and trade data<br>from those bio banks and so we've<br>learned a lot from this process already<br>the first big lesson from the two<br>studies that I that I mentioned in the<br>in the Pakistani populations is that<br>looking at concern when it's individuals<br>gives us an enormous yield of rare<br>homozygous knockout<br>so nearly half of the individuals of<br>concern when its origin in these<br>populations carry at least one rare<br>knockout gene that's way higher than<br>what we see in a European population for<br>instance where that number is closer to<br>1% so that's a huge enrichment that we<br>can really use to study the effect of<br>particular knockout genes on on the<br>human organism a second big lesson is<br>that this Finnish biobank data is<br>extremely powerful if we want to be able<br>to go from genotype through to phenotype<br>and so really having data that's been<br>collected over many decades on these<br>individuals and<br>often multiple questionnaires collected<br>across many years as well as electronic<br>medical record data combined is<br>extraordinarily useful to build up a<br>much richer picture of exactly what is<br>wrong with a particular individual and I<br>can across many samples and so far we've<br>been able to discover over<br>two-and-a-half thousand genes with<br>homozygous loss of function and at least<br>one sample across these three<br>populations so we're starting to get<br>well over ten percent of the human<br>genome that contains at least one<br>knockout and there's already been some<br>early insights into human biology so<br>we'll just give you one example of that<br>this is a gene called a possi 3 it was<br>well established we actually known for<br>about for over five years that<br>heterozygous that is single copy of loss<br>of function of their approach a three<br>gene results in a reduction in<br>triglyceride levels that is fat levels<br>in the blood and also reduces people's<br>risk of heart attacks early onset heart<br>attacks but what we weren't able to see<br>from these even from relatively<br>large-scale population studies is what<br>happens if you knock out both copies of<br>the Episode III gene and this is really<br>important because we already have four<br>different pharmaceutical companies that<br>are developing inhibited inhibitors to<br>this gene and it's really unclear what<br>would actually happen<br>we know the 50% reduction of the protein<br>product is probably okay but it's really<br>unclear what would actually happen if<br>you developed extremely efficient<br>inhibitors that completely knocked down<br>so function of our PC 3 in a living<br>human so we said about trying to trying<br>to explore that in the Pakistani<br>population so what we found and this is<br>work that it's been led by Dinesh pretty<br>and cite is is in our initial analysis<br>of 10,000 sickness individuals from the<br>premise from the promess study that is<br>the Pakistani group in Pakistan for whom<br>we have access to blood plasma data we<br>were able to show that there were these<br>people are in fact complete knockouts<br>for apple OC 3<br>those copies are inactivated and the<br>appo c3 protein levels in their serum<br>down to 0 so there's a complete absence<br>of any protein expressed from this<br>particular gene so that's useful in that<br>it tells us that our OC 3 knockouts can<br>actually survive to adulthood but we can<br>learn a lot more about these individuals<br>by studying these these permanently<br>large pedigrees that exist in this<br>particular population so here's one<br>pedigree this individual in purple here<br>is an individual who was a knockout for<br>the for the episode 3 gene so zooming in<br>on that person and that person's<br>pedigree here you can see he's married<br>exactly once he's had nine children and<br>so we then said about genotyping each of<br>the 11 individuals within this family to<br>see what<br>any of the other children this family by<br>chance may actually also have ended up<br>as being knockouts for this gene and<br>what we found to our surprise and<br>delight was that the wife is also at<br>home as I guess a knockout for the<br>episode through June so this is one of<br>those cases where despite the random<br>mating of humans we've got it up with<br>exactly the mating that we would want in<br>this particular family and as a result<br>of course all of the children are<br>obligate knockouts for the upper teeth<br>region as well so we were able to get<br>our samples in from seven of the kids<br>and they gave us nine children nine<br>individuals in one family that we could<br>use to study the effects of the loss of<br>the complete loss of this protein on on<br>various different aspects of biology and<br>one important outcome from this is that<br>these two individuals up here both of<br>whom are now in their 60s appeared to be<br>relatively healthy so there was no major<br>clinical syndrome that appears to result<br>from the total loss of our program again<br>that's extremely reassuring to the<br>pharma companies that are running these<br>analyses but we can also do intervention<br>studies in these families so what we did<br>is a very simple dietary fat challenge<br>this basically meant taking individuals<br>who are knockouts<br>knockouts or wild-type that is what that<br>is normal for the oppo so through gene<br>and exposing voice to a dietary fat<br>challenge which in non-technical terms<br>meant giving them a huge fatty milkshake<br>they could drink that and then we can we<br>measure the the fat levels in their<br>blood over time and what we see when we<br>do that is this so individuals who have<br>functional copies of the abc3 gene their<br>fat levels spike in over a ton course of<br>hours after drinking a milkshake<br>containing lots of fat people who have a<br>Percy 3 knocked out their fat levels<br>change barely at all so that tells us<br>that if you're missing this protein<br>there basically is a fundamental defect<br>in the absorption of that and that<br>results in this incredibly low basal<br>triglyceride levels in the serum of<br>these individuals and that would be<br>expected if anything to result in even<br>stronger protection against myocardial<br>infraction so on the strength of this<br>multiple pharma companies are now<br>proceeding forward but with programs<br>targeting very substantial reductions in<br>FOC 3 levels in in adult humans so<br>there's a lot more that we can learn<br>from these knockout individuals this<br>plot these plots which are extrapolated<br>forward from the relatively small<br>numbers of samples we sequence today<br>give us an idea about how many more<br>human knockouts we could get if we were<br>to sequence more larger and larger<br>numbers of samples and basically that<br>that<br>number grows much more rapidly if we<br>sequence individual populations with<br>high levels of consanguinity like<br>Pakistan and it obviously grows much<br>more modestly for the sequence African<br>or European populations and these<br>numbers on the x-axis are not completely<br>ridiculous here we're talking about<br>50,000 sample sequence down here<br>actually getting pretty close to that in<br>the Pakistani population between various<br>different collaborators we're now open<br>about 40,000 individuals of Pakistani<br>origin and sequenced and we would expect<br>that to approach about a hundred<br>thousand individuals as we as over the<br>next couple of years so with a hundred<br>thousand samples sorry<br>with a hundred thousand samples we will<br>be approaching the point where you have<br>about a third of the genes in the human<br>genome we've seen at least one<br>homozygous knockout individual that we<br>can that we can study there's only so<br>much you can learn from one individual<br>but I think combining that with data<br>from model organisms and other and other<br>analyses will give us a lot of<br>information about the biology of those<br>genes so there's still plenty of<br>challenges ahead though one challenge<br>remains that although we can predict<br>loss of function mutations relatively<br>well that is variants that are predicted<br>to break for the function of a protein<br>coding gene for some of those we find<br>out that there are various ways that the<br>cell can actually escape loss of<br>function and so I think we will need to<br>move towards having large-scale RNA seek<br>and high spirit experimental assays to<br>resolve exactly what the functional<br>impact of each of these variants are<br>secondly we still have a lot of work to<br>do to handle the amount of data that's<br>required for to sequence these number of<br>people and then finally it's important<br>to remember that human genetic variation<br>although it's something that I love and<br>that we spend a lot of time thinking<br>about is really only just one line of<br>evidence for understanding biology and<br>that's no surprise to this crowd is this<br>human knockout is equivalent to lifetime<br>whole-body exposure to the loss of a<br>particular gene that's not always the<br>thing that we want to know when we're<br>studying the function of a particular<br>gene and unless we integrate that with<br>all the other sources of information<br>that are available such a course as<br>model organism information we'll be<br>missing a big chunk of the picture so<br>human genetics is useful but it's far<br>from the only thing that we need to be<br>studying and again that's not not a<br>lesson that won't resonate here okay so<br>with that I'll move on to the the final<br>story that I wanted to tell tonight<br>which is a story about how we can use<br>these large-scale databases of human<br>variation to learn more about the how<br>likely it is that a given variant a<br>variant in ER disease gene is that will<br>actually cause disease in that<br>individual so this is work on<br>horrible set of diseases called prion<br>diseases and it was led by Eric Phillip<br>Medical who at the time was an analyst<br>in my group for about two years and Eric<br>had a very personal reason to be<br>interested in studying prion diseases so<br>this is Eric's wife Sonya shown here on<br>the left<br>she's shown here at the beginning of<br>2010 with her mother at that time at the<br>beginning of the year<br>Sonia's mother was perfectly healthy and<br>there was no sign of any disease at all<br>over the course of 2010 she underwent a<br>terrifying cognitive decline and passed<br>away actually before the end of year it<br>was completely unclear what caused this<br>decline but on autopsy it was revealed<br>that her brain was full of protein<br>accumulations called called prion<br>tangles and it was clear that she had<br>died of one of a number of different<br>prion diseases in this case the disease<br>called fatal familial insomnia so this<br>is this is in some cases a genetic<br>disease I'll tell you more about that in<br>a minute so as a result Sony's mum was<br>sequenced for the P R and P gene which<br>is known to be the gene that contains<br>our variants the predisposed towards<br>prion disease and she was found to carry<br>a mutation that had previously been<br>reported as a dominant and basically 100<br>percent penetrant cause of severe prion<br>disease so them in that Sonya Sonya<br>herself had a 50% chance of carrying<br>that particular mutation and if she did<br>carry that mutation she would then go on<br>to suffer the same fate as her mother I<br>reckon Sonya then had to decide with it<br>whether Sonya would get tested and they<br>agonized over they agonized about this<br>for a number of days and then they<br>decided they would go ahead and get<br>tested and in early 2011 they received a<br>report from a clinical geneticist they<br>told them that Sonya does actually carry<br>the g17 a 10 mutation the mutation that<br>killed her mother and that means that<br>Sonya who at the time was in her early<br>30s was destined to suffer the same fate<br>so there are many different ways of<br>responding to this kind of information<br>and Sonya and Eric responded in a way<br>that I think many of us to be stay find<br>incredibly inspirational they quit their<br>jobs Eric at the time was a town planner<br>Sonya had graduated from law at Harvard<br>at Harvard so both of them ended those<br>careers and they began retraining as<br>biomedical scientists with a<br>single-minded goal to what's coming up<br>with the therapy for handling prion<br>diseases in the 20 years that they have<br>left before Sonya suffers the same fate<br>as her mother did<br>and so at the time that Sonia learned<br>this news she had of course one massive<br>question in her mind it was a question<br>that she asked the clinical geneticist<br>given that I carried this particular<br>mutation how likely is it that I will<br>actually go on to carry to manifest this<br>disease is this 100% certain or is 50%<br>what is my risk it's actually very<br>difficult to answer this and it's<br>difficult to answer this because of this<br>question and penetrance that is the<br>probability that a carrier of a specific<br>disease genotype will actually go on to<br>get that disease which turns out to be<br>largely unknown for most of the variants<br>that actually cause rare severe diseases<br>for some of them we have we have an<br>underlying assumption that they're fully<br>penetrant and that's true a lot of the<br>time we actually don't know we often<br>don't have large enough datasets to<br>really get a good estimate of that of<br>that number and penetrance is known to<br>be incomplete and variable for me<br>many disease-causing mutations the most<br>perhaps the most famous example of<br>course is the brac alone mutations which<br>cause disease in somewhere between 60<br>and 80 percent of the women who carry<br>them but some women who carry these<br>mutations will never go on to get breast<br>cancer and it's not really clear why and<br>again for most disease-causing variants<br>we actually have absolutely no idea what<br>the underlying penetrance estimates are<br>so so we know so here's what we know<br>about prion disease it is a rare disease<br>although not that rare actually about<br>our annual our lifetime risk of getting<br>prion disease is about one in 5,000 and<br>the interesting thing about brain<br>diseases although 15% of the cases are<br>genetic most of the cases as far as we<br>can tell a sporadic that is there<br>appears to be not no specific cause but<br>at some point during a person's life<br>prion proteins begin to accumulate and<br>once they do there was a feed there was<br>a positive feedback loop that results in<br>all of the prion protein in a person's<br>brain accumulating resulting in these<br>these horrible prion accumulation<br>diseases and of course there is also a<br>notorious 1% of prion diseases that are<br>caused by infections and that is by the<br>ingestion of meat that contains that it<br>contains a particular type of unfolded<br>protein and if you ingest that and<br>you're unlucky enough to have one of<br>those molecules propagate into your<br>brain that can then trigger again prion<br>accumulation within the brain and death<br>the most notorious example of this of<br>course is the variant creutzfeldt-jakob<br>disease epidemic in the UK which was<br>triggered by the ingestion of beef<br>contaminated from animals who had mad<br>cow disease so the genetic cases are<br>known to be caused by a whole variety of<br>different dominant gain-of-function<br>mutations in the<br>P Jean and we actually know a lot about<br>these mutations because almost every<br>individual who gets prion disease in<br>industrialized countries gets reported<br>to surveillance centers so that means<br>that nearly all of the diagnosed cases<br>actually have the PNP gene sequenced but<br>again like most diseases the penetrants<br>of these variants is very poorly<br>characterized so the clinician who Sonia<br>asked that question I wasn't able to<br>answer the question about how likely it<br>was that she was going to be able to go<br>on and get the disease except to be told<br>that it was very likely<br>so we then so Eric with help from my<br>team then said about answering that<br>question in much more rigorous and<br>robust fashion what what he did was - he<br>firstly set out with a I think perfectly<br>reasonable assumption that the variants<br>that cause prion disease in aggregate<br>should be no more common than the<br>disease itself so it is because this is<br>a dominant disease it must be the case<br>that all of the variants combined must<br>have an aggregate frequency that is less<br>than the disease itself that's a<br>reasonable and important calculation<br>it'll come up in a second<br>so what we were then able to do is to<br>collect a whole bunch of individuals<br>with information on the prion gene from<br>the general population at the time our<br>61 thousand individuals from X AK who<br>some of whom may well go on to get prion<br>disease but when these none of these<br>were ascertained on the basis of<br>numerator generative neurodegenerative<br>diseases so they should be broadly<br>representative of the general population<br>we're also through collaboration with<br>23andme able to get access to about half<br>women just over half a million customers<br>from this company all of whom had<br>consented to have their data used in<br>research and they had been genotyped for<br>16 variants that have been reported<br>previously to be pathogenic that is<br>causes of very severe prion disease and<br>then so that gives us our our controls<br>in terms of collecting cases Eric<br>through a pretty remarkable series of<br>feats of political jujitsu was able to<br>collect ten and a half thousand<br>individuals who had had who had had<br>prion disease and had the P R and P gene<br>sequenced and this actually amazingly<br>represents basically all definite or<br>probable prion disease cases in the US<br>Europe Australia and Japan over 15 to 20<br>years this is basically as a larger<br>collection of prion disease cases as we<br>are every likely to go and this this<br>collection allowed us to really start to<br>compare the frequency of the variants<br>that we found in our cases to the<br>frequency controls to learn more about<br>how likely each of those variants is to<br>cause disease so we can estimate with<br>some hand-waving<br>how many people in<br>exact collection of 61 thousand people<br>will actually go on to get prion disease<br>due to one of these genetic mutations<br>and that number I'm not going to go<br>through the calculations here they're<br>relatively robust that adds up to about<br>somewhere between one and two<br>individuals within our sixty one<br>thousand people that will actually go on<br>and get this disease but when we looked<br>at the sequencing data from the P R and<br>P gene across our sixty one thousand<br>samples what we found is a number that's<br>much higher so the thirteen different<br>variants that actually were previously<br>reported to cause prion disease and the<br>total number of carriers of each of all<br>of these variants combined was 52 that<br>is 30 times higher than what we see then<br>what we would expect to see given the<br>rank of prion disease in the population<br>there's a couple of possible<br>explanations here it could be that prion<br>disease is terrifyingly higher than what<br>we thought it was but that's pretty<br>unlikely given had with the surveillance<br>centers are it's possible that some of<br>these variants actually don't cause<br>prion disease at all in fact it's likely<br>that some of them don't that is they<br>were variants that just happen to be<br>observed in people who have sporadic<br>prion disease and as a result they've<br>ended up in these databases of disease<br>causing mutations but they don't<br>actually cause the disease and then a<br>final possibility is that some of these<br>variants may actually have some effect<br>on the risk of prion disease but not<br>100% so as they may have incomplete<br>penetrance but we haven't we have no<br>idea exactly what the numbers are of<br>each of those variants so here is how we<br>figured that out this plot is very<br>simple on the x-axis we have the number<br>of times that we have so for each of the<br>each of the previously reported variants<br>that have been reported to cause prion<br>disease on the x-axis we have the number<br>of times that were seen in our ten and a<br>half thousand cases on the y-axis we<br>have the number of times that were seen<br>in our sixty one thousand population<br>controls and and we can use this to<br>filter these variants out into three<br>broad categories on the x-axis we have<br>variants that are only seen in cases and<br>never seen in controls and these are<br>variants where we can be actually<br>statistically almost completely<br>confident that these cause disease in<br>everyone who carries them and<br>unfortunately that turns out to include<br>siRNAs variants d1 780 m so as far as we<br>can tell no no human being who has ever<br>carried the g17 ADN mutation and lived<br>into their 50s has not gone on to pass<br>away from prion disease on the y-axis we<br>have variants that are seen at a<br>relatively low frequency in cases but at<br>roughly the same frequency in controls<br>and so that what these tell us is that<br>these variants they're not enriched in<br>the Coase population and they almost<br>certainly are completely incidental<br>there's absolutely no association with<br>these prion disease we can actually<br>discard these these are errors in the<br>database<br>that actually fits with the literature<br>on each of these variants most of them<br>actually have incredibly unconvincing<br>associations with disease and no and no<br>links to family history for instance and<br>then in the middle we have this<br>intriguing class of variation three<br>variants that are too common in cases to<br>be completely to be completely benign to<br>common in controls to be completely<br>pathogenic so these must be variants<br>that have some effect on a prion disease<br>risk but not a hundred percent and in<br>fact we can actually calculate exactly<br>what sort of effect those variants have<br>that ranges between 0.1% so that is one<br>in a thousand a lifetime risk up here<br>and down to about ten percent risk for<br>for individuals who carry that fare and<br>down here so for these three variants<br>although there was some literature<br>suggesting that there might be an<br>incomplete effect on risk of disease<br>this is the first time we've been able<br>to actually put numbers on a person's<br>risk given that they carry that<br>particular variant and that of course<br>has really profound consequences for<br>genetic counseling for the carriers of<br>these of these particular variants and<br>just to give you one example about how<br>this information can actually have a<br>real impact on people's lives we<br>published this paper back in early 2016<br>it was accompanied at the time by a<br>opinion piece by Robert Greene and<br>colleagues and Robert who had actually<br>had access to data from our study<br>earlier than the rest of the population<br>had had actually been counseling a<br>person in his clinic a woman whose<br>mother had died from fatal familial<br>insomnia where the woman had also found<br>out that she carries the e 196 a variant<br>which had previously been reported to be<br>prion disease causing and she had<br>actually been told<br>based on carrying that variant that she<br>work was definitively going to go on and<br>actually suffer from prion disease in<br>fact e 196 a is the variant about which<br>we have by far the most statistical<br>confidence that it has absolutely no<br>association with prion disease and that<br>is consistent with all of the evidence<br>there's no Mendelian segregation data in<br>the literature there's no family history<br>in patients with the variant and the<br>frequency is absolutely no higher in<br>prion disease cases than it is in in<br>ancestry match controls so we can be<br>very confident this variant actually<br>doesn't increase her risk of prion<br>disease above baseline as a result<br>Robert was able to go back and tell her<br>that she doesn't in fact have a death<br>sentence in fact she has no more no more<br>risk of prion disease than anyone else<br>in the population which of course is<br>completely transformative so for Eric<br>and Sonja the news was not was not great<br>from this from this study what we<br>learned from this is that as far as we<br>Intel Sonja is<br>probability of going on to get this<br>disease assuming no therapy's developed<br>is 100% but there was a little bit of<br>good news and that came in the form of<br>these three variants up here these three<br>variants are variants that are protein<br>truncating variants in the P R and P<br>gene these are variants that are<br>predicted to completely obliterate the<br>expression of the prion protein and all<br>three of these were found in healthy<br>individuals healthy individuals who<br>survived require in old age actually one<br>of these individuals was 85 years old so<br>these these variants which fall very<br>early in the protein told us that<br>actually if you knock out 50% of the<br>prion protein in a person's brain we<br>actually did Western blots that I'm not<br>showing here to show you that protein to<br>show to demonstrate the protein levels<br>were reduced that that's actually<br>compatible with life and that's also<br>consistent with animal models that tell<br>us that if you knock out the prion<br>protein actually doesn't affect a human<br>lifespan and that's important because<br>what it tells us is that if we were to<br>design inhibitors that reduce the levels<br>of prion protein in Sonia's brain that<br>that will actually reduce her risk of<br>prion disease but what likely won't have<br>any severe adverse consequences so again<br>it's likely that inhibition of the prion<br>proteins is actually safe so Eric and<br>Sonya and now both PhD students at<br>Harvard actually quite remarkable that<br>remarkably as a result of donors at the<br>brode they've been given their own lab<br>at the Broad Institute they have a fully<br>kitted out by a safety lab that allows<br>them to study prion diseases at the<br>biosafety that's that's required then I<br>spent all of the time working on<br>developing therapies for prion disease<br>now working with six different<br>pharmaceutical companies to do so Sonya<br>is actually pregnant now she had<br>prenatal genetic testing and she is due<br>in July with a child who will not carry<br>the prion disease mutation and so of<br>course we wish them extremely well in<br>their venture forward into basically a<br>20-year process during which they need<br>to be able to figure out how they can<br>actually design a way of knocking down<br>the prion protein in Sonia's brain<br>getting that through the archein<br>processes that are involved in<br>regulating a delivery of a particular<br>drug and then deliver that to Sonya in a<br>way that will actually reduce her risk<br>of going on to get this disease so in<br>terms of whether or not this will<br>actually mean that Sonia goes on to get<br>the disease the statistics is relatively<br>clear everyone as far as we can tell who<br>was ever carried this particular<br>mutation and survived into their 50s has<br>gone on to die from prion disease but if<br>you are Sonya this question she says no<br>and the reason for that is that<br>firmly believes that it is actually<br>possible given how much we know about<br>the biology of this protein and exactly<br>what is required to engage this target<br>she thinks there is still a very good<br>shot of actually developing a therapy<br>that will save her in the 20 years that<br>she has left and to be honest I kind of<br>believe that so just very so ending on<br>that note this is actually very<br>difficult to transition from that story<br>into anything else but I wanted to<br>finish by concluding with it with a<br>couple of points about things we've<br>learned from studying large amounts of<br>human genetic variation data the first<br>is that naturally occurring human<br>genetic variation is all over the place<br>our population is full of it and we can<br>use it to learn more about the function<br>of individual human genes once we<br>collect enough information about<br>sequence and phenotype data this the<br>second is that the more humans we have<br>and the more diverse their ancestry the<br>more powerful our resources become and<br>so I think as we it is not just the case<br>that we need to sequence more<br>individuals we also need to think very<br>carefully about how we can sequence more<br>individuals in a way that benefits more<br>populations around the world and that<br>ensures that rare disease patients<br>regardless of their ancestry have well<br>matched reference samples and that is<br>something I think that the community the<br>human genetics community as a whole has<br>a moral obligation to work towards and<br>honestly at the moment we still don't do<br>a great job of this nonetheless over the<br>next five years it is unambiguously the<br>case that we will collect genome and<br>clinical data for millions of people and<br>if we can combine that in the right way<br>and if we can ensure that that data is<br>collected ethically and shared in a way<br>that benefits the rest of the community<br>this will have a profound impact on our<br>understanding of human genes if we can<br>combine this with other resources such<br>as model organism data cell models and<br>other information this really will<br>result in profound impact profound new<br>insights into human biology and also<br>allow us for the first time I think to<br>really understand the impact and the<br>penetrance of human disease causing<br>genetic variation so this is a<br>tremendously exciting time to be doing<br>human genetics and to take back human<br>genetics and convert it into an<br>understanding of human biology so that<br>I'll finish by thanking everyone who's<br>been involved in this work particularly<br>members of my lab which particularly<br>highlight my uncle lek here monk all is<br>a postdoc who actually led all of the<br>exact work mochel is actually just in<br>the process of starting up a new lab in<br>Yale I hope we starting that in November<br>so is there any any graduate students in<br>the audience who are interested in doing<br>postdocs in human genetics non-call is<br>open for business I also wanted to thank<br>all of the members of the production<br>team at the broad who are responsible<br>for building the infrastructure<br>for running a lot of the analyses that<br>are required to make this this process<br>work Ben Conrad and Matt built the X<br>acronym at websites that a big team was<br>involved in actually getting the ethics<br>together to make sure that we had the<br>consent to do all the things that we<br>need to do across 140,000 people the<br>broad genomics platform generated 90% of<br>the sequencing data and and funded<br>actually a lot of the work that was<br>required for X academia to exist and<br>Hale is a key a piece of software that<br>allowed us to run quality control across<br>140,000 people simultaneously and<br>finally big thanks to all of the 107 now<br>is that large number all the 107 our<br>principal investigators involved in the<br>Nomad consortium who agreed to donate<br>their data and not just willingly but<br>enthusiastically to the Nomad project<br>project for the purpose of benefiting<br>not themselves but the rest of the<br>community and particularly rare disease<br>patients and thank you all so much for<br>listening<br>thank you very much Daniel for a superb<br>talk there must be some questions I'm<br>sure yeah all right so Mike but the<br>x-axis x AK and nomads databases you<br>also have things abroad like the gtex<br>and other databases is there any attempt<br>to you know the the genomes all in one<br>part of the picture is there any attempt<br>to integrate these things so that if<br>I've type in a certain variant if I<br>typed in one of those prion variants I<br>would see not only from nomads but I'd<br>see from gtex the expression and<br>different things like that yeah that's a<br>great point so we're actually in the<br>middle of building a database called DB<br>LOF of database of loss of function<br>variants the goal for that database is<br>to take a particularly deep dive into<br>variants predicted to cause protein<br>truncation and that will actually<br>incorporate not just data for mexico<br>nomads but also data from the gtex data<br>set so that if we're lucky enough that<br>one of the nine hundred or so people in<br>G takes actually carries that variant<br>you'll be able to immediately see what<br>effect that actually has on allylic<br>allylic specific expression as well as<br>any deviation away from normal splicing<br>code that is triggered by that<br>particular variant we still have a lot<br>of work to do that is we still only have<br>been G takes only 900 individuals for<br>whom we have transcriptome data and I<br>think it's the in order to have you know<br>a much wider sense about how<br>particularly very rare variants impact<br>transcriptome those numbers need to get<br>much much larger so multi tissue<br>transcriptome data from many many<br>individuals will be needed to really<br>start to dig into the functional impact<br>thanks for a great talk just wondering<br>what the ETA on the copy number<br>variation data would be are you just<br>doing it on the whole genomes or not<br>trying to do an exam we will do it on<br>both there's a new a new copy number<br>variant caller for exomes that's<br>actually just been rolled out in gatk<br>and that we will be running that across<br>all all<br>126,000 exomes that'll be available but<br>most of the workers as you guessed is<br>being done in the whole genome data so<br>that's four different structural variant<br>callers that we run across the all of<br>the beginning with our 15,000 currently<br>available samples and eventually scaling<br>it up to all 50 to 60,000 in the next<br>release<br>we're hoping to have an SV call set<br>released by the end of this year at<br>least for the 15k for the big data set<br>that will likely be sometime in the<br>first half of next year behind you hi<br>Daniel and with your concern when you<br>use families that always have trouble<br>saying you're looking at a particular<br>loss of function variants associated<br>that with phenotype I'm just wondering<br>how many other loss of function variants<br>there are in general in those families<br>yeah it's a great question because<br>obviously given that on average you have<br>like a sixth of the genome that is<br>orders I guess in these individuals it's<br>pretty hard to say if you've got one<br>individual it's pretty hard to say<br>whether a given phenotype is due to that<br>specific homozygous very no there may<br>just be some other recessive variant<br>elsewhere in the genome so the average<br>number of rare homozygous loss of<br>function variants in the carrier<br>individuals is one so that that's ended<br>up in quite useful 42 percent of people<br>have at least one and the the modal<br>number is exactly one but nonetheless we<br>find individuals who have many more rare<br>homozygous loss of function variants the<br>record so far is eight one individual<br>who has eight separate genes knocked out<br>which is pretty impressive actually<br>and and so in those individuals<br>obviously it's very difficult to make<br>any specific phenotypic correlation with<br>the genotype it is and will require I<br>think typically having substantial<br>numbers of knockout individuals<br>preferably across multiple families to<br>account for genetic background to really<br>to really be definitive in saying this<br>gene when you knock it out does this<br>exact thing for the cases where we only<br>have one individual knocked out I think<br>that can give us some clues and if it<br>also you know lines up with things that<br>we see in knockout mice or other systems<br>then then we might start to believe that<br>that phenotype is actually real but it's<br>gonna be it's going to take a lot more<br>convincing based on nf1 to actually get<br>to that that level of biological<br>understanding<br>for individuals who you've done either<br>whole genome or exome sequencing on is<br>ancestry self-reported and if so how<br>does ancestry or ethnicity correlate<br>with the genomic variation yes there's a<br>good question and in practice we<br>actually ignore self-reported ancestry<br>entirely so we actually we do have it I<br>think in about a third of the samples in<br>the in the database but we completely<br>ignore it because the ancestry estimates<br>we get from our genetic data are now so<br>much cleaner than anything we could get<br>from self-reported data that we have<br>tended to not look at it too much in the<br>cases where we have taken a deeper dive<br>I guess the one the one place where we<br>have looked a little bit more closely is<br>individuals who self-report as being<br>african-american and they're the the<br>estimates are pretty good basically<br>people who report as being<br>african-american or 80% African on<br>average consistent with other studies<br>but we get a pretty wide spectrum of<br>variation within that and we find some<br>individuals who self-report as African<br>American who as far as we can tell a<br>completely European so there's a bit of<br>a mix I'm just wondering about the that<br>curve the MSM taupe towards 8,000 genes<br>yeah you're gonna reach that's it's a<br>third over a third of all human genes<br>have you discovered any in congruence<br>between say mouse knockout phenotypes<br>and what you're observing in those<br>individuals yeah we have and I'm still<br>not quite sure I want to make them so in<br>in actually both the papers that we put<br>out over the last couple of years on<br>human knockouts and concern were just<br>individuals there were genes in that we<br>report in those papers is being knocked<br>out where there is a homozygous knockout<br>phenotype that is extremely severe<br>actually okay there's a multiple cases<br>like this so that could be one of two<br>things in some cases it could well be<br>that the human biology is different from<br>the mouse biology that's not not a<br>terrible surprise that obviously happen<br>sometimes it could be the case that<br>there's a zygosity difference in the<br>impact between between humans and mice<br>perhaps it may be that for some some of<br>those variants we've just missed<br>annotated their loss of function<br>variants so they do look like they're we<br>have filtered out the annotation areas<br>as best we can but it may still be that<br>the human that these humans actually<br>carry a variant that effects a<br>transcript that the cell doesn't need<br>for some reason or there's some other<br>way that the cell can actually escape to<br>loss of function that's possible as well<br>maybe the mouse phenotype report is<br>wrong as well we don't but we don't<br>really know<br>we did we are very interesting into<br>those actually we've got a graduate<br>student the moment who's done some<br>digging at the comparison between the<br>discrepancies in human and knockout and<br>mouse knockout phenotypes and the plan<br>is to dig into a bunch of those and see<br>if we can actually try to untangle the<br>possible the possible explanations a<br>little bit further just wondering are<br>you planning to release the clinical<br>data from the patients that are in an<br>exact or no mesh right here no we'd love<br>to and I think the challenge there<br>there's a couple of challenges that one<br>is that the clinical data in many ways<br>is much more identifying than the data<br>that we have in nomad<br>it's so it's difficult to figure out a<br>way of releasing that in a way that<br>doesn't allow you to triangulate back to<br>kind of individual level genetic data<br>which we have to be very careful about<br>so I mean as you know X second Nomad at<br>the moment we release frequencies across<br>all of the individuals but we don't<br>release the individual genetic variation<br>calls and that that makes it kind of<br>difficult to combine the two what I<br>think we will be able to do is for<br>individual variants we'll be able to<br>hopefully be able to release at least<br>some phenotypic annotations so that we<br>can say particularly for the loss of<br>function variants these are these are<br>variants where carriers have been<br>observed to have this particular range<br>of phenotypes but not give you detailed<br>per individual phenotype data across<br>those individuals and so that is the<br>current plan we've been rolling that out<br>within the consortium and we're hoping<br>which we will be able to really silly<br>some version of that next year yeah one<br>last question what we took Daniel pretty<br>hard so just following them from the<br>last question with the Nomad and X add X<br>act the clinical data that you have the<br>relative ratios of clinical data that<br>you have across the different population<br>groups how does that compare pretty<br>uneven actually that's a good question<br>I'm just going to go through it in my<br>head so we have so we have a lot of<br>there's some weird<br>there's some weird clustering actually<br>within the Nomad dataset so for instance<br>if you if you look within our European<br>cluster a lot of our Europeans or<br>Scandinavian ancestry we have this<br>enormous over-representation of finns<br>for instance which is a consequence of<br>the fact that we have that so much human<br>genetics work has been done in those<br>populations actually if you are a<br>Finnish ancestry your odds ratio for<br>being an X AK is 70 so this huge over<br>representation of those those<br>individuals and for the Finns of course<br>our the amount of Finnish<br>we have is astronomical so basically<br>everyone in that in that collection has<br>buried any type data for the individuals<br>of South Asian ancestry most of those<br>actually fall into the promess cohort<br>our Pakistani cohort there we have lots<br>of finished up data but it's very<br>cardiovascular focused and then there's<br>that then it's sort of windle's down a<br>little bit after that so that we have<br>there's a bunch of other cohorts within<br>the dataset with longitudinal<br>longitudinal health information I think<br>they're looking thinking through it<br>relatively scattered across the<br>different ancestry groups and the one<br>ancestry group where we have almost no<br>phenotype data is East Asians where for<br>various reasons the cohorts that fall<br>over there we just don't have good<br>electronic medical record data for so<br>that's a I don't know that's a bit of a<br>vague answer we have basically it's not<br>there's no predictable patterns between<br>ancestry and and an electronic medical<br>record data and we still have a lot of<br>work to do in particular ancestral<br>groups to really pull that dataset<br>together okay well look I think I might<br>draw it to a close there we've had a<br>real treat tonight thank you so much<br>Daniel for a superb tool</p></main><footer style="margin-top: 2rem; background: #0001; padding: 2rem; text-align: center;"><p>We Are The University</p><ul style="list-style-type: none; padding: 0; margin: 0;"><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li></ul></footer></body></html>