<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>UC scientists make biomolecular breakthrough – WATU</title><link rel="stylesheet" href="/styles.css"></head><body><header><h1 style="font-family: 'Arial Black', Gadget, sans-serif;font-style: italic;font-weight: 900;text-transform: uppercase;">We Are The University    </h1><nav><ul><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li><li><a href="/blog">Blog</a></li><li><a href="/archive">Archive</a></li><li><a href="/authors">Authors</a></li><li><a href="/games">Games</a></li><li><a href="/reading-groups">Reading Groups</a></li></ul></nav></header><main><h2>UC scientists make biomolecular breakthrough</h2><p> <a href="https://www.scoop.co.nz/stories/SC1609/S00058/uc-scientists-make-biomolecular-breakthrough.htm">Original post</a></p><h3><span>university-of-canterbury</span></h3><p>Tue Sep 20 2016 12:00:00 GMT+1200 (New Zealand Standard Time)</p><p><h1>UC scientists make biomolecular breakthrough</h1>
<p><strong>Tuesday, 20 September 2016, 10:04 am</strong><br><strong>Press Release: <a href="https://info.scoop.co.nz/University_of_Canterbury">University of Canterbury</a></strong></p>
<p><strong>UC scientists make biomolecular breakthrough</strong></p>
<p>A team of University of Canterbury researchers has made a scientific breakthrough in biomolecular interactions, which will help discover the determinants of gene expression, in new research published today.</p>
<p>Discovering how gene expression is controlled is a major research question in molecular biology. Of the large amounts of variation in gene expression, the best studies can explain around 50 per cent of this variation, UC Bioinformatics Senior Lecturer Dr Paul Gardner says.</p>
<p>“We have discovered a previously overlooked mechanism that controls additional variation in expression. Our new approach considers the extent of how random interactions between messenger RNAs (that are translated into proteins) and non-coding RNAs can influence gene expression,” Dr Gardner says.</p>
<p>The UC team has proven, using some large and complex datasets, that these interactions play a major role in gene expression. The findings, which used UC High Performance Computing, have been published today by eLife scientific journal in a research paper titled: <em>Avoidance of stochastic RNA interactions can be harnessed to control protein expression levels in bacteria and archaea</em>, proving that stochastic mRNA and non-coding RNA hybridisation has a major impact on protein expression.</p>
<p>Advertisement - scroll to continue reading</p>
<p>Members of UC’s Biomolecular Interaction Centre – doctoral student Sinan Uğur Umu, Associate Professor Renwick Dobson, Associate Professor Anthony Poole and Dr Gardner – were involved in the research project. Umu spent his three-year PhD predominantly working on this project after Dr Gardner proposed the original hypotheses over four years ago.</p>
<p>Dr Gardner says there are a number of potential biotech applications as a result of the research findings.</p>
<p>“Particularly in the space of designing mRNAs for genes so that protein production can be improved. We are working with colleagues at Callaghan Innovation and Powerhouse on the best way to do this.”</p>
<p><strong>What is RNA?</strong></p>
<p>Ribonucleic acid, or RNA is one of the three major biological macromolecules that are essential for all known forms of life (along with DNA and proteins). A central tenet of molecular biology states that the flow of genetic information in a cell is from DNA through RNA to proteins: ‘DNA makes RNA makes protein’.</p>
<p><strong>About the research</strong></p>
<p>Many genes carry information for making proteins. To make a protein, a working copy of the information stored in DNA is first copied into a molecule of messenger RNA. These RNA messages are then interpreted by the ribosome, the molecular machine that makes proteins. Many messages are produced from each gene, and each message can be read multiple times. Thus, it should follow that the number of messages produced dictates the number of proteins made. However, this is not the case and the number of proteins produced cannot be completely predicted from knowing the number of messenger RNAs.</p>
<p>Cells control how much of a given protein they produce through interactions between the messenger RNAs and other regulatory RNAs. The regulatory RNAs bind directly to a message and impede protein production. Because there are millions of RNAs in a cell, these interactions have evolved to be highly specific. Nevertheless, it seems inevitable that messenger RNAs would encounter other RNAs too, which could “short-circuit” gene regulation and lead to less protein being produced.</p>
<p>The UC research team asked if such short-circuit events are selected against during evolution. Computational tools were used to predict the strength of binding between the RNAs found in the dominant forms of microbial life on Earth: the bacteria and the archaea. This approach revealed that the majority of messenger RNAs bind more weakly to the most common RNA molecules found in cells than would be expected by chance. Weakened binding should prevent the RNA molecules from becoming tangled with each other and ensure that protein levels are not perturbed by unintended interactions between highly expressed messages and other RNAs.</p>
<p>To test this hypothesis further, the researchers generated versions of the gene for a green fluorescent protein that differed only in how well their messenger RNAs could avoid interacting with the most abundant RNAs in <em>E. coli</em> cells. Those messengers that were designed to avoid interacting with other RNAs yielded far more protein than those that were not.</p>
<p>The findings show that taking this kind of avoidance into account can improve predictions about how much protein will be produced and should therefore make it easier to control protein production in experimental systems.</p>
<p>Finally, the messenger RNAs of some bacteria do not show such clear avoidance. However, these bacteria have a more complex internal cell structure. This finding hints at an alternative means for avoiding short-circuiting events that could be used by more complicated cells, such of those of animals and plants, which also contain much larger numbers of RNAs.</p>
<p>The link for the team’s paper is <a href="https://elifesciences.org/content/5/e1347">https://elifesciences.org/content/5/e1347</a></p>
<p>ends</p>
<p><a href="http://www.scoop.co.nz/about/terms.html">© Scoop Media</a>  </p>
<p>Advertisement - scroll to continue reading</p>
<p>a.supporter:hover {background:#EC4438!important;} @media screen and (max-width: 480px) { #byline-block div.byline-block {padding-right:16px;}}</p>
<h3>Using Scoop for work?</h3>
<p>Scoop is free for personal use, but you’ll need a licence for work use. This is part of our Ethical Paywall and how we fund Scoop. Join today with plans starting from less than $3 per week, plus gain access to exclusive <em>Pro</em> features.  </p>
<p><a href="https://pro.scoop.co.nz/Individual/?from=ProIn24">Join Pro Individual</a> <a href="https://pro.scoop.co.nz/using-scoop-for-work/?from=ProIn24">Find out more</a></p>
<p>Find more from <a href="https://info.scoop.co.nz/University_of_Canterbury">University of Canterbury</a> on InfoPages.</p>
</p></main><footer style="margin-top: 2rem; background: #0001; padding: 2rem; text-align: center;"><p>We Are The University</p><ul style="list-style-type: none; padding: 0; margin: 0;"><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li></ul></footer></body></html>