<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>We Are The University</title><link rel="stylesheet" href="/styles.css"></head><body><header><h1 style="color: #fff;font-family: 'Arial Black', Gadget, sans-serif;font-style: italic;font-weight: 900;text-transform: uppercase;">We Are The University    </h1><nav><ul><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li><li><a href="/blog">Blog</a></li><li><a href="/videos">Videos</a></li><li><a href="/authors">Authors</a></li></ul></nav></header><main><h2 style="text-align: center;">COVID-19 in Aotearoa: insights from genomics Presented by Dr Jemma Geoghegan [46:07]</h2><p style="text-align: center;"><a href="https://www.youtube.com/watch?v=KBK-d8oQpjc" target="_blank">Watch on Youtube</a></p><p style="text-align: center;"><a href="https://www.youtube.com/channel/UCPVjZXotZ5oX9jyXlFFRj2w" target="_blank">University of Otago - Ōtākou Whakaihu Waka</a></p><img src="https://i.ytimg.com/vi/KBK-d8oQpjc/maxresdefault.jpg" alt="Thumbnail for video titled: COVID-19 in Aotearoa: insights from genomics Presented by Dr Jemma Geoghegan" style="width: 100%;"><div class="tags"></div><h2>Description</h2><p>March with Otago lecture series. COVID-19 in Aotearoa: insights from genomics Presented by Dr Jemma Geoghegan</p><h2>Transcript</h2><p style="opacity: 0.9; font-size: 0.8em">Transcripts may be automatically generated and may not be 100% accurate.</p><p>my name is sheila murray i am the<br>director of development and alumni<br>at the university of otago and it's um<br>our team that's put on this lovely<br>webinar tonight and it's<br>my privilege to welcome you all here and<br>just ask you to please make sure that<br>you're on mute um throughout the<br>presentation if you were um here for the<br>first one which professor tony valentine<br>did we will follow exactly the same<br>format we will have a presentation from<br>gemma<br>um<br>which she will be introduced by<br>professor tony valentine tonight and<br>then we'll have time for some questions<br>which will be managed through the chat<br>function so if you have a question<br>please put it into the chat in the end<br>um and then once again as we have with<br>the first presentation we will um put<br>this up onto our youtube and let you<br>know uh when it's available because we<br>are recording tonight's function<br>so i'd like to um uh welcome professor<br>tony valentine he's the deputy vice<br>chancellor external engagement and he is<br>um tonight introducing gemma um thanks<br>versa valentine<br>uh<br>so many thanks sheila and it's great to<br>see you all and thank you for joining us<br>for this the second of<br>the uh march with otago lecture series<br>we have a really fantastic speaker this<br>evening an outstanding researcher who<br>also happens to be an otaku graduate so<br>it's my real pleasure to introduce<br>dr mcginn<br>uh<br>gemma is an evolutionary biologist and a<br>neurologist who has a really strong<br>research focus on emerging infectious<br>disease gemma's research has focused on<br>several areas<br>across the field of virology these<br>include determining the fundamental<br>patterns and processes of viral<br>evolution<br>ecology and emergence and using<br>metagenomics to relate to diversity<br>reveal the diversity structure and<br>evolution of the virus sphere and<br>examining case-specific evolution of<br>major viral infections that affect<br>humans and animals<br>now she's also been called a modern day<br>explorer<br>and her background is steeped in<br>exploration<br>born just outside of edinburgh and in<br>cooper<br>where her first love and expertise was<br>dance<br>she was accepted into the scottish<br>ballet and performed in london glasgow<br>and edinburgh<br>however deciding that ballet was not a<br>lifelong career she enrolled at the<br>university of strathclyde and completed<br>a bachelor of science with honours and<br>genetics<br>specializing in forensic biology she<br>then went on to undertake a phd here at<br>the university of otago with professor<br>hamish spencer<br>she then carried on with her exploration<br>moving to the united states where she<br>worked at new york university with the<br>group working on hiv which was her first<br>experience of working with viruses and<br>where she found her knowledge of<br>evolutionary biology brought a different<br>and important perspective from most<br>other virologists whose background was<br>in a clinical setting<br>her next stop on her journey was a<br>post-doctoral research fellowship at the<br>university of sydney before getting her<br>own lab at macquarie university<br>in early 2020 she returned to otago for<br>a joint position with otago's department<br>of microbiology in immunology and the<br>institute of environmental science and<br>research and was just in time for the<br>first case of covert to arrive in new<br>zealand so<br>perfect timing or terrible timing i'm<br>not sure which one<br>um and for another course occurred in<br>february 2020.<br>from here<br>gemma became a household name in new<br>zealand as the population was introduced<br>to the importance of being able to uh<br>genomic sequence covert positive cases<br>as part of al tildau's arsenal in the<br>fight against the pandemic<br>in 2020 gemma was one of ten to be<br>awarded a prestigious rotherford<br>discovery fellowship<br>and with this fellowship she will<br>continue with her research to<br>investigate how viruses evolve<br>what barriers may hinder or promote<br>virus emergence<br>we are so delighted that gemma navigated<br>a course after those lengthy and<br>complicated travels back to us here at<br>the university of otago and appreciate<br>the amazing work that she's done<br>alongside a lot of other otago<br>researchers and informing and leading<br>new zealand's response to the covert<br>pandemic<br>so i'll hand over to gma in just a few<br>seconds but just remember you can put<br>your questions into the chat function<br>and those will be processed at the end<br>of jimmy's talk and remember to keep<br>yourself on mute<br>through the presentation so i'll now<br>pass over to gemma who is going to talk<br>to us on covert 19 and old tierra<br>insights from genomics<br>many thanks<br>gemma<br>hi everyone thank you very much tony um<br>for that introduction and um yeah so i'm<br>going to talk about a little bit about<br>my role in um the coven 19 response<br>using genomics to better inform<br>how we dealt with this pandemic in new<br>zealand<br>so<br>it is likely that all viruses that<br>infect humans have origins in animal<br>hosts but for the most of these viruses<br>we have no idea where they came from and<br>when they made that jump into humans so<br>for most viruses that make this leap<br>across a species barrier<br>um they just end up being dead end<br>spillover infections where they can't be<br>transmitted between person to person<br>but sometimes<br>viruses can get lucky<br>and<br>when they get lucky they can genetically<br>or ecologically adapt to spread between<br>people and sometimes they can cause<br>epidemics or even pandemics<br>so i have spent the past<br>nearly decade studying how this<br>processes work and what makes viruses<br>more likely to jump to new hosts<br>so on the 30th of december 2019 the<br>world was alerted to four cases of an<br>unexplained pneumonia at wuhan central<br>hospital by this very memo that i saw on<br>twitter<br>and<br>just using genomic sequencing technology<br>the very technology that we've been<br>using today<br>um chinese scientists were able to find<br>a cause the causative pathogen for these<br>um cases<br>and so just 12 days after those first<br>cases were admitted to hospital<br>uh the the coronavirus genome of this<br>new virus was shared publicly on twitter<br>by my good friend eddie holmes<br>and this very first genome allowed the<br>first diagnostic test to be developed as<br>well as the first vaccines to be<br>designed it also set the scene for data<br>sharing during this pandemic which has<br>been vitally important<br>so i guess the highest stakes question<br>in this pandemic is where did this virus<br>come from<br>so having this genome of the new<br>coronavirus enabled us to identify the<br>closest genetic relatives<br>and these viruses were from bats and<br>pangolins however it's important that<br>these viruses<br>of these species are not the actual<br>source of the new coronavirus that was<br>circulating in humans because there's<br>about 30 years of evolution between the<br>viruses and bats and pangolins and the<br>one that causes copper 19.<br>the fact that we don't know about where<br>this virus has directly come from yet<br>isn't strange though and coronaviruses<br>like many viruses love to jump to new<br>hosts in fact there's been about nine<br>coronaviruses that we know of that have<br>been found to infect humans<br>but<br>but seven of these um<br>have actually made that jump into humans<br>over the last 20 years<br>and that seemed to be at the center of<br>most of not all of these emerging events<br>in coronaviruses so we know that bats<br>are important reservoir and natural<br>reservoir for coronaviruses<br>so my colleagues and i<br>looked at where those first early cases<br>in december where those patients both<br>worked and lived and their proximity to<br>both<br>the live animal markets in wuhan and the<br>wuhan institute of virology to find if<br>we could tell where the origin was and<br>we found that the patterns of virus<br>spread strongly suggested that the<br>wildlife markets were a likely origin<br>which is actually the very place where<br>you'd expect a virus emergence event to<br>occur<br>so i think it's fair to say that<br>genomics has played a starring role in<br>this covid19 pandemic with over nine<br>million genomes of the virus shared<br>online now but of course only a tiny<br>fraction of these have been sampled from<br>new zealand<br>so new zealand's 2020 epidemic looked<br>like this where we had a mixture of<br>imported cases during that very steep<br>first wave<br>followed by a brief elimination of the<br>virus in the community i think we became<br>quite smug then<br>followed by numerous border leaks which<br>led to<br>community transmission<br>so throughout this new zealand has<br>largely had a scientific based approach<br>in its decision making around its<br>response to coping 19 and dealing with<br>the pandemic right from the beginning<br>you know testing was available serology<br>projects were underway and then later we<br>had bilateral agreements with many<br>vaccine companies<br>and from the beginning and what started<br>out just as a research project with me<br>and my colleagues at esr we were able to<br>we were ready to begin sequencing the<br>viral genomes when that first case<br>so what is genomic sequencing um well if<br>you think that it's staring at a bunch<br>of letters on a computer screen you<br>would be correct<br>and we made a short movie about how this<br>all works though and the link is in is<br>on there and i'll put it put it in the<br>chat later<br>and you can feel free to watch that it<br>is aimed at kids but it's it's really<br>for everybody<br>to enjoy<br>but since this is a university lecture<br>this is phylogenomics 101. so<br>this is how<br>this is how your list this is about your<br>lesson about what we do with those<br>genomes once we've sequenced them and<br>this is a cartoon of five genomes from<br>five different infections from cases a b<br>c d and e<br>and these colored dots represent<br>different genetic mutations in these<br>genomes<br>so by first aligning these genomes we<br>can compare their similarities and<br>differences<br>and then<br>understand where these mutations relate<br>to each other<br>so using this alignment we construct<br>their evolutionary history by estimating<br>a phylogenetic tree so a phylogenetic<br>tree is a bit like a family tree and it<br>shows how cases are related to each<br>other so in this example a and b are<br>genetically identical meaning that<br>they're genetically and likely<br>epidemiologically linked whereas they're<br>distantly related to case e<br>so it's important to monitor the ongoing<br>evolution of crovid 19 virus or czar's<br>coronavirus ii um but very early on<br>there was various groups that propose<br>nomenclature to to name different<br>lineages or variants of the virus and it<br>became a problem it was quite messy as<br>this headline suggests and so the who<br>did step step in<br>naming variants of interest variants of<br>concern or variance under monitoring<br>usually designated a greek letter which<br>i'm sure we'll run out of soon enough<br>but importantly monitoring this<br>evolution is important it first provides<br>us a timeline so we can see that all of<br>these cases had a common ancestor which<br>which um converge around november 2019<br>suggesting it's unlikely that the virus<br>was around before that time<br>it also suggests that this b lineage<br>here and you can see in orange is the<br>most dominant lineage um around and in<br>fact all variants of concern have come<br>out of that lineage<br>and in fact new research suggests that<br>lineage a that red one at the top and b<br>were actually two separate host jumps<br>from from species likely at that wohan<br>market<br>so<br>let's go back to that first wave in<br>early 2020 when we had one of the<br>steepest epidemic curves observed<br>worldwide<br>so we began collecting samples from<br>testing labs sequencing the genomes of<br>positive cases and at that time you know<br>the biggest struggle was actually<br>getting positive cases to sequence<br>because this was our research project<br>and it was never part of the national<br>response at this time<br>having said that we got really good<br>temporal and spatial distribution of of<br>cases to sequence<br>um most dhbs um contributed positive<br>cases for us to sample and sequence<br>so we got a really good spread over the<br>country<br>so this is a phylogenetic tree where all<br>the colour dots are represented cases<br>from new zealand and all the gray dots<br>are cases from overseas<br>and the timeline you can see at the<br>bottom there represents that first wave<br>in 2020<br>so all the different colors represent<br>different genomic variants that were<br>circulating in that time<br>and this map shows where these genetic<br>variants ended up which was pretty much<br>everywhere in fact<br>new zealand saw just about every genetic<br>variant that was circulating around<br>during that time in that first wave<br>and from these data we can estimate that<br>there was about 300 separate<br>introductions of the virus into the<br>community over that time frame<br>but interestingly though we found that<br>even though there were so many<br>introductions only about 20 of these<br>introductions actually led on to further<br>transmission and this is probably<br>because we had one of the strictest<br>lockdowns really early on in the<br>pandemic<br>but most of these introductions that led<br>on to further transmission were actually<br>originated from places in north america<br>which is quite surprising given that the<br>virus emerged in asia as well as you<br>know um<br>australia being our closest geographical<br>neighbor but it probably suggests that<br>the virus is much more prevalent in<br>north america than it previously<br>recognized<br>so<br>another thing that we can estimate using<br>genomics is a really important parameter<br>in infectious disease called the<br>reproductive number or r naught<br>and it's basically a number of of<br>infection rate or rate of transmission<br>so if r knot is bigger than one the<br>infection will always spread<br>and this is kate winslet an excellent<br>movie contagion explaining this<br>and are not varies depending on the type<br>of infection and also the environment<br>so arnold can vary um for example in<br>seasonal flu on average um it's about<br>just over one meaning just over one<br>people on average gets infected by an<br>infected person but for measles this can<br>be as high as 18 meaning one infected<br>person<br>on average infects about 18 other people<br>so using genomics though we can estimate<br>that are not for individual clusters<br>so during that first wave our biggest<br>cluster was the bluff wedding cluster if<br>you remember ashley bloomfield talking<br>about that<br>and so we remember at the very start<br>this was a super spreading event that<br>happened just before lockdown<br>and are not we estimated to be around<br>seven<br>and then by the second week of lockdown<br>this fell to nearly zero so we can<br>estimate um are not at different time<br>frames at different<br>in different clusters and and this<br>points the fact that lockdown was really<br>so because we haven't until you know<br>august last year um when we had delta we<br>hadn't had to deal with huge amounts of<br>community transmission in new zealand um<br>that means that we've been able to<br>conduct really detailed genomic<br>investigations that about really sort of<br>rare events that would really really be<br>overlooked if it happened elsewhere<br>so for example you might remember this<br>headline um where a man hit his positive<br>for corporate 19 after spending 14 days<br>in miq with two negative tests but later<br>in the community a week later he tested<br>positive<br>so we'll call this man g and then on the<br>same day his household contacts h and i<br>also tested positive<br>but using genomics we were able to link<br>this person back to a nine-person chain<br>of transmission that went all the way<br>back to india<br>so what happened was a b and c<br>gotten a flight from<br>india to christchurch<br>and um were moved to a hotel in in<br>christchurch for an miq facility now a<br>and b and c were all on the same flight<br>that was about 30 occupancy so this<br>flight was um relatively empty and they<br>were all physically distanced but they<br>just happened to be within a few rows of<br>each other<br>and a and b tested positive on day one<br>and were moved to the isolation section<br>of the<br>hotel um kc didn't test positive until<br>day 12. this late test<br>um really points to<br>evidence suggesting that<br>it was likely in flight transmission<br>given that it was to see was just in a<br>few within a few rows of cases a and b<br>so then they were moved to the isolation<br>section of the hotel as well<br>so kc just happened to be an adjacent<br>room to d and e before they were moved<br>to the isolation section<br>and<br>and there was no physical contact<br>between the kc and case d and e<br>the only um<br>the only surface that they might have<br>both touched was a bin lid outside of<br>the hotel in the hotel corridor because<br>they had to empty their own rubbish<br>and if you remember back then um there<br>was immediate blame for this bin lid and<br>the ministry of health turned all the<br>bin lids into food pedals<br>however we looked at um the cctv footage<br>of when they were in miq together and we<br>found that there was a four there was<br>over 20 hours between<br>um c touching the bin lid and d and e<br>touching the bin lid so we thought that<br>was improbable<br>but what we found was on day 12 when c<br>was when they were when they were tested<br>on day 12<br>and what happened was you know the nurse<br>knocked on kc's door they stepped out of<br>the room into the unventilated corridor<br>they were swabbed by the nurse and then<br>stepped back into the room and the door<br>was closed<br>then there was a 50-second window<br>between closing that door to kc's room<br>and opening the door to case d and e's<br>room<br>and and then they were swabbed as well<br>and so<br>there was never any um<br>physical contact or they were never in<br>the same place at the same time<br>but yet we think this window of<br>opportunity provided aerosol<br>transmission um as a router transmission<br>in this case so we had an engineer look<br>at these rooms and there was no<br>connecting vents or plumbing<br>but what they found was<br>a net positive air pressure inside the<br>rooms meaning when you open a door to an<br>unvented corridor the air flows out<br>meaning it you know um exacerbates the<br>sort of aerosol particles that carry<br>virus virus in them and and um<br>you know<br>it helps a aerosol transmission happen<br>so the aerosol transmission is really<br>hard to prove and this is one of the<br>first evidence worldwide it helped prove<br>that covert 19 was airborne<br>so d and e became infected on day 12 but<br>obviously tested negative so they left<br>miq on date or day 14 and they just<br>happened to get a flight back up to<br>auckland and they were just happened to<br>be seated directly behind case g<br>so case um g became infected on that<br>flight it was just an hour long flight<br>and both of these travel groups went on<br>to infect their household contacts<br>so this wouldn't be possible without<br>genomics um genomics were able to tell<br>us how these cases were linked and by<br>studying the mutations between each case<br>we were able to understand who infected<br>who in the likely timeline and direction<br>so another example is when we found a<br>very peculiar genomic situation in miq<br>which related to this like headline here<br>cover 19 genomics transmission in<br>auckland's jet park miq hotel<br>so what happened was two separate<br>two separate groups traveled to new<br>zealand on different flights on<br>different days from different countries<br>so case a was in one room and the other<br>travel group shared an adjoining room<br>adjoining rooms right across the<br>corridor and importantly this corridor<br>was over two meters wide thought to be<br>far too<br>far too much of a distance for a droplet<br>transmission to occur<br>in cases a and e<br>were positive on their day one of<br>revival and the rest of the the cases um<br>ease travel group slowly gradually<br>tested positive during their stay<br>and the only case not to test positive<br>was case f of this travel group and<br>interestingly they were the only person<br>that had to receive the vaccine at that<br>time so none of this sounds unusual at<br>this point but when we sequence the<br>virus genomes we find that the rest of<br>cases e's travel group were not actually<br>infected by case e but instead they were<br>infected by case a who was across the<br>corridor<br>and these tests were repeated and<br>repeated again and performed by<br>different laboratories because we wanted<br>to rule out contamination or accidental<br>sample swapping<br>so again we studied the cctv of the time<br>that they were in miq and we found there<br>was three simultaneous door openings of<br>about three to four seconds long<br>usually when food was delivered to the<br>rooms<br>and again we think this provides really<br>good evidence the aerosol that<br>the aerosol transmission happens and<br>copen 19 is airborne and this might not<br>sound surprising but actually it's been<br>a really hot topic during this pandemic<br>of the likely um<br>likely<br>the best route of copen 19 transmission<br>and the most prominent route is airborne<br>transmission<br>so all of our border incursions have<br>been informed by genomics and linking<br>cases to the border or identifying<br>cluster memberships of cases in the<br>community and this has all helped to<br>eliminate coping 19 in the community so<br>all of these many trees that you can see<br>on this figure represent border leaks<br>that we've had before delta and<br>identifying with which border case that<br>they they um they were leaked from<br>so<br>genomics had really been used in a<br>unique way in new zealand<br>always at the front of informing how<br>cases are linked and linked to the<br>border<br>but delta didn't only just change new<br>zealand's strategy for dealing with<br>copin 19<br>changing from elimination to a<br>suppression strategy but it also changed<br>the role of genomics in this pandemic<br>here too<br>so now you know my colleagues at esr we<br>continue to sequence the proportion of<br>genomes to monitor the viruses spread in<br>evolution but of course you know the<br>sheer number of positive cases as well<br>as the number of at-home tests that<br>people are doing mean that um there are<br>fewer and fewer actual samples to<br>sequence however<br>continuing to sequence um especially<br>those cases that end up in hospital and<br>others that are deemed sort of high risk<br>so today we're dealing with several<br>outbreaks in the community which are<br>dominated mainly by two um lineages of<br>omicron and these are called ba1 and ba2<br>you might have heard about these<br>lineages in the news recently so you've<br>probably heard that ba2 is even more<br>contagious compared to ba1 and these two<br>sister lineages are both classes omercon<br>and they both emerged in<br>southern parts of africa around about<br>the same time and ba1 was the first to<br>take off worldwide but it's been<br>replaced by b82 not entirely but ba2<br>seems to be increasing increasing in<br>frequency and frequently and also<br>dominating in many countries now<br>and it's the case in new zealand where<br>ba2 now represents um nearly 80 percent<br>of the genomes that are being sequenced<br>and sampled<br>so in fact with ba2 it sneaked into new<br>zealand several times and a few of these<br>incursions have really taken off<br>spreading<br>spreading countrywide causing widespread<br>outbreaks so you can see um in this<br>phylogenetic tree all these blue<br>lines represent um ba2 outbreaks these<br>things were associated with things like<br>the sound splash event so quite super<br>spreading events as you would expect but<br>it seems to be increasing in frequently<br>in frequency and it does have a slight<br>growth advantage over ba1 so it's even<br>more contagious meaning it's about 30<br>percent um more contagious compared to<br>ba1<br>so it's really important even though<br>we're not informing the elimination<br>strategy and sequencing every single<br>positive case in new zealand it's<br>important that ongoing genomic<br>surveillance is um is part of our<br>infectious disease program going forward<br>and this isn't just to monitor the<br>evolution and emergence of new novel<br>variants which will come um but it's<br>also for detecting things like reverse<br>pseudonyosis events um this is where<br>viruses spill back to from humans to<br>animals can potentially then<br>um go undergo lots of mutation and<br>adaptation in the animal<br>reservoir and then re re-emerge into<br>humans and this is a potential<br>hypothesis for the emergence of of<br>especially omercon because it's so<br>divergent compared to the other variants<br>and finally i'd just like to highlight<br>some of the work that me and my team<br>have done<br>where we've<br>implemented this data into public facing<br>interactive narratives using um<br>using neck strain which um luckily um a<br>friend in new zealand um built as part<br>of his phd<br>and so next strain um has been a tool<br>used worldwide and we've been able to do<br>a new zealand specific version where<br>we're able to<br>have display our data and our<br>interpretation of it as well as you can<br>download this data and play around with<br>it you can make your own phylogenetic<br>trees and and study the spread of the<br>virus through time and space<br>great thank you so tony are you going to<br>uh<br>thank gemma<br>i i can thank you so much gemma it's<br>fantastic to<br>have uh both if you like the biological<br>foundations of what we're dealing with<br>explained so clearly but also the the<br>contribution of the the kind of science<br>you're at the frontiers of<br>to our response and<br>as i said before as an institution we're<br>incredibly proud of the contribution our<br>researchers have made to our national<br>response<br>our response to azure observed has been<br>really firmly anchored in the science<br>and we're really pleased that you've<br>taken the time out this evening to share<br>expertise with us and i can see that<br>there are quite a few questions<br>and hands up and questions will come<br>through the chat so<br>thank you again and i'll pass over to<br>sheila who will moderate the q a<br>thank you very much tony thanks tony<br>um so the first question is um<br>given only a few coronaviruses have<br>leaked what are the chances of two in<br>close succession<br>yeah so um as i said in my talk there<br>have been nine coronaviruses which have<br>leaked um<br>and seven in the last 20 years so it<br>does seem that um<br>that the frequency of virus host jumping<br>generally not just for chronoviruses is<br>increasing because we're you know ever<br>encroaching on on wildlife's habitat so<br>um<br>generally it's not particularly just<br>coronaviruses but the the frequency of<br>virus hosts jumps into humans is<br>increasing um<br>who knows if it's going to be any the<br>next one is going to be a coronavirus or<br>not<br>so how do you think the pandemic would<br>have played out if we were not able to<br>do not genomic sequence jimmer<br>um<br>yeah so our knowledge of this um of the<br>evolution of this virus um has all come<br>from genomic sequencing the the<br>knowledge that we've gained from<br>um from the emergence of new variants to<br>um the development of mrna vaccines<br>that's all come from genomic sequencing<br>and so we would never have um had had<br>vaccines this quickly or or the mrna<br>vaccines that have proven so effective<br>against combating the virus and um<br>without the genome so<br>genomics has played a huge role um not<br>just for you know in new zealand where<br>we've used it to inform elimination but<br>but most countries have been<br>contributing at least some genomes and<br>to monitor the situation in different<br>areas of the world<br>so um it will continue to to be a huge<br>role and as i said nine million genomes<br>so can you tell us anything about the<br>headlines we've seen about a hybrid<br>between delta and omicron<br>yeah so we know that um recombination is<br>a really important part of adaptation um<br>generating new variation<br>in in virus genomes and<br>and<br>it's a really important part in<br>coronaviruses in particular but it<br>happens in all viruses either they<br>recombine or reassort depending on their<br>type of genomes they have and it's not<br>surprising that we're seeing the um more<br>um reports of recombination it's sort of<br>hard to detect because um you're<br>detecting basically somebody's had a<br>mixed infection of of two variants at<br>the same time and then the chances of<br>actually sampling and sequencing that um<br>that infection and then detecting that<br>that hybrid so it's called a<br>recombination event and it's not that<br>surprising that now<br>delta and omicron were just so prevalent<br>in the communities worldwide that um the<br>chances of finding those<br>so um this question is so the probable<br>probability of further<br>mutations is quite high question mark<br>is there any future time where it could<br>be possible that covert 19 invited<br>commas runs out or do you see it<br>mutating and jumping between humans and<br>animals and back forever<br>good question um<br>i<br>you know<br>it's<br>it's likely that it's now endemic in<br>humans um endemic doesn't necessarily<br>mean forever but<br>um<br>but for the foreseeable future um<br>there's no way of eliminating it from<br>from the human population and um and<br>that's mainly because of the inequal<br>distribution of vaccines and and so on<br>which always allows the virus to hang<br>around in in various populations and the<br>various um the very populations where<br>we've been able to detect new variants<br>um<br>tend to be populations with much lower<br>vaccination rates and so<br>um you know<br>these populations um are sort of<br>reservoirs of of um new virus variants<br>because of the high transmission<br>there's also you know<br>possibilities of um diverse suanose's<br>events like i talked about um to create<br>new variants as well um as well as<br>populations of immune suppressed people<br>for example you know south africa is<br>also gripping with uh um<br>huge hiv pand epidemic and and so um you<br>know lots of patients that would become<br>infected with corporate 19 would have<br>this sort of suppressed immune system<br>allowing um chronic long-term infections<br>providing sort of a training ground for<br>you know adaptation to<br>to humans which means um<br>new variants are always going to come<br>along um so i can't see this ever<br>so is there any delta circulating in new<br>zealand at present the graph<br>that you had showed the two omicron<br>variants during the last few weeks is<br>there delta<br>so<br>about two weeks ago we last sampled<br>delta in the community it was only a<br>fraction of genomes compared to the huge<br>amount of omicron genomes however at the<br>moment you know because the vast<br>majority of people are doing a rapid<br>antenna test usually at home meaning<br>that there's no sample to send you know<br>to esr for sequencing so basically the<br>only um<br>the really the only um samples that<br>we're getting to sequence are ones who<br>end up in hospital which is really<br>important to do in fact that's kind of<br>the most important to do to understand<br>if there's any differences in severity<br>in the variants um that we're<br>experiencing here<br>um<br>and and so that's only a really tiny<br>fraction um so we're sequencing under<br>500 genomes a week at the moment which<br>is um<br>a very small proportion of actual<br>reported cases and a tiny proportion of<br>actual infections um given that reported<br>cases are probably about a third of of<br>actual infections so um for the past two<br>weeks we haven't actually sequenced any<br>delta but doesn't mean it's there um<br>it's likely remaining at low levels i<br>mean<br>even the most stringent lockdown level<br>four didn't um eliminate delta from the<br>community so and there's still lots of<br>susceptible hosts so i'd be surprised if<br>delta's completely burnt itself out<br>okay<br>you um you showed us some images from<br>social media in your presentation gemma<br>has social media changed how science has<br>worked in the global response to the<br>pandemic has it helped scientific<br>collaboration<br>it's definitely changed um you know<br>people are tweeting results before<br>even a pre-print um which is a<br>pre-publication um<br>pre-peer-reviewed publication comes out<br>so<br>um definitely science um<br>the way that people share data and share<br>results um has changed it's not always<br>for the good um because you know the<br>scientific um peer review processes has<br>become still extremely valid and<br>important but<br>it's really um sort of allowed um<br>you know data to be accessible as well<br>as people to kind of keep on top of the<br>huge amounts of of scientific literature<br>that's coming out daily um there are<br>always sort of summaries um or threads<br>posted online which really helps digest<br>yeah<br>okay is there um is there any other<br>questions that anyone has tonight<br>we've run through the ones that have<br>been put up in the chat<br>um is there anything else people would<br>like to<br>ask<br>if not um<br>we can bring tonight<br>oh got one coming here um<br>in your opinion do you think the general<br>public have a better understanding of<br>science now<br>oh definitely i mean science has really<br>been at the forefront um of this<br>response especially in new zealand you<br>know um<br>you know the prime minister is talking<br>about mutations and genomic variants and<br>so on so it's<br>genomic sequencing particularly i think<br>has been in you know used in everyday<br>language now um whereas before i don't<br>think most people kind of was were even<br>aware of it so i think that um<br>it it's definitely helped to promote<br>science and and hopefully that's sort of<br>translated to the sort of next<br>generation who want to come and study it<br>too<br>yeah absolutely there's a whole lot of<br>new terminology out there that's become<br>very common in every day now so yeah<br>okay all right well um we don't seem to<br>have any other questions and we've had a<br>great response to<br>uh tonight so um i'd like to thank you<br>once again gemma i know tony has<br>formerly thanked you but thank you um<br>and thank you for everybody who's come<br>along and listened tonight<br>as we said the beginning we are<br>recording the session so<br>um<br>we will um<br>get this out to you as soon as we can<br>just one last question that's just<br>popped in very quickly<br>um<br>is anyone looking at human<br>genome information ari who's<br>susceptible or who's not<br>yeah of course there's quite a lot of<br>research into that um especially when it<br>comes to sort of long-term<br>effects um and who is susceptible to<br>more severe disease and long covered and<br>so on and there there does seem to be<br>some report some reported um results<br>suggesting that there is a hereditary um<br>element of it and it's not clear<br>um if it's sort of a genomic or<br>epigenomic or um what sort of<br>genes might be involved in that but um<br>there's some evidence clearly that<br>suggests that<br>um it could be hereditary<br>we don't know too much about that yet<br>though<br>okay all right so we've got another<br>quick question i've got time so there's<br>no rush but um we have got time do you<br>have a focus on how an animal to human<br>transmission can be reduced or prevented<br>yeah i mean we can first of all start by<br>not having live animal markets i think<br>that would be a really good<br>a really good start um<br>i think the wildlife trade um you know<br>particularly the sort of illegal<br>wildlife trade that occurs um<br>is you know um is a good place to start<br>and<br>and farming practices um<br>but<br>having said that we need better<br>surveillance as well so we can do a lot<br>more to to survey this sort of um human<br>animal interface so um sampling both<br>people and animals at this interface<br>people that work with animals farmers<br>and people that you know<br>work in slaughterhouses for example<br>there's lots of places that you can you<br>could just survey just to make sure that<br>these host jumps are not um occurring<br>but if they do you catch them<br>immediately<br>great okay<br>all right we might wrap it up here um<br>what we can do is if you have got<br>questions uh that you think of<br>absolutely oh people are now flying in<br>um<br>uh okay there's a oh i don't know i'll<br>ask that question um<br>i can answer that question<br>all right i'll do the one that's not um<br>tough has secretly shown which bearing<br>is more likely to give long covert<br>i know um<br>that's an interesting question um we<br>don't even know if omicron is likely to<br>give you long covered yet because it's<br>not been around long enough so<br>um<br>we know that vaccines um<br>largely reduce the chances of of getting<br>loan covered<br>and there hasn't been<br>enough investment in long-coded research<br>yet um clearly it is in its embassy so<br>we don't know too much about which<br>variants are more likely to do that<br>in fact<br>even<br>really mild cases of covert 19 can cause<br>long-term problems so it's not really<br>about the severity of of the illness at<br>the time um that dictates whether or not<br>okay do you want to ask that last<br>question gemma have we<br>yeah so um have we completely excluded<br>the wuhan institute of virology as a<br>possible source of or accidental release<br>or is it too controversial well um<br>so you know the scientific process looks<br>at the evidence and there's not a single<br>bit of evidence that suggests that has<br>come from the wuhan institute of<br>virology um whereas there is a lot of<br>evidence to suggest that it's come from<br>live animal markets um<br>in particular<br>um just about two weeks ago there was a<br>couple of papers that a couple of<br>pre-prints that um have made<br>a huge amount of headlines worldwide<br>because um<br>researchers looked at<br>you know<br>environmental swabs of those animal<br>markets and the proximity to where cages<br>that contained mammals were<br>and um<br>and you know it's it's really clear that<br>both lineages a and b were present at<br>that market so the chances of both<br>lineages coming to the market from mohan<br>institutology is very rare as well as um<br>it's a proximity to we where we know for<br>a fact that it was cages containing live<br>mammals like ferret badgers and um<br>things like that so<br>i think there's lots of evidence<br>supporting um supporting that as an<br>origin as well you know live animal<br>markets where um<br>there's a lot of evidence to suggest<br>that they were uh important in the<br>emergence of tsar's kobe one for the<br>first time uh in 2003 so um<br>yeah so<br>and it's not surprising right that that<br>that would be the very very very place<br>where a new virus would emerge<br>okay<br>so last question last question and if<br>there's any more then please um we will<br>follow up with you um<br>and answer that last question so there's<br>no indication of ge in the original<br>genome a exact base insert or splice<br>sites now that's very technical<br>so i'm guessing they mean genetic<br>engineering and and<br>and by splice sites i guess you mean<br>where the um where the virus binds to<br>our cells and there is uh a site called<br>a furin cleavage site um and that very<br>site um contains about five amino acids<br>that helps the the virus bind to our<br>cells and when it first emerged um the<br>virus first emerged it was actually um<br>ill-adapted to humans and the very same<br>sequence of that fury and cleavage site<br>was found in<br>loads of other coronaviruses within the<br>coronaveridae family<br>for example feline coronaviruses have<br>the exact same<br>um urine cleavage site and and lots of<br>others do as well um so it's nothing<br>special about the virus that causes<br>copen19 and now it's it's um that<br>cleavage site is actually<br>much better adapted to humans so if<br>you're engineering a virus to cause a<br>pandemic in humans why would you make it<br>ill-adapted to humans um<br>so none of that really makes sense<br>okay all right<br>uh rebecca's come through who asked the<br>question with an explanation of what a<br>splice site is but um<br>so um how long will it take for an<br>omicron specific vaccine is available in<br>your in your in your thinking<br>well i know that um with pfizer um and<br>moderno sort of announced as soon as<br>serve omicron emerged that they were<br>making an omicron specific vaccine<br>i'm not sure if that's the best idea i<br>mean<br>i'm not a vaccinologist but um you know<br>as we've seen over this um last two<br>years you know new variants has emerged<br>and dominated every about six months um<br>and so by the time<br>something's manufactured and widely<br>distributed um we'll be on to the next<br>variant i suppose so<br>i'm not sure if that's exactly<br>the best<br>the best case scenario i think that<br>perhaps working on vaccines that<br>can be<br>give protection against<br>a wide range of variants could<br>potentially<br>be really good but they both said within<br>a hundred days they could start<br>manufacturing it so<br>who knows if we'll get in here<br>okay all right thank you once again<br>gemma i found it absolutely fascinating<br>um<br>and thank you once again for the great<br>explanations um<br>thank you everybody for coming along uh<br>with the final in our march with otago<br>uh presentations is on the 30th so in a<br>fortnight we look forward to seeing you<br>all then and as i said we will be<br>sending out the recording um with the on<br>youtube with a link to that um in the<br>next few days thank you very much and<br>you</p></main><footer style="margin-top: 2rem; background: #0001; padding: 2rem; text-align: center;"><p>We Are The University</p><ul style="list-style-type: none; padding: 0; margin: 0;"><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li></ul></footer></body></html>