<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>We Are The University</title><link rel="stylesheet" href="/styles.css"></head><body><header><h1 style="color: #fff;font-family: 'Arial Black', Gadget, sans-serif;font-style: italic;font-weight: 900;text-transform: uppercase;">We Are The University    </h1><nav><ul><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li><li><a href="/blog">Blog</a></li><li><a href="/videos">Videos</a></li><li><a href="/authors">Authors</a></li></ul></nav></header><main><h2 style="text-align: center;">Antimicrobial resistance in the genomic era: Combating a major threat to global health [56:46]</h2><p style="text-align: center;"><a href="https://www.youtube.com/watch?v=uNdWpc6HDBI" target="_blank">Watch on Youtube</a></p><p style="text-align: center;"><a href="https://www.youtube.com/channel/UCPVjZXotZ5oX9jyXlFFRj2w" target="_blank">University of Otago - Ōtākou Whakaihu Waka</a></p><img src="https://i.ytimg.com/vi/uNdWpc6HDBI/sddefault.jpg" alt="Thumbnail for video titled: Antimicrobial resistance in the genomic era: Combating a major threat to global health" style="width: 100%;"><div class="tags"><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#antibiotics</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#health</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#antimicrobial resistance</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#lecture</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#international health</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#mrsa</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#research</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#metagenomics</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#2015</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#University</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#sharon peacock</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#otago international health research network</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#genomics</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#annual conference</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#public health</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#microbial sequencing</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#oihrn</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#surveillance</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#resistance</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#global health</span></div><h2>Description</h2><p>Professor Sharon Peacock presents the McAuley Oration at the 2015 Otago International Health Research Network (OIHRN) Annual Conference in Dunedin, New Zealand.</p><h2>Transcript</h2><p style="opacity: 0.9; font-size: 0.8em">Transcripts may be automatically generated and may not be 100% accurate.</p><p>this is a university of otago podcast in<br>a manner in a way our clinic dragon<br>theorem are not cattle alone el toro<br>cora i TV with my basket and with your<br>basket that people's lives will prosper<br>cure everyone and welcome here tonight<br>for our 2015 Macaulay duration my name<br>is Richard Blakey I have a great<br>privilege to be the deputy Vice<br>Chancellor for research and Enterprise<br>at the University of Otago I was going<br>to say here at the University of a tiger<br>but in fact we're not at the University<br>of Otago we're here in the Hadden<br>theater of otago museum so we are very<br>proud to be members of the tiger<br>community and in fact in this unique<br>place that we have where our museum are<br>a university our politic and our<br>wonderful sports precinct are so well<br>coordinated and organized together we my<br>job is just to welcome you here tonight<br>on behalf of the University and just to<br>say a few words about about the<br>Macquarie oration and then I'll hand<br>over to Professor Philip Hill who will<br>introduce our speaker tonight professor<br>Sharon peacock we have through the<br>University of Otago a number of<br>connected research activities and one is<br>our Otago international health research<br>network that is currently hosting its<br>8th 8th international meeting here in<br>Dunedin and we are very strongly<br>encouraged both research interaction and<br>engagement but also public interaction<br>and engagement with all of our research<br>activities and so this particular<br>research theme in activity has taken<br>that on board and every year they hold<br>an oration public oration now the two<br>two of the key leaders in this Philip<br>Hill and John Crump who were both with<br>us have their positions supported with<br>endowed chairs one is the Macauley chair<br>and one is the McKinley chair so every<br>year the Eurasian changes from one to<br>the other so this will be the fourth<br>Macaulay oration and it will have been<br>already for McKinley orations and<br>Macaulay duration is named after<br>Catherine McAuley who's<br>a figure who's quite prominent in the<br>establishment of Mercy Hospital in<br>Dunedin and through Mercy Hospital that<br>the Macauley name is able to be used for<br>both one of our endowed chairs and also<br>for the serration Kathryn was born in<br>Dublin in 1778 to an upper-middle-class<br>Catholic family with a father who had a<br>great concern for the poor but died when<br>she was 5 years old and a mother who has<br>said is not a particularly good manager<br>of the family estate and died herself<br>with when catherine was only 20 years<br>old but through her life catherine<br>developed great philanthropy and also<br>shared concern for her peoples and your<br>communities and she found that the<br>Sisters of Mercy established for the<br>visitation of the sick poor and the<br>protection and instruction of poor<br>families and it's through that that is<br>the connection to our Mercy Hospital<br>here that then enables us to be able to<br>have this as the the fourth University<br>of Otago and Otago international health<br>research network Macaulay oration so<br>with that brief introduction and by way<br>of some words of the background to the<br>name I'll hand over to Professor Philip<br>Hill to introduce our speakers tonight<br>thank you very much thank you mr.<br>Blakely it's my pleasure to introduce<br>you to our speaker tonight professor<br>Sharon peacock is a global leader in<br>infectious diseases and has recently<br>become the foundation director of the<br>new Bloomsbury Research Institute which<br>is a partnership between the London<br>School of Hygiene and Tropical Medicine<br>and the University College London the<br>new Institute brings together more than<br>200 scientists dedicated to addressing<br>the global challenge of infectious<br>diseases prof peacock originally trained<br>in medicine after nursing training in<br>fact and specialised in infectious<br>diseases in microbiology she gained her<br>PhD in 2003 for work on staphylococcal<br>pathogenesis which was conducted at the<br>University of Oxford and Trinity College<br>Dublin she then won a Wellcome Trust<br>intermediate fellowship and clinical<br>Tropical Medicine and spent six years at<br>the Wellcome Trust major overseas<br>program and Thailand from 2003 she then<br>became she became head of microbiology<br>at that unit she later returned to the<br>UK in 2009 to a chair in clinical<br>microbiology at the University of<br>Cambridge and she developed a research<br>programme which focused on the<br>translation of sequence based<br>technologies and diagnostic and Public<br>Health microbiology prof peacock plays a<br>number of important roles in the UK<br>she's an honorary faculty member at the<br>Wellcome Trust Sanger Institute and as<br>Deputy Chair of the Medical Research<br>Council infection and immunity board<br>she's made important contributions to<br>the UK chief medical office officers<br>reports on antimicrobial resistance and<br>has chaired the working group on<br>scientific priorities for the infectious<br>diseases component of the Department of<br>Health 100,000 genome project a<br>characteristic of profits groups<br>research which I noticed very strongly<br>when I was with that group last year on<br>sabbatical is that it's an intensely<br>practical practically focused and is<br>directly relevant to health while at the<br>same time being leading-edge<br>multidisciplinary and published in the<br>very best journals and the group have<br>now published over 300 publications a<br>key focus that they have is on the use<br>of microbial whole genome sequencing in<br>diagnostic and Public Health<br>microbiology and its application to<br>outbreak investigation and antimicrobial<br>prescribing prof P pop peacock is also<br>undertaking a series of Nomex<br>of genomic studies to identify<br>reservoirs of antimicrobial resistance<br>in health care facilities farms the food<br>chain and the environment prof PP Cox<br>group receives funding from the Medical<br>Research Council<br>through a large translational infection<br>research unit of consortium grant from<br>the Department of Health and Wellcome<br>Trust throughout a Health Innovation<br>Challenge Fund award among other funders<br>she continues to collaborate on research<br>in lower and middle-income countries<br>especially in Asia prof peacock is a<br>fellow of the academy of medical<br>sciences and was awarded a CBE for<br>services to Medical Microbiology in 2015<br>we're delighted Sharon that you just<br>accepted our invitation to come to New<br>Zealand to give to give them a call<br>Eurasian and also to stay for a longer<br>period on the William Evans fellowship<br>and I would like to invite you now to<br>come forward and deliver the 2015<br>well thank you very much Philip all I<br>can say is where did the years ago when<br>you hear about your life described like<br>that you suddenly realize that a lot of<br>water has passed under the bridge but<br>hopefully some very enjoyable and<br>successful water it's been too so I just<br>like to say how delighted I am to be<br>here today I've been in New Zealand now<br>for about a week and a half and the<br>hospitality I've been shown it's been<br>absolutely extraordinary and the scenery<br>I've seen has been quite breathtaking<br>and so there's no doubt Phillip that I<br>will be back as they say is this Arnold<br>Schwarzenegger I would say so it's a<br>deep honor to give the serration and my<br>own really in the next hour is to really<br>describe the critical situation we have<br>reached in relation to antimicrobial<br>resistance now I want to describe how we<br>reached this point and to consider how<br>do you know it technologies can address<br>the fundamental scientific questions as<br>well as how we can if this can be used<br>in a clinical setting to detect contain<br>and control the problem so for the<br>purposes of the next 45 minutes I'm<br>really going to focus on bacteria but<br>there's a problem of resistance in<br>viruses fungi and parasites and that's<br>no less important I just don't have time<br>to cover everything so there's no way of<br>starting a discussion about antibiotics<br>without mentioning Alexander Fleming<br>though he's credited for the observation<br>that bacterial growth on an agar plate<br>was inhibited by a mold called<br>Penicillium so you'll see here a<br>bacterial plate with a bacterial staph<br>aureus growing on it with a zone of<br>inhibition around this mold now actually<br>what the history books don't often tell<br>you is that he was not the first person<br>to observe this it's often the case when<br>we did back in history that many other<br>people have been involved in such<br>discoveries and I've just mentioned some<br>people here including this gentleman who<br>was a third year PhD student when he<br>described this phenomenon but wasn't<br>able to take it forward now we all know<br>from the history books that Alexander<br>Fleming was<br>was recognized the importance of his<br>observation but could not take it<br>forward into clinical practice and this<br>relied on Florey and chain shown down<br>here who were able to scale up<br>production of the mold juice and put it<br>into production so that by the time of<br>the Second World War were able to use it<br>for people on the battlefield when<br>affection would often be a major cause<br>of death if injury was not the cause of<br>death so that that period really was<br>followed by a heyday of antibiotic drug<br>discovery we discovered numerous drug<br>classes each having a different back to<br>your target they're listed there and<br>what you'll see here though is that<br>although we have antibiotics antibiotics<br>introduced down here and a nice long<br>list along here what you'll notice is<br>that antibiotic resistance was<br>identified very close to the time when<br>the antibiotic came into use now more<br>worrying still is that the drug pipeline<br>for tomorrow's antibiotics looks very<br>grim this graph shows the data for the<br>number of new drugs approved by the FDA<br>that's the approval body in the u.s. for<br>new drugs between 1983 and 2012 and I<br>don't need that you don't need me to<br>tell you what that graph is telling you<br>now there's been a lot of breast-beating<br>about why we've got to this position<br>where there's so much resistance but<br>we've got so few new drugs actually the<br>answers to that are really quite simple<br>to formulate first of all research and<br>development for any drug is lengthy<br>risky and costly so you'll see here at<br>the drug pipeline and it will take<br>around 15 years for drugs to get from<br>the first identification right the way<br>through to screening it's an expensive<br>and lengthy process you might say well<br>actually this is the same for any drug<br>so why are we failing when I said what<br>it's when we're not failing on cancer<br>drugs and other things where there are<br>several compelling reasons<br>and we need to understand this in order<br>to reverse the problem the first is that<br>the technical hurdles for antibodies is<br>now very high we've got all the easy<br>wings we know how to pick off bits of<br>the bacteria that is essential to life<br>and so we can kill them we've got those<br>nailed so the technical hurdles are much<br>higher now for new antibiotics now<br>antibiotics work very well if the bug is<br>susceptible but they're only used for<br>short periods now if I'm a<br>pharmaceutical company and I can market<br>a diabetic drug or a hypertensive drug<br>and someone takes it for the rest of<br>their life it's much better for might<br>might returns my profit if I do that<br>rather than an antibiotic which is only<br>needed for a week or two beats the other<br>thing is that antibiotics do have a<br>limited lifespan because once we start<br>to use them you get resistance emerging<br>there are also complications in terms of<br>clinical trials for antibodies they're<br>difficult to do and I won't go into<br>detail but that is one issue that is<br>another hurdle bottom line is that<br>profit margins are low and confidence<br>inability to recoup the cost is low for<br>antibiotics what does this mean it means<br>that most of the players have moved out<br>of this discovery market so you'll see<br>here in this chart a progression of the<br>that were the attrition actually of<br>companies falling out of drug discovery<br>so the state the word at the moment in<br>fact a couple of those companies have<br>since merged is that very few big<br>pharmaceutical companies are investing<br>in antibiotic discovery so where are we<br>at the moment what's the situation at<br>the moment well a comprehensive report<br>that captures this very clearly was<br>published by the World Health<br>Organization in 2014 now they looked at<br>just seven common bacteria that cause<br>disease from bloodstream infections<br>through to gonorrhea and they map rates<br>of resistance to these around the world<br>now here are some of the results so<br>you'll see some of the organisms here<br>which you may recognize ecoli which<br>causes you and your tract infections of<br>bloodstream infections and so on you can<br>see these seven<br>down here and then what's listed is the<br>drugs that are commonly used to treat<br>these types of infections<br>now they looked at rate spoon-fed of<br>resistance around the world and there's<br>quite a large range the question is what<br>does it mean to patients well this is<br>what it means so if you look at this<br>column here the deaths but people that<br>have infected with a resistant organism<br>compared with something that is not<br>resistant to that particular antibiotic<br>they have about two times the risk of<br>dying so it's very clear that patients<br>infected with resistant strains die more<br>now the impact of antimicrobial<br>resistance was also considered in health<br>and economic terms by Jim O'Neill so he<br>retired as the chairman of goblin sex<br>asset management in 2013 and he's best<br>known for actually coining the term<br>brick which is a that describes rapidly<br>developing countries that are really<br>shifting the global economic power away<br>from the developing g7 economies now he<br>was appointed by a prime minister in the<br>United Kingdom and he employed rand<br>europe and auditors KPMG very very<br>respectable organizations to try and<br>forecast the future if we don't do<br>anything about resistance now their<br>report has been really instrumental and<br>quite scary they predict in this chart<br>that antimicrobial resistance will cause<br>10 million deaths per year by 2050 so if<br>you look at this here you'll see the<br>inner circle is where we are at the<br>moment and this outer circle is where<br>we'll be in 2050 what you'll see is if<br>you compare it to cancer antimicrobial<br>resistant infections will be where head<br>of cancer now they also looked at this<br>in relation to economics and what they<br>forecast was that if we do nothing the<br>world will lose a GDP of a hundred<br>trillion dollars by 2050 so that's<br>effectively loss of GDP in the UK every<br>year for the next thirty five years now<br>some people have looked at these figures<br>and they think the evidence based on<br>which this is based<br>it's too flimsy to be this categoric but<br>what it has done is it's actually<br>brought people to start to think that<br>action must be taken so people around<br>the world are sitting up and taking<br>notice now you won't be very surprised<br>to find that there's inequality in the<br>impact on death if romantic romantic<br>micro resistance by 2050 so this is<br>again a report from a patron gym owners<br>report and you'll see that then that the<br>size of the circle relates to the number<br>of deaths that will occur most of the<br>deaths will fall in Asia and in Africa<br>and there are some figures that have<br>been produced by his group for example<br>one in four deaths in Nigeria by 2050<br>will be due to anti mccurry resistant<br>organisms so this is something that we<br>must sit up and take notice and do<br>something about now in a country like<br>New Zealand who's going to be affected<br>once antimicrobial resistance really<br>gets to grips well the fact is all of us<br>it will affect people throughout their<br>life stages from infancy to old age it<br>would off vector of a livestock and our<br>pets and this is something you might not<br>have thought about but actually if we<br>don't have effective antibiotics you<br>can't have cancer chemotherapy because<br>that makes people very in their<br>compromised and that often but risk of<br>infection if you can't treat that then<br>replacement very very common you get<br>antibiotics before the surgeon makes the<br>first incision that would mean that if<br>we don't have effective antibiotics<br>things like routine joint operations are<br>going to be much more risky okay so<br>having said that that rather apocalyptic<br>scene what I'd like to do now is to turn<br>my attention to the advent and utility<br>of genomic technologies now a discussion<br>about genomic technologies Israel is<br>really about the legacy developed by<br>Fred Sanger<br>he played a really central role in<br>development of sequencing technology so<br>he was a double-double a Noble<br>when a very few people on the planet<br>achieve even one let alone two and in<br>1977 he was awarded a Nobel Prize for in<br>fact earlier than that wasn't it you can<br>see on that 1955 he determined the<br>complete amino acid sequence of insulin<br>and what that did was really underpin<br>the discovery of how DNA was actually<br>structured so his work underpinned by<br>the work by Crick and colleagues he then<br>went on to develop something called the<br>died the oxy method for sequencing DNA<br>molecules here and he then sequenced a<br>genome of afar an infecting a virus that<br>affects bacteria so he was awarded two<br>Nobel prizes for that now his method was<br>actually very labor-intensive and it<br>became automated by the applied of<br>Biosciences to create this first<br>sequencing machine now some of the<br>people in the room might remember using<br>these in fact might still even use them<br>this is a capillary sequencer first used<br>or automated in 1987 and it I won't go<br>into the technology but essentially it<br>the output is a chromatogram a colored<br>chromatogram where each color represents<br>one base in the DNA now this is despite<br>its labor-intensive nurse this was used<br>for their first human genome sequence in<br>2001 now since then the pace of change<br>in sequencing technologies were really<br>nothing short of absolutely<br>extraordinary so you can see here that<br>back in 2000 let's just focus on<br>bacteria which are about four four<br>million base pairs in the year 2001 it<br>would cost me half a million pounds to<br>secret one genome and it would take 18<br>months now in 2014 and in fact the<br>figures are even better than this now it<br>would cost me 50 pounds and it would<br>take me two days so the technology is<br>really nothing short of staggering<br>so my next few slides really describe<br>the broad approaches that can be taken<br>in microbial sequencing to study<br>antimicrobial resistance now in the<br>early years of sequencing this was used<br>to generate whole genomes of single<br>single bacteria and actually that was<br>used for numerous bacteria<br>so this really defined the entire<br>genomic repertoire of a given strain<br>including drug resistant elements so for<br>example this image is actually an NMR<br>assay methicillin-resistant<br>staphylococcus aureus which contains an<br>element encoding and methicillin which<br>is shown here I don't see that clearly<br>we learnt a lot by doing that but in the<br>last five years the big change is that<br>we've been able to scale up the number<br>of genomes or can sequence so we can<br>really start to look at genomes in<br>population terms now what we can do with<br>this is really look at two things we can<br>look at the population phylogeny and<br>evolution nets of a population based on<br>something called the core genome now the<br>core genome is the genes that are<br>present in all members of that species<br>and so you can start to look at the<br>evolution of resistance you can look at<br>the carrier lineage is for resistance as<br>well as the genes that encode it we're<br>also interested in the accessory genome<br>that's the bits of DNA that get brought<br>into the bacterium and they're often<br>encode resistance now if you have a look<br>at this tree I'm just going to walk you<br>through this because I will show some<br>more trees in the future so if you're<br>not if you're not familiar with these<br>trees I'll just briefly mentioned what<br>what we've done here we've sequence a<br>collection of bacteria we've pulled out<br>of the genome the core genes that is<br>conserved across the species and we've<br>mapped it to a reference genome so<br>literally just lined it up to see what<br>similar or different and these<br>differences here are based on what's<br>called single nucleotide polymorphisms<br>or mutation single mutations and this<br>gives us this this figure here and it<br>shows us how related assets are to each<br>other now you can just see here in more<br>detail that this is the reference genome<br>here a part of one and this is how we<br>tile up the output of secrecy machines<br>and then we spot single points that are<br>different to our reference which then<br>forms one branch of a tree so that just<br>gives you some basic understanding of<br>how we do these trees if this isn't what<br>you do normally we're also interested in<br>as I said in in the XS<br>these are this is a an alignment of a<br>resistance transposed on a piece of DNA<br>that's encoding vancomycin resistance in<br>an organism called Enterococcus faecium<br>and we can just see the differences<br>across the this population in what that<br>looks like what genes it has what genes<br>it hasn't we can also do something<br>called meta genomics now in my previous<br>analyses we simply look at genes and<br>genomes from bacteria that grown in a<br>lab culture but we know that that's not<br>how bacteria survived that around us<br>there in populations and not not as<br>single isel's they're in very complex<br>populations so what we do here is we<br>really extract DNA from a complex<br>material such as soil or feces and then<br>we can type based on two types of<br>analyses we can just sequence all of the<br>DNA in there and there's little<br>fragments of lots of lots of different<br>pieces from different types of virtual<br>species or we can clone those fragments<br>into something called a vector and we<br>can get that bacterium to express the<br>phenotype so for example the piece of<br>DNA may be encoding resistance so really<br>what I'm trying to describe here is the<br>genomic tools we have available now to<br>start to study antimicrobial resistance<br>so having done that I'll now discuss the<br>examples of how these are being applied<br>so the first question I want to really<br>explore is when did antimicrobial<br>resistance begin and why so to try and<br>gain an understanding of this<br>fundamental question it's really quite<br>essential to us to developing our ideas<br>around prevention as well as predicting<br>future trends now if we look at the<br>timeline of antibiotic discovery I've<br>already mentioned to you that pretty<br>much as soon as we have a drug available<br>resistance emerges this suggests that<br>resistance is not a modern phenomenon<br>and was present before the drug was<br>developed for therapeutic use so what<br>I'd like to do is describe two studies<br>that have been published that look at<br>this problem how old is resistance<br>so the first was an analysis of genetic<br>material from the Late Pleistocene<br>permafrost sediments in Alaska so what<br>what this investigators did was they<br>drilled down to the permafrost which<br>hasn't actually thought for 30,000 years<br>that's because it was covered by<br>volcanic material so they they obtained<br>that very securely and the top view<br>shows where they actually did the<br>drilling here and this bottom view<br>really shows the core that they took and<br>how they actually took pieces of this<br>core to do sequencing and this has comes<br>back to this metagenomics approach where<br>they sequence the entire material so<br>what you'll see here is the animals that<br>they detected such as the map mammoths<br>and the bison now what they found<br>actually was that there was resistance<br>to a host of modern antibiotics well<br>before they were discovered penicillin<br>tetracyclines vancomycin and vancomycin<br>is particularly noteworthy because this<br>this resistance has popped up in the 80s<br>in a back to that causes infection in<br>hospitals called enter corpus PCM so<br>what they did was they took the sequence<br>and of these very ancient 30 at least 30<br>thousand years old and they aligned it<br>to modern sequences and in fact you can<br>see that the samples that are from the<br>soil are actually just labeled numbers<br>and they're very very related they did<br>some more work actually they looked to<br>see if these genes that they discovered<br>actually encoded something active and<br>they confirmed that the genes did<br>encodes on the active they encoded in<br>the enzyme and they also showed using 3d<br>3d structure using crystallography here<br>that super imposing the the modern<br>vancomycin resistance forgets the the<br>ancient one it was that they were very<br>very similar so resistance has been<br>around for a very long time now a second<br>study undertook a mic microbiome and<br>resistant analysis of bacteria on the<br>skin in the mouth and stone the yellow<br>mummy American American marry Indian<br>village which is a remote village that<br>live here now they according to history<br>have never actually been<br>contact with modern world they've never<br>seen antibiotics I think the estimate is<br>at least ten thousand years they haven't<br>come into contact with model to the<br>modern world and this was the first time<br>that they did contact the modern world<br>and what this what these investigators<br>found was that despite the fact that<br>they couldn't have possibly seen<br>antibiotics before they had 28<br>functional genes in their microbiota<br>that conferred resistance and this is<br>some of the drugs including some of the<br>semi synthetic and synthetic antibiotics<br>that's that means ones that are not<br>naturally made but ones that are<br>artificially made so resistance has been<br>around for a very long time so what I<br>want to come onto now is briefly touch<br>on antibiotics as selectors and<br>accelerators of resistance is always<br>around us it's been there forever but<br>what are we doing to drive the problem<br>now I've been in a seminar in it all day<br>seminar today and people have referred<br>to farmers and the use of antibiotics in<br>farms and this is just some of the<br>evidence that's been quoted to link<br>antibiotic use with farming now animals<br>receive antibiotics for three reasons<br>they can be treated because they're sick<br>they can be given antibiotics because<br>their growth promoters are you they make<br>the animal put on weight more quickly<br>they can also be used as a prevention to<br>try and prevent infection so if you look<br>at this chart here this shows the<br>millions of pounds sold of antibiotics<br>over time in the US and this bottom line<br>is antibiotic so to treat sick people<br>and the top line is antibiotics sold<br>from meat and poultry production so it's<br>very obvious that we use a large amount<br>of antibiotics in animal husbandry now<br>what happens if you take antibiotics<br>away from farming well in Denmark they<br>have done a series of studies and they<br>have very tight antibiotic control there<br>so in this in this figure here what<br>you'll see is the percentage of bacteria<br>this particular one is Enterococcus<br>faecium<br>that are resistant to this and this<br>antibiotic Avila mice in which is very<br>similar to one<br>using clinical practice now when they<br>first started to monitor this in 1980 at<br>1995 rates were very high but then as<br>they withdrew over my seniors so did the<br>resistance for so there's a direct<br>result between driver and the amount of<br>resistance and I won't go into too much<br>detail about the graph at the bottom but<br>this is about eco light they that<br>encodes resistance to what's called<br>extended spectrum beta lactamase so<br>their resistance the common drugs we<br>give people and again you'll see in pigs<br>this time that as as the drugs Bala<br>Kevlar spoons in 2010 bit then<br>resistance fell now that's good evidence<br>for a correlation and what this has led<br>to for example in the CDC report is that<br>really they put humans at the center of<br>antimicrobial resistance and they<br>predict how resistant spreads from<br>animals who get antibiotics that<br>breathes out drug-resistant bacteria<br>that gets into meat in our food chain it<br>contaminates the environment and<br>patients who go into hospital who also<br>have antibiotic resistance emerging<br>during their treatment and going around<br>in a cycle in the human population I<br>think we have to be slightly careful<br>because life is never that simple<br>and what I haven't heard commonly say is<br>that humans give resistance to animals<br>now what is the evidence for this well<br>there is one particular study which i<br>think is is absolutely groundbreaking<br>actually performed by colleagues in<br>cambridge and what they looked at was<br>something called salmonella typhimurium<br>dt 104 now this is an organism that is<br>spread from the thought to be spread<br>from animals to humans and it causes<br>gastroenteritis it's multi<br>drug-resistant which is why it's one<br>important and it's widely disseminated<br>across the world now the paradigm is<br>that that this is spread from animals to<br>humans both through direct and foodborne<br>transmission now what Allison did here<br>was she actually took 135 humanized<br>at 130 and 13 veterinary isolates from<br>Scotland and she compared them using<br>whole genome sequencing and what they<br>found was really quite surprising what<br>this tree shows is as follows so the red<br>is human isolates from humans<br>the blue is isolates from animals and<br>there are several observations the first<br>is that the most deeply rooted that's<br>the ones down here branches for isolates<br>are from here as that means that this<br>originated from humans are not from<br>animals<br>you'll also observe that colors cluster<br>together so isolates from a single host<br>tend to cluster together rather than<br>free mixing and when she looked at the<br>rate of animal to human or human to<br>animal events they were very few so this<br>actually is the first piece of evidence<br>that it's not quite as simple as are<br>saying that animals give us all the<br>resistance and we simply have to stop<br>antibiotic use in farming although I<br>would accept that it's an important<br>point what she also looked at was the<br>presence of antimicrobial resistance<br>determinants in the population so some<br>resistance genes genes that incur<br>resistance were shared between the two<br>populations you can see here but a<br>greater number of determinants were<br>actually in human derived isolates<br>compared with animal derived isolation<br>so antimicrobial resistance isn't just<br>about farming so my final section really<br>is to go on to look at the use of<br>microbial genomics in clinical practice<br>and in particular I'm going to focus on<br>outbreaks and surveillance I'm going to<br>focus on Staphylococcus aureus because<br>actually it there's quite a lot of work<br>done in this but it could be used for<br>pretty much anything<br>now in this room about 40 percent of you<br>will be carrying this organism it's<br>associated with a wide range of diseases<br>and but there's a problem with<br>methicillin-resistant so that's the<br>antibiotic that tends to be used for<br>staph aureus it's readily spread from<br>person to person and it can cause<br>outbreaks in hospital you will have<br>heard about Mercer or and<br>say in the newspapers outbreaks in<br>hospital are often caused by a very<br>specific clerical strain it's adapted to<br>hospital environments and current typing<br>methods really like the resolution to<br>tell members are the same back to your<br>clone apart now this particular piece of<br>work is actually actually one of the<br>very first by Simon Harris and<br>colleagues in 2010 to start to sequence<br>back to your populations and this is a<br>really one of the most key papers in in<br>that cho-jen omics actually in the<br>literature still it actually seems quite<br>simple he took 63 isolates of lineage of<br>staph aureus which he called sequence<br>type Tofino and that's what the current<br>typing method calls it you can see the<br>timescale there 1982 to 2007 including<br>20 isolates from a single hospital over<br>seven months and from 15 countries and<br>five continents and it was very simple<br>he grew the bugs he extracted the DNA<br>and he compared the genetic sequence now<br>this is the phylogenetic tree and it's<br>color-coded by continent so what you'll<br>see here is that despite the fact that<br>by conventional typing methods they're<br>all the same there's a huge amount of<br>discrimination and that is very useful<br>for clinicians the other thing to note<br>is that there's something called file a<br>geography going on here so the colors<br>cluster together<br>that means there's genomes are actually<br>located in particular parts of the world<br>now see here this blown-up blue is Asia<br>and there are two Reds there which is<br>from Europe well how is that we track<br>those cases down and in fact they both<br>had very strong Asian links so what this<br>showed for the very first time is we can<br>use sequencing to track back to movement<br>through continents across continents<br>which is very powerful indeed now this<br>is for the the genomic geeks that<br>hopefully there'll be some in the<br>audience one of the things we really<br>need to think about is how fast are<br>bacteria evolving because if I want to<br>work out if an individual has spread and<br>where I say to another individual I need<br>to know how quickly the bacterium is<br>evolving so that I<br>tell whether say 20 mutations is<br>important or a hundred mutations is<br>important between the two bacteria so<br>what's the speed of evolution actually<br>we can do this very straight in a<br>straightforward way now if you remember<br>the tree that I showed you previously<br>what we do is simply look at the<br>distance from the root to the tip of<br>that tree versus year of sampling and we<br>end up with a graph like this now here<br>you have these isolates plotted out in a<br>different way<br>distance from the root to the date of<br>isolation and then what you can do is<br>two things you can extrapolate to when<br>that clone first emerged and you can<br>also work out the rate of mutation how<br>many how many single nucleotide<br>polymorphisms per year the answer is<br>seven and a half unlike forty to our<br>crop resisted saying that so so what we<br>then did having done that that proof of<br>principle study was to go on to look at<br>how we could use this in the clinic this<br>was actually one of the first studies<br>published in the literature using<br>sequencing in clinical practice we<br>spotted an amorous a outbreak in<br>intensive care unit neonatal intensive<br>care unit and it's really simple we<br>secret stem RSA from seven cases in the<br>outbreak and for that we didn't think<br>we're part of the outbreak and actually<br>the tree was beautiful what we had was<br>we had a cluster of eye slits here which<br>demonstrated that we had an outbreak<br>going on confirmed that and then we had<br>our reference and non-related isolates<br>up here yes you can use sequencing to<br>investigate outbreaks we actually we<br>actually set ourselves a much more<br>demanding task then and that is to use<br>sequencing in real time to see if we get<br>interrupting outbreak and I think that<br>more that by luck than judgment actually<br>we also revealed how we're completely<br>unaware of the extent to which hospital<br>outbreaks spilled into the community so<br>what we did here was again we worked in<br>intensive in it in a special care baby<br>unit and in this unit babies are swapped<br>every week for MRI say<br>and at this point our infection control<br>doctor said we have a problem because we<br>have three cases of mrs a carriage all<br>at the same time what is going on now<br>they've been scratching their head for<br>about six months because they've seen a<br>trickle of mrs a case is coming through<br>the unit but never enough for an<br>outbreak because there were gaps in the<br>in the time line here and here so they<br>didn't really ever think there was<br>enough for an outbreak so they knew that<br>they had these babies with MRI se but<br>they didn't know much more than that<br>they didn't know whether they were<br>related or not so what we did was we<br>took these Isis and we sequence them and<br>here you've got a slightly different<br>tree in that you've got this like target<br>like a darts board and you can see from<br>here the way it steps out relates to the<br>number of mutations and the relatedness<br>now there are only 20 mutations that<br>differentiated there's 14 isolates and<br>so we were confidently able to say you<br>have an outbreak and at that point we<br>knew that we heard an issue now what we<br>then did was we wondered whether this<br>could have extended any further and of<br>course this is the dark matter an<br>outbreak because we never actually have<br>the ability to see this spreading into<br>the community so we went to all out our<br>macro laboratory and we said we want all<br>of the isolates from the freezer that<br>have the same antibiotic resistance<br>pattern as our outbreak strain we got 19<br>isolates we sequence them and actually<br>ten of them fitted our tree very nicely<br>indeed they were related to the outbreak<br>but so we got their notes out and found<br>that they were people living in the<br>community and nearly all of them had<br>disease they had skin a soft tissue<br>infection they had infection of the<br>breasts during breastfeeding so we<br>wondered how they might link up now<br>looking at the notes became very obvious<br>how they linked up to here they're<br>color-coded on the tree so here we find<br>that a baby and a mother shared the same<br>strain that here a family shared the<br>same strain we find here that actually<br>mothers have been spreading mr-s a on<br>the maternity ward none of this was<br>known through conventional infection<br>trol if we were secret seeing these<br>isolates on a regular basis in fact even<br>right back here<br>we could have nip this in the bud now<br>the interesting thing about this story<br>is everything went away for 64 days and<br>everyone breathe a sigh of relief until<br>this infant was admitted and had MRI say<br>we sequenced it and it was actually on<br>the genetic tree just here party<br>outbreak missing well that's very odd<br>where has that come from we thought that<br>the only answer could be a member of<br>staff so we swapped 150 members of staff<br>that work in the unit and we found an<br>individual that was carrying em RSA now<br>this time we actually sequence 20<br>separate colonies from their nasal swab<br>and what we found was that it was right<br>here this baby was clearly implicated<br>with his health care worker we<br>decolonized that person successfully<br>they went back to work and the outbreak<br>was over I have one last outbreak story<br>to tell you and this is really to try to<br>work out who gave what to whom now we<br>can see you can see how things are<br>associated with outbreaks but we want to<br>know who gave what tuned because<br>actually we need to control the<br>situation so this time it's in it's in a<br>veterinary hospital in Cambridge so this<br>our station came in with a severe skin<br>infection caused by a different pathogen<br>Pasteurella<br>on day five the animal got worse and mrs<br>a was isolated from their wound at that<br>time they died on day 11 and a<br>post-mortem was done and they swapped<br>the animal from numerous sources in<br>their body and started swabbing of staff<br>and other animals so here's the here's<br>the veterinary hospital in fact they<br>swap 97 staff and 158 animals over 71<br>days now 7 staff were positive and 3<br>more animals became positive and what<br>they did this time was instead of just<br>picking one colony off the agar plate 2<br>secrets they picked 20 now this reveals<br>a level of complexity<br>that clinicians and the audience might<br>feel quite uncomfortable about because<br>it means that sequencing and clinical<br>practice is not straightforward what<br>you'll see here is that you have along<br>here each row is a member of staff and<br>you have your index dog and another<br>three dogs were found to be positive<br>over the swabbing period now here you've<br>got a circle to show the number of<br>colonies that are in this particular<br>lineage the colors relate to the clade<br>so actually three of them belong to this<br>some liquid sequence type 22 the same<br>clone but they were actually different<br>clades so the color coded in red blue<br>and light blue I think that some of the<br>things that really surprised us or one<br>of the things is that people don't carry<br>one strain they carry multiple strains<br>or so if we just picked a single colony<br>from safe staff a if we picked a blue we<br>think they had carried the balloon image<br>but in fact they carried two lineages<br>the same was true for another member of<br>staff here and the dog so we also have<br>to worry about how many strains people<br>are carrying now I won't go through this<br>in any detail but from that data we were<br>able to infer the transmission pathways<br>from the sequence amongst those animals<br>and you can see the directory is<br>incredibly complicated so for example<br>staff member F gave their mmrsa to staff<br>member a to index the index dog and<br>another staff member<br>probably to another another member of<br>staff G so these patterns it's not a<br>straight forward to infection control<br>isn't just a straight forward pattern of<br>A to B so finally how can we use this on<br>a national scale what we decided to do<br>was look at the mrs a population in the<br>UK over a decade and we took this number<br>of i stat 780 isolates they were from<br>all over all over the country actually<br>highly representative sampling from<br>across the country and it was this<br>single lineage which we call mrs a-15<br>this is the thing that we get in our<br>hospitals and in the UK now sequencing<br>alone gives you this hairy comet it's<br>got most of the isolates up here with<br>this tail here which is absolutely<br>meaningless<br>no idea what I could do with that so<br>somebody one of my colleagues came up<br>with a very neat idea that we actually<br>started setting up the country into what<br>he called referral networks what he<br>means by that is if you take a single<br>referral network say that one not<br>important to do there 80% of all<br>patients go to hospitals within that<br>referral network and a small proportion<br>go to the next referral network what we<br>were then able to do was actually color<br>code that her Hairi comment head by the<br>the region so what you can see here<br>running it my pointer to work is that<br>actually there some very clear clusters<br>based on geography what that means is<br>that the country looks like this for mrs<br>a lineages and this forms the basis for<br>surveillance of mmrsa spread between<br>hospitals and actually even within<br>hospitals it forms a really key<br>contextual database and in fact in the<br>seminar I heard earlier today people<br>were talking about the importance of<br>contextual genomic databases so this is<br>what the power of large gar sequencing<br>can do now this is a tree of all of<br>those isolates that I've just shown you<br>from the UK but what I've done is I've<br>put isolates in from both of those<br>outbreaks that I just mentioned to you<br>and so you can see immediately how<br>useful this contextual outbreak is it<br>wouldn't take a lot to say there's an<br>outbreak going on there and that's why<br>these databases are so incredibly useful<br>so I'd like to conclude by saying that<br>genomics is going to bring a massive<br>revolution to the way that we understand<br>their control antibiotic resistance<br>there are other things out there that we<br>need to worry about for example this is<br>the proportion of companion resistant<br>Klebsiella pneumoniae islets of<br>gram-negative pathogen carrying<br>something that makes the organism almost<br>untreatable actually and the red is the<br>danger areas if you like coming up in<br>Europe I'm sure that you will have you<br>may want have masks what hopefully you<br>won't have maps like this but in Europe<br>we have major problems and so we have to<br>deal with this<br>genomics can help us go forward so hope<br>for the future well I think that<br>genomics will be important in infection<br>control and in surveillance I haven't<br>talked about rapid Diagnostics<br>fundamental science and so on but I do<br>think that genomics will have an<br>important part to play in the control of<br>antibiotic resistance in the future so<br>I'd like to thank you very much for<br>listening and to thank my fantastic<br>research group we're celebrating a<br>Christmas party and also my funders at<br>the bottom I'd welcome any questions<br>Thank You Sharon for a wonderful talk<br>I'm I'm Jose offered to take some<br>yes<br>yeah<br>I think that's actually important<br>perhaps it's such an important point<br>that you raised I think that there's two<br>ways we can tackle it you're absolutely<br>right and I mean what what what this<br>refers to is selection biases so if you<br>if you cherry-pick the ice lips that you<br>really that interested in secrecy you<br>won't get a true representation and I<br>think there's two ways that we can deal<br>with this one is that there are big<br>initiatives going on to get<br>representative sampling without<br>selection bias for all of the major back<br>to your pathogens and that's got to be<br>shared open access I haven't mentioned<br>open access but it's one of my hobby<br>horses we have to have open access so in<br>a project that we're doing at the moment<br>we're sequencing 10,000 genomes about<br>your genomes but that isn't true you're<br>compared to the massive efforts going on<br>I mean hundreds of thousands so we will<br>get good representation the other thing<br>that we can do is work at the other end<br>and say if we can capture all of the<br>data that's generated by reference<br>laboratories but by regional hospitals<br>that are doing sequencing the real trick<br>these days I think is to collect that<br>data in a cloud-based or other base and<br>actually start to make sense of it in<br>real time and so in the end the<br>selection bias will wash out there's<br>always going to be selection bias if you<br>sequence for clinical reasons I think<br>that these these will form the baseline<br>but over time we will actually move away<br>from you know a reference layer pieces<br>just send us something if it's got X<br>like carbapenem resistance then because<br>regional hospitals will sequence for<br>different reasons to reference labs but<br>you're absolutely right selection biases<br>is a problem that I think that is<br>recognized actually as a real issue<br>because yeah you have to have unselected<br>collections but thank you for that<br>question<br>James<br>yes<br>yeah<br>I think within a couple of years it's<br>going to become very very easy to put<br>some bacterial species into context and<br>so I know several groups who are<br>developing automated interpretation<br>tours and so you'll need to run your<br>sequencer get your sequence data and<br>then put your sequence data into the<br>tool and that will automate confirmation<br>of the species confirmation of the<br>resistance pattern and then give you<br>some basic data on on genotype and will<br>then actually draw a tree to the most<br>related isolates to the ones you've got<br>now if you're in a hospital you'll need<br>to build up your own context to make<br>that worthwhile but I think that that's<br>probably one or two years away it's not<br>available at the moment I do know people<br>who are developing it and that's the<br>absolute bottleneck in my in my view<br>because if you sequence in the context<br>of a hospital you haven't got to worry<br>about ethics you work within a firewall<br>of treating patients it just becomes<br>another diagnostic test people worry<br>about shooting up into the cloud you<br>know data but you do it within your own<br>Hospital so I think the single<br>bottleneck is informatics interpretation<br>but it will come and what people are<br>working on is that the report looks very<br>much like a report you would see as a as<br>a working clinician it's been derived in<br>a different way but you get the added<br>value of this strain is highly related<br>to a strain in the patient in the bed<br>sort of you know highly related to<br>another strain and you can then go along<br>as an infection control doctor and make<br>the connection the other thing I would<br>say and it's not work that I've shown<br>here is that our conventional view of<br>infection control has to change because<br>with the ventricle high for example<br>we're seeing outbreaks across<br>specialities across wards which would<br>never see necessarily within a standard<br>epidemiology you know she would never<br>epidemiology but I've got off the point<br>a little bit but you we will see<br>interpretation software coming along<br>and I'm very hopeful that it'll be open<br>access because we don't want to be<br>locked in to somebody who charges a lot<br>of money for him what should you be<br>sequencing<br>I think that the you've got to have<br>somebody who can interpret things<br>obviously I think that I think that the<br>answer is different for reference labs<br>versus clinical laboratories I mean I<br>saw a fantastic talk this morning by<br>David Williamson and they were they were<br>sequencing things related to outbreaks<br>which i think is absolutely right in<br>hospitals if you haven't got the<br>informatics tools to do the<br>interpretation it's difficult to start<br>now but what you have I think you'll be<br>looking for the same sorts of things<br>you'll be looking to spot transmission<br>events so infection control at the key<br>out breaking visits or outbreak<br>investigations in the community but also<br>ultimately predicting drug resistance<br>but the organism has to be a rapper a<br>slow grower compared to phenotypic tests<br>which are still cheapest chips and easy<br>to do so TT B's exception to that<br>by the way hood that we could have a<br>longer conversation and I suspect we<br>will about people practice any other<br>it's<br>I absolutely agree I mean I heard your<br>talk this morning which I like very much<br>and you know I like the idea of you know<br>the news New Zealand being sort of you<br>know the green country where you can<br>actually have very low rates of<br>resistance and if you want to control<br>what resistance you have got I think<br>that working on that to look at isolates<br>the effect we've just done something<br>like this in the UK where we took I<br>slips from we've done eat coli and<br>enterococci from farms and from<br>wastewater actually and from humans in<br>in hospitals and that would actually<br>show you because I mean there's two<br>things the carrier lineages are they're<br>the same across the species and the<br>resistance plasmids are they the same<br>and that would prove definitively<br>whether you know one practice needs to<br>change but I think there's a tendency to<br>say it's all the farmers fault that all<br>to blame I actually don't think that's<br>necessarily true and we've just finished<br>a study in in in Cambridge at East of<br>England where the lineages of E coli in<br>farm animals and and humors completely<br>different actually there's no overlap so<br>I think that we should we should we<br>shouldn't be like sheep is it worth<br>following on with very ones else's view<br>saying it must be the farmers we have to<br>get the evidence and this is the<br>opportunity to do it so I think that New<br>Zealand will be a very good place to do<br>I can just I can just see the grant<br>application now actually I think it<br>would make a very good reading oh it<br>would bring us smart on my face so<br>perhaps if you're interested we could<br>talk about that later but I think that's<br>a very good point and important for New<br>Zealand's actually thank you very much<br>I just like to thank those of you who've<br>come as today to to hear our Asian<br>especially those from the public and<br>from a wide range of parts of the public<br>and I'd like to thank again sheriff we<br>were talking we just like to give you<br>something uniquely from New Zealand and<br>I'll ask prof. Blakey to to give that to<br>you</p></main><footer style="margin-top: 2rem; background: #0001; padding: 2rem; text-align: center;"><p>We Are The University</p><ul style="list-style-type: none; padding: 0; margin: 0;"><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li></ul></footer></body></html>