<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>We Are The University</title><link rel="stylesheet" href="/styles.css"></head><body><header><h1 style="color: #fff;font-family: 'Arial Black', Gadget, sans-serif;font-style: italic;font-weight: 900;text-transform: uppercase;">We Are The University    </h1><nav><ul><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li><li><a href="/blog">Blog</a></li><li><a href="/videos">Videos</a></li><li><a href="/authors">Authors</a></li></ul></nav></header><main><h2 style="text-align: center;">Conservation time machine: Harnessing degraded DNA for wildlife protection - ERI Seminar - August 23 [52:56]</h2><p style="text-align: center;"><a href="https://www.youtube.com/watch?v=ubU3O1GtWS4" target="_blank">Watch on Youtube</a></p><p style="text-align: center;"><a href="https://www.youtube.com/channel/UCYzot7AKCB5paA4vp8wXuvw" target="_blank">The University of Waikato</a></p><img src="https://i.ytimg.com/vi/ubU3O1GtWS4/maxresdefault.jpg" alt="Thumbnail for video titled: Conservation time machine: Harnessing degraded DNA for wildlife protection - ERI Seminar - August 23" style="width: 100%;"><div class="tags"><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#eDNA</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#genomics</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#historic biodiversity</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#wildlife conservation</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#museum science</span><span style="background: #0003; border-radius: 0.3em; padding: 0.3em; display: inline-block; margin: 0.2em; font-size: 0.8em">#CSIRO</span></div><h2>Description</h2><p>Dr Erin Hahn<br><br>In my talk, I will share several cutting-edge research tools I have developed which utilise degraded sources of DNA for non-invasive monitoring of wild populations across time and space. <br><br>I will present reliable methods of whole genome sequencing from formalin-preserved museum specimens as well as metabarcoding from specimen media. By combining eDNA methodology with innovative DNA extraction techniques, we can non-destructively recover genetic data from precious specimens. This approach enables us to characterize historical ecosystems, uncover species interactions, and explore the dynamics of past environments. Going beyond the genome to the epigenome, I will give an overview of Archival Micrococcal Nuclease Sequencing (aMNase-Seq), which leverages historical formalin preservation to estimate historical gene activity and track temporal gene expression trends. By pushing the boundaries of museum-based genomics, we gain valuable insights into historical biodiversity, expanding our understanding of species dynamics over time.<br><br>I will also discuss recent advancements in airborne eDNA technology and highlight opportunities to adapt successful aquatic methods for terrestrial eDNA analysis. By acknowledging technical limitations and harnessing the power of terrestrial eDNA, we can efficiently generate informative species data, enabling effective non-invasive monitoring and management of terrestrial ecosystems.<br><br>Together, these new tools improve understanding of species responses to environmental changes, facilitate predictions of species resilience, and enhance conservation strategies for biodiversity management.<br><br>Dr Erin Hahn is a Research Scientist at the CSIROâ€™s Australian National Wildlife Collection leading projects aimed at developing reliable platforms for airborne eDNA monitoring. She recently completed a postdoc at CSIRO focused on extracting genomic and epigenomic data from century-old formalin- preserved museum specimens. Prior to joining CSIRO she completed a PhD at the University of Arizona where she received a National Science Foundation Interdisciplinary Graduate Research Traineeship to support her study of temporal population genetics and epigenetics of endangered Sonoran pronghorn. Erin is a passionate science communicator and Science and Technology Australia Superstar of STEM alumnus.</p><h2>Transcript</h2><p style="opacity: 0.9; font-size: 0.8em">Transcripts may be automatically generated and may not be 100% accurate.</p><p>uh I think maybe I can now I'm recording<br>now so whatever I did works okay<br>apologies for that slight<br>um issue with everybody<br>um all right uh amigiana<br>or waikato<br>um<br>hello everybody and welcome to today's<br>um seminar uh I am going to start off<br>just by introducing our speaker today so<br>Dr Erin Han is a research scientist at<br>Cyrus csiro in Australia where she's<br>based at the Australian National<br>Wildlife collection<br>um prior to starting as the research<br>scientist there she was a postdoc at<br>Saro as well uh and then a little bit<br>prior to that she was also based at<br>Australian National University across<br>the road which is where we met and<br>um have known each other for a few years<br>since okay I think I'm just joining<br>um<br>again because I pressed connecting to<br>zoom too many times anyway<br>um after prior to starting uh coming to<br>Australia um Aaron completed a PhD at<br>the University of Arizona<br>uh and she sort of has a background in<br>looking at uh temporal population<br>genetics and epigenetics throughout this<br>time but in addition to that errands are<br>really excellent science Communicator as<br>we're going to hear today and um she's<br>also one of the Australia superstars of<br>stem alumni so I'll let you<br>um take it away from here uh yeah and<br>we'll look forward to hearing your your<br>talk today<br>how are we going how's that yep we can<br>okay<br>there we go because I can see everyone's<br>faces instead of my slides and that's no<br>good I know what my slime look like so<br>long as they're not blocking for you<br>guys that's probably okay<br>um so hello everyone uh thank you so<br>much for giving me the opportunity to<br>talk to you today I was really hoping to<br>be there in person but life sort of got<br>in the way so I'm glad to be able to at<br>least be giving this talk uh virtually<br>today I'm going to give you a bit of a<br>whirlwind tour of the sort of work that<br>I do using degraded DNA much of it<br>extracted from historical specimens to<br>gather as much information as I can<br>about our changing Planet but before I<br>dig in I'd like to say that I'm going to<br>be talking a bit about old Museum<br>specimens for a good portion of this<br>talk and I want to emphasize that old is<br>relative tens of thousands of years<br>before the first csiro scientists began<br>their work here where I'm sitting in the<br>shadow of Black Mountain<br>Australia's first scientist called This<br>Place by a different name gallenberry<br>and while the data that I'm going to be<br>presenting today is old by genetic<br>standards it barely scratches the<br>surface of the knowledge that First<br>Nations people have about our planet its<br>environment and the cosmos<br>so I'd like to acknowledge that I'm<br>joining you today from unseated<br>territory traditionally earned by the<br>nunawa and Embry people and I'd like to<br>acknowledge their enduring connection to<br>land water and Community upon which I<br>live work and play and pay my respects<br>to their Elders past present emerging<br>and also extend that respect to the<br>waikato tanui and any First Nations<br>people that we have joining us today<br>as and mentioned I'm at the csiro and I<br>work in the Australian the national<br>research collections of Australia and we<br>house over 15 million specimens<br>collected over the last 150 years or so<br>spanning vertebrates insects plant fungi<br>and I would like to acknowledge also<br>that many of these specimens were<br>collected without consultation with<br>traditional owners and in ways that<br>disenfranchised and exploited those<br>communities this is something that our<br>collections are in the process of<br>acknowledging more openly and beginning<br>to try to reconcile and so I encourage<br>you to reach out if you have any<br>questions regarding that very critical<br>work<br>and while my work spends many different<br>taxa the work that I primarily conduct<br>is in the Australian National Wildlife<br>collection where we primarily focus on<br>reptiles amphibians birds and mammals<br>and we preserve them in a whole bunch of<br>different ways both dried as puppets or<br>skins as well as we have a big egg<br>collection which many of it was donated<br>to us by people who were conducting egg<br>collections illegally we also have a<br>bone and skull Vault and the place where<br>I spend a lot of my time is in the<br>spirit vault which is housing specimens<br>that were preserved with formaldehyde<br>and them popped into ethanol for<br>I'm a geneticist and so when I work in<br>the collections the sort of data that<br>I'm focused on is their DNA and the<br>field of museum curation has been having<br>a big boom in genetic research and more<br>and more we're able to use these<br>specimens to not just gain little<br>Snippets of DNA but we're also being<br>able to reconstruct whole genomes from a<br>whole variety of Museum specimens<br>preserved in different ways<br>but our whole wing of the collection has<br>been largely inaccessible to genetic<br>research and that's those specimens<br>preserved in formalin the information if<br>we were able to get any of it whatsoever<br>out of these specimens was highly<br>fragmented and so most museums<br>considered those collections completely<br>off limits to genetic work and wouldn't<br>allow sampling<br>and this is fairly unfortunate for<br>researchers who want to get historical<br>genetic information from taxa like<br>reptiles amphibians and fish because<br>it's these specimens that are most<br>frequently preserved in our wet<br>collections so they were from almost<br>completely lacking in historical genetic<br>information<br>at the beginning of my postdoc here at<br>Cyro almost four or five years ago now I<br>set out to kind of break open the vaults<br>and give access to these sorts of<br>specimens so that we can make the most<br>of the specimens that we have in<br>informing about historical change<br>the methods that I developed for<br>extracting DNA from hormone preserved<br>specimens start at designing a system<br>for kind of predicting the suitability<br>of the specimen 4 sequencing so that we<br>can limit how much destructive sampling<br>we do of specimens unnecessarily because<br>we already have a good idea that they're<br>not going to produce usable DNA<br>the I went then went and tested a bunch<br>of different methods uh that had been<br>previously used both in museum specimens<br>as well as formal and fix and paraffin<br>embedded histological samples and came<br>up with a method that works really<br>really well on specimens that are of<br>moderate or good uh preservation quality<br>even if they've been exposed to form one<br>and it involves a very high-tech method<br>of popping it into a ph-13 buffer<br>putting it in the autoclave for about 15<br>minutes and then doing an old school<br>phenyl chloroform extraction we then do<br>very very heavy sequencing followed by a<br>trimming free alignment which maximizes<br>the amount of endogenous DNA that we are<br>mapping to our reference genome<br>and using this method I've been able to<br>recover DNA that will give genomic<br>coverage of specimens upwards of 25x<br>which is starting to get to the point of<br>the sort of cover that you would get<br>from a fresh sample so no longer is<br>formalin a barrier to whole genome<br>sequencing of historical specimens I've<br>applied this method to specimens as old<br>as 1905 and with the Eastern water<br>dragon got up about around 11x cover and<br>with some teeny teeny tiny tuna larvae<br>which were so small that you could pick<br>them up in a pipette tip I was able to<br>get 25x cover from<br>about 45 or so individuals and this<br>method has now been published since<br>about last year and we've had a lot of<br>uptake from it from curators and<br>researchers around the world using this<br>method so that we can have more reliable<br>sequencing of these kind of specimens<br>and if you're interested I've popped up<br>the QR code to that paper for easy<br>access<br>included in that paper is like I said a<br>decision-making system of determining<br>whether or not a given specimen is going<br>to be suitable for sequencing and this<br>is something that can be used by both<br>curators and researchers in order to<br>determine whether or not it's really<br>worth it to us sample that individual<br>and conduct sequencing and it could be<br>done without disturbing the specimen at<br>all essentially just look at it and you<br>do what we call the ick test if the guts<br>are discernible and you can tell the<br>organs from one another there's a pretty<br>good chance that it's going to be<br>sequenceable and then we just do a test<br>of the specimen media make sure that it<br>doesn't have really really low PH or<br>extremely high formaldehyde which might<br>indicate that it was preserved using<br>methods that weren't particularly ideal<br>for sequencing<br>and then kind of depending on what how<br>much DNA you get out of the specimen<br>will determine what sort of success rate<br>you would have given your particular<br>genomic output so if you're trying to do<br>your whole genome sequencing you'd want<br>to limit to those specimens that you get<br>the most DNA out of but pretty much<br>anything that yield s more than like 20<br>mg of DNA we've had<br>trying to amplify very small fragments<br>so you can do some kind of specimen ID<br>at a minimum with most samples<br>so I'm going on from sequencing<br>individual specimens I also had at the<br>same time a small kind of side project<br>which I affected affectionately called<br>bucket seek which you'll see for reasons<br>in just a moment<br>I know that I can say I can sequence the<br>specimens in the jar themselves and this<br>got me thinking what else is floating<br>around in this jar in the specimen media<br>itself because if I can use the tissue<br>to sequence the individuals then perhaps<br>also DNA shed by those individuals or by<br>their Associated parasites and microbes<br>I would also be able to pick that up in<br>the specimen media and sequence that and<br>get an idea of the whole ecosystem of<br>organisms that are preserved in that jar<br>so I went digging in the collections<br>literally in buckets in our prep Lab in<br>order to test whether or not we can use<br>specimen media itself for sequencing<br>and the workflow that I developed starts<br>with those buckets of specimen media and<br>we apply every dna-like approach where I<br>filter off the specimen media and then<br>put that filter into the extraction<br>buffer the same kind of extraction<br>buffer that I use for my tissue<br>extraction so that really high ph and<br>pop it into the autoclave and so that<br>contends with any potential formaldehyde<br>preservation that the media has been<br>through and then I apply a metabarcoding<br>approach and so in this particular essay<br>I was looking at the 12s cytochrome and<br>cytochrome B loci as well as a 16sr RNA<br>Locus which was designed to Target some<br>flies that we thought were potentially<br>associated with the specimens in our<br>collection<br>I applied the method to a bucket where I<br>knew what media I put in it I went<br>around the collection kind of went<br>shopping for media poured it into a<br>bucket and then I also applied it to<br>those buckets in the in the collection<br>in the prep lab where I have a good idea<br>of what specimens are in our collection<br>but I wasn't<br>exactly sure which of the jars were put<br>into the waste bucket so this one<br>um I would call my unknown<br>just to give you some preliminary<br>results because I'm still sifting<br>through these data in the bucket where I<br>knew which species I put into the into<br>the jar I was able to recover sequence<br>for uh three species two species as well<br>as um assigning uh OTS to uh to Genera<br>that I was I knew I put in the bucket so<br>this is just looking at the very very<br>top hits from an analysis program called<br>Kraken and I know that there's lots and<br>lots of more data in there and now it's<br>just up to me to decide where those<br>particular cutoff thresholds are for<br>believing how much of the sequence reads<br>were actually from these species<br>and even more exciting is that unknown<br>bucket where I wasn't sure what species<br>went into it but in just sifting through<br>the top 15 hits on Kraken I was able to<br>find a whole bunch of species that were<br>very believably in that bucket both<br>because we have them in our collection<br>and we were actively curating them at<br>the time so we have lots of different<br>back species that were being processed<br>as well as Central bearded dragon and<br>two species that came up in this<br>analysis that were really cool were the<br>powerful Owl and the kagyu because these<br>are specimens that we don't have in<br>spirit we don't have them in wet<br>collections but at the time that I was<br>doing the collection of the specimen<br>media they were processing some of these<br>species in the prep lab to make us skins<br>and so the DNA that I picked up in the<br>waste bucket is very likely DNA that was<br>floating around in the air landed in the<br>waste bucket was fixed in formaldehyde<br>because it contained formaldehyde in the<br>bucket and then I was was able to<br>recover that and so this is evidence of<br>Airborne Edna floating around in the<br>prep lab that I was able to pick up<br>so that adds another layer of complexity<br>knowing that the DNA in the bucket isn't<br>necessarily DNA that came from specimens<br>in the bucket<br>um but I think that's valuable<br>information to have and also kind of<br>shows the um you the usability of using<br>a formal and based media in order to<br>preserve DNA collected in the air<br>oh and over here you guys are covering<br>it uh but we also picked up<br>um some sequence for a blowfi and this<br>is one that is associated with necrotic<br>tissue so this was likely larvae that<br>were either laid in some of these<br>individuals after they were<br>um before they went into the collection<br>so sometimes we get roadkill and that<br>might have been the way that that made<br>it into our sample or potentially<br>um went some way through the the<br>processing of these specimens<br>so this all has me optimistically<br>looking into the Vault for additional<br>ways of using the DNA that's preserved<br>in these jars because I think that this<br>would be a really cool way to either<br>recover whole mitochondrial sequences<br>from these specimens without having to<br>do any destructive sampling which would<br>be really useful if we are sequencing<br>Hollow types of individuals these would<br>be specimens that are preserved<br>singularly by themselves in a jar to<br>serve as the type specimen for a given<br>species and we don't want a destructive<br>destructive sample these ones because<br>it's the only one that we have and we<br>use it as a reference but if we can use<br>the gunk in the bottom of the jar to<br>collect genetic information from it then<br>that would be an invaluable resource for<br>people who are doing lots of other<br>comparative genomics and also e-dna<br>studies around the world so this is<br>something that I'm actively developing<br>both in our collection as well as<br>looking at in collaboration with the<br>fish collection and Hobart<br>so I've now been able to provide a<br>method that both reliably sequences<br>specimens preserved in formalin starting<br>to get evidence that we can get genetic<br>information from the specimen media<br>itself and now I wanted to take those<br>specimens preserved in formalin a step<br>further and quite literally flip it on<br>its head<br>so back to the formal and problem we<br>have all of these reptiles amphibians<br>and fish preserved in formaldehyde which<br>yes is annoying if you study these<br>species but it's particularly vexing<br>because it's these species that at least<br>amongst vertebrates are some of the most<br>biodiverse and environmentally sensitive<br>taxa and if you think about it each and<br>every single one of these is a little<br>record of how that species was coping or<br>perhaps not with the environmental<br>challenges of its day and it's only<br>these specimens in our collections that<br>are preserved whole everything else<br>largely has their internal organs<br>removed and are either preserved as a<br>skin or bone and so if we're looking at<br>organismal response the sort of<br>information that bone is going to tell<br>you about say gene expression isn't as<br>informative of how a species is perhaps<br>responding to a novel pathogen that<br>attacks their lungs if they don't have<br>their lungs so these specimens are all<br>preserved with their individual organs<br>and if there were a way to go in and<br>extract the gene expression information<br>in a tissue specific way from these from<br>these specimens then we would be opening<br>up a whole door to looking at how<br>species respond in a very<br>individualistic way<br>and the way that we might go about this<br>is by looking at the epigenome and<br>there's a whole bunch of different marks<br>that we use on top of the DNA that we<br>call epigenomic markers either DNA<br>methylation which is one of the most<br>commonly used also histone modifications<br>to different chemical modifications add<br>to the packaging material and the genome<br>also aware of those nucleosomes are<br>sitting along the genome can tell you<br>information about the activity of a<br>given Gene as well as potentially RNA<br>and individual proteins in the cells so<br>if we're able to access this information<br>then that gives us information about<br>what genome is actually doing at the<br>time in the context of the environment<br>that that specimen was collected<br>this would be an amazing approach to<br>apply in collections because we have<br>millions of specimens collected all<br>around the world and most of that<br>activity was occurring in coinciding<br>with a period of Rapid environmental<br>change over the last 150 years or so in<br>response to things like climate change<br>and introduction of Novel pests and<br>pathogens so potentially sitting at our<br>fingertips is over a Century's worth of<br>environmental response data if only we<br>could get to it<br>so back to form one formlin was thought<br>to damage DNA because it goes in it<br>cross-links all sorts of proteins and<br>various cellular components to the DNA<br>and then when you try to get it out it<br>fragments it beyond belief and you can't<br>extract it well my work has shown that<br>you can get the DNA out it is still very<br>highly fragmented but it's still there<br>it's preserved and it's largely the<br>extraction method that is what disrupts<br>all the DNA<br>formalin isn't seen as a complete<br>barrier to a molecular analyzes in other<br>fields of biology in chromatin biology<br>research and immunological assays we use<br>formalin in order to lock in place all<br>of the cellular Machinery in in the cell<br>in order to learn information about what<br>it's doing so approaches like chip seek<br>high C Fair sake and M A seek all use<br>formalin in order to gain information so<br>clearly it's not a complete blocker to<br>molecular analyzes<br>and the subject of my postdoc was to try<br>to adapt some of these methods to<br>applications and Museum specimens to see<br>if we can take advantage of historical<br>formal and preservation to gain<br>historical gene expression and<br>epigenetic information<br>so we've tried a bunch of different<br>assays and the one that we are just<br>ready to start talking about is our<br>adaptation of micrococcal nuclease<br>sequencing to museum specimens and what<br>micrococcal nuclease does is it's an<br>enzyme that goes in and it chops on<br>either side of the nucleosome and then<br>eats away at the intervening DNA in<br>between them and so you get an idea of<br>where individual nucleosomes are sitting<br>in the genome<br>to develop this method I didn't want to<br>go and start using precious Museum<br>samples so I put together a mock Museum<br>collection which consisted of both<br>inbred lab mice as well as some wild<br>caught Australian mice we processed the<br>samples the same way we do as anything<br>else that enters the collection we took<br>a fresh liver tissue sample popped it in<br>the minus 80 to do Downstream rna-seq as<br>well as a micrococcal nuclease treatment<br>of fresh tissue we also put the<br>remaining carcass into formal and fixed<br>it for three days and then washed it and<br>put it into ethanol and by the time that<br>we got around to processing these<br>samples for sequencing they'd been<br>sitting in the collection for between<br>two and five years so these were<br>properly archival and much much heavier<br>fixed than say something you would see<br>in a histological lab which might be<br>fixed for 15 minutes so these are really<br>properly heavily fixed<br>this is quite complicated here it's many<br>many steps but<br>um this is the method that we ultimately<br>landed on that was re allowing us to<br>recover historical information from<br>these kind of specimens we've just put<br>this up on bioarchive and again there's<br>a QR card if you're interested in<br>reading the nitty-gritty but the bulk of<br>it is just processing the tissue<br>appropriately so that we can get to the<br>chromatin and allow the micrococcal<br>nuclease to do it work do its work<br>that's what all the little steps in<br>there do and then we just do really<br>really heavy sequencing of the resulting<br>DNA fragments and map that and then do<br>the same kind of trimming free alignment<br>that we did with our whole genome<br>sequencing approach and then we apply a<br>enrichment analysis program called Dan<br>Post which works for both enrichment as<br>well as depletion signals within those<br>alignments<br>and I'll just give you a taste of what<br>the signal looks like after applying<br>this archival micrococcal nucleus assay<br>so these are both I'm just whole genome<br>sequencing between the archival sequence<br>data on the bottom and that on the top<br>and you can see that the cover is pretty<br>similar between fresh and archival<br>tissues which in and of itself is pretty<br>exciting<br>um but once we apply the micrococcal<br>nuclease treatment and start eating away<br>that intervening DNA we can start to see<br>that differences emerge in the coverage<br>patterns between the fresh and archival<br>tissue<br>and when we apply that analysis of Dan<br>Post looking at the difference between<br>the micrococcal nuclease treatment and a<br>input control we start to see that those<br>differences between the fresh and the<br>archival samples become even more<br>exacerbated<br>in the fresh samples what we typically<br>see is a positive signal so enrichment<br>that is associated where the nucleosomes<br>would be sitting in that particular<br>stretch of DNA so we see a signal of<br>positive or Peak gain<br>on the flip side in archival samples<br>rather than seeing a signal of peak gain<br>we see a signal of peak loss<br>and this is although mirrored it allows<br>us to look at differences in sequence<br>cover that we can associate with gene<br>expression<br>to then so that was looking at one gene<br>looking across the entire genome on the<br>left hand side this is looking where um<br>Peak gains has been to have been<br>detected as the transcription start site<br>lining up all the genes across the<br>genome there when you see a signal of<br>enrichment in response to micrococcal<br>nuclease treatment you see this kind of<br>tornado effect so in Peak gains we see<br>that tornado effect in the fresh tissues<br>but not in the archival tissues however<br>when we look at Peak losses across the<br>genome we see we don't see the tornado<br>in fresh signals but we do see it<br>consistently in archival signals so the<br>signal that we're getting from<br>micrococcal nuclease treatment of<br>archival tissues is the mirror image of<br>fresh tissues and the signal isn't<br>completely lost because of heavy formal<br>and fixation it's just inverted we had<br>to look at it differently<br>so that paper is going to be coming up<br>pretty soon which opens up this whole<br>idea of using historical gene expression<br>information to start tracking<br>environmental change and that's really<br>really exciting but that was on mice and<br>what we want to do is actually look at<br>some species that might be of<br>conservation concern or are otherwise<br>relevant to our local environments so in<br>addition to putting it to work to<br>testing it in Mouse color in our Mouse<br>samples I also wanted to apply it to<br>some actual archival samples<br>and for this component I teamed up with<br>some researchers at the University of<br>Queensland Celine Freire and Nicola<br>Jackson<br>Celine and Nicola they study Eastern<br>water dragons and they are looking at<br>the response the Adaptive response of<br>this species to urbanization in the<br>Brisbane area<br>and they've collected a whole heap of<br>modern data including behavioral<br>morphological genetic differentiation<br>data as well as the prevalence of a<br>fungal pathogen that is being pretty<br>devastating for this species and their<br>system is pretty cool because they have<br>this whole gradient of urbanization<br>across the Brisbane area which includes<br>populations that have been in urbanized<br>areas for longer or relatively recently<br>so they can look at the adaptation in<br>real time for these water dragons but to<br>look at how those populations have<br>changed through time they needed to get<br>access to historical information<br>and for that they came to us and and the<br>first instance we sent Nicola into the<br>Queensland Museum to find some specimens<br>to do some testing on she took our<br>specimen vetting system with her and<br>applied it to a bunch of jars in the<br>collection and found some that we<br>thought would work really well for<br>sequencing and these included five<br>specimens four males one female and the<br>oldest of which was collected in 1905.<br>we then put that through our both our<br>archival genomic sequencing as well as<br>archival myocardial nuclease assays to<br>look for both genetic and gene<br>and these are those sort of tunnel pots<br>there's funnel pots that I was showing<br>before and what we're looking at is the<br>five individuals and whether or not we<br>saw enrichment at the transcription<br>start sites in response to our<br>micrococcalonuclease treatment<br>and the first thing that we see is that<br>there is a albeit weaker but consistent<br>signal of expression across samples<br>collected over the last hundred years<br>and what's really cool is that the<br>oldest sample is also showing a signal<br>so there doesn't seem to be an effective<br>age of whether or not the assay Works<br>which is consistent with our results<br>with genome sequencing in that there<br>doesn't seem to be any kind of effective<br>age so long as it was preserved well in<br>the first instance it's kind of just<br>sits there waiting for you until you're<br>ready<br>and the second thing that we notice is<br>that in combining both the genomic<br>sequencing and the micrococcal nuclease<br>data we get really good cover of the<br>genome uh upwards of 14x in some of the<br>more recent samples and this is enough<br>to call Snips across an entire genome<br>for uh historical sampling so this<br>allows you to do phylogenetic analysis<br>in addition to gene expression<br>information analyzes for specimens up to<br>so that's all well and good but what can<br>we actually do with this can we pull out<br>biologically relevant information from<br>these kind of specimens now we do only<br>have five samples here so we do have to<br>you know kind of take this with a grain<br>of salt but what I did was take the<br>micrococcal nuclei sequence patterns<br>across the entire genome and just look<br>for pairwise differences between<br>individuals using a piercing correlation<br>analysis popped them into a PCA and the<br>first thing we notice is that all of the<br>samples collected<br>um sorry all of the male samples<br>collected in the city kind of cluster<br>down in the bottom wait hold on have<br>arrows here there we go okay so we have<br>a cluster of males collected in the city<br>which are all together despite over a<br>hundred year difference between them<br>we have one in individual female that's<br>up in our own little world and then we<br>have another male that was collected in<br>when we then look at PC2 and PC3 we see<br>that the influence of sex drops away so<br>that one female who is off in her own<br>little world then clusters more tightly<br>with the male that was collected also in<br>natural habitat and so we see that in<br>addition to there being an influence of<br>sex on epigenetic uh signature we also<br>see a signature of habitat<br>differentiation and these are both<br>things that you would expect and it<br>which is why it's really really<br>important to do sex matching if you're<br>doing to do any epigenetic analyzes so<br>while this doesn't give us any direct<br>insight into this particular species it<br>does give us guidance on how to conduct<br>future sampling and it does give us an<br>inclination that the information that<br>we're getting using this assay does<br>reflect what we would expect<br>The Next Step that we are engaging in is<br>to create a fundamental data set that<br>will enable other researchers to do<br>smaller scale epigenetic research and<br>then put that into the context of what<br>they would expect to see across the<br>genome given our much better<br>understanding of how the genome responds<br>in a whole bunch of different taxa so<br>we're going to be sequencing a thousand<br>epigenoms collected over 100 years for<br>10 different taxa and this will form a<br>fundamental data set for reptiles<br>mammals as well as amphibians in order<br>to then put that into context and<br>understand how different species are<br>changing Through Time<br>so this is something that we are<br>actively looking for partners to kind of<br>Select our species because we want to<br>make sure that the species that we<br>select are both representative of the<br>kind of taxa that people want to see<br>information for and also that will be<br>immediately usable for someone's<br>research so if you're particularly<br>interested in this hit me up and we can<br>see if we can potentially look at a<br>partnership to include one of your<br>species in this kind of work<br>so the last thing that I'm going to<br>touch on uh is to then get just give an<br>overview of the sort of work that I'm<br>doing now at the csiro all of that work<br>was mostly um to do with my postdoctoral<br>work I am seeing it through but I'm now<br>been<br>um kind of tasked with leading an effort<br>to do an Airborne e-dna study at the<br>csiro and so I'm coordinating a whole<br>bunch of different researchers across<br>different units and we're going to be<br>trying to push this technology forward<br>in order to develop and deliver new<br>capabilities<br>so just uh quick little touch on why e<br>DNA I like to think of Edna a bit like<br>glitter if you have children like I do<br>you know that they come home with<br>artwork and glitter ends up everywhere<br>it's no different for environmental DNA<br>we have been able to use Edna in all<br>sorts of different ecosystems from those<br>that are really really difficult to<br>access to ones where we are very<br>sensitive environments we don't<br>necessarily want to go tromping through<br>them we can use DNA shed by a variety of<br>different organisms to track their<br>movement through different ecotypes and<br>to survey what species are there and so<br>this sort of technology has been really<br>well developed in aquatic systems and<br>it's only more recently that we start<br>we're starting to think more and more<br>about terrestrial Edna but coming from a<br>conservation genetics background we<br>didn't used to call it terrestrial Edna<br>we used to call it non-invasive sampling<br>we do lots of Edna collection from scats<br>from feather from you name it if we<br>wanted to pick up speed traces of<br>individuals we're quite creative in the<br>ways that we collect DNA we just didn't<br>call it e-dna and now that we're<br>starting to you know revisit what we're<br>calling those kind of samples we're<br>being even more creative with the<br>sources that we're getting Edna from and<br>in the last few years there's been a<br>couple studies literally plucking DNA<br>out of the air in order to track a whole<br>bunch of broad taxa<br>that technology I think is really ready<br>to take off but we want to take a moment<br>to think about how we can take advantage<br>of the vast amount of information that<br>has come from using Edna and Aquatic<br>systems before we get too excited<br>um and apply it inappropriately to<br>Airborne Edna<br>so there's lots of advantages to<br>um using Airborne or using Edna in<br>aquatic systems it's non-invasive so you<br>don't have to go out and do electrical<br>fishing to detect fish you it's very<br>sensitive you can just from you know a<br>small amount of water collected you can<br>get lots of different taxa and it's also<br>comprehensive it's something that can be<br>used to simultaneously assay both<br>animals and plants as well as fungi and<br>microbes and to that regard it's very<br>cost effective because you don't need<br>bespoke methods for detecting a whole<br>bunch of different species if you can do<br>it all from one little pot of DNA<br>but we can't talk about the advantages<br>of Edna without also acknowledging its<br>limitations<br>it's largely viewed as not quantitative<br>you don't know how many fish are in the<br>water and the fish that you are able to<br>detect or whatever taxol that you're<br>looking at it's entirely dependent on<br>the reference database that you're using<br>and those reference databases have<br>biases in them both in terms of which<br>species are in them and how um what<br>quality of the DNA sequences are in<br>those databases there's also a bazillion<br>questions around DNA persistence if you<br>pick up a particular species does that<br>mean that it was there this morning or<br>does it mean that it was there 30 years<br>ago that always it isn't always clear<br>and of course the expense of Edna is in<br>both columns because while it can be<br>very cost effective to simultaneously<br>assay a whole bunch of different taxa if<br>you're particularly a group that doesn't<br>do molecular work this can be something<br>that is extremely expensive for you to<br>try to kick off an Edna program<br>in aquatic DNA they're starting to break<br>down the barriers on these different<br>limitations and it's just acknowledging<br>what Edna can do where it can be<br>appropriate and then really focusing on<br>the ways that we can apply it so that<br>it's most useful<br>in that quantitative question I think<br>it's pretty well understood that no it<br>doesn't help you count things but we are<br>seeing more and more papers which are<br>associating the amount of DNA that you<br>have or the number of hits that you get<br>from a different sampling site to<br>estimate at least abundance or biomass<br>in some way and you can have a pretty<br>good sense of being able to tell there's<br>more of that thing than there used to be<br>or there's more we're getting more hits<br>something's changed so while it's not<br>entirely quantitative it is giving us<br>simple information about<br>um population changes<br>the other question is about those<br>reference databases there seem to be a<br>bazillion different reference database<br>projects up and running now and at the<br>international Congress of genetics I<br>heard about a whole bunch of new efforts<br>that are going on all around the world I<br>will just highlight one that we have at<br>the csiro we are building a national<br>biodiversity DNA library and our aim is<br>essentially to go out and sequence one<br>of everything and that sequence is also<br>tied to a voucher specimen in a<br>collection so that you can always go<br>back and taxonomically identify the<br>species from a very well curated<br>database and so that's going to be a<br>step better than using ncbi which is a<br>collection of sequences that kind of<br>anybody wants to put in there and so<br>hopefully in a couple of years that<br>reference database question isn't going<br>to be as such much of an issue<br>around the question of DNA persistence<br>there's a whole bunch of studies coming<br>out around the Ecology of Edna and one<br>that I'll just I thought was pretty cool<br>was looking at the relative abundance of<br>Edna versus Erna<br>um and since RNA is a bit more fragile<br>looking at its stability through an<br>ecosystem in relation to DNA can start<br>to give you information<br>and that can allow you to predict how<br>old that DNA or how long ago it was shed<br>so again we might not be able to say<br>that it was shed 17 hours ago but you<br>might be able to get a good sense of was<br>it should today or was it should what<br>quite a long time ago which can be<br>really helpful for wildlife managers<br>the last question is that expense it is<br>still very expensive for someone to<br>Kickstart an Edna program in their lab<br>um even if they do even if they have<br>molecular experience but we're starting<br>to see service providers like Wilder lab<br>he might have heard of over in New<br>Zealand and this sort of these sorts of<br>service providers are going to be taking<br>it out of the hands of individual labs<br>to do Edna research because they'll send<br>you a kit you do your sampling you send<br>it back and they give you a whole list<br>of species that they found in your area<br>I think this is really where the field<br>is going to take off because now it's<br>not going to be something that each lab<br>has to do individually because they can<br>do things in bulk we're going to see a<br>lot more work done so I'm really excited<br>to see these groups start to take off<br>and that's where I think the next stage<br>is going to be in Airborne Edna because<br>air like water to fluid stuff moves<br>around we can get DNA out of it we just<br>need to make sure that we can do it<br>effectively and reliably<br>and some of the really exciting Studies<br>have come out in the last couple years<br>on Airborne Edna one which was here in<br>current biology last year they put out<br>an air sampler in a zoo and pretty much<br>detected<br>the distance so the further away from<br>something that you are the less likely<br>you are to pick it up as well as sort of<br>the more biomass there is it makes it<br>um you know more likely to pick up that<br>species<br>another really cool study was using air<br>quality monitors to do Edna sampling<br>there's these systems that are already<br>in place going out and telling you all<br>about the pollution in your area and<br>they took the filters from those did any<br>DNA approach and they were to record<br>they were able to recover over 180<br>different species associated with the<br>local environment so this is showing<br>that the text that you're able to pick<br>up Species even ones that you wouldn't<br>necessarily think you could such as<br>amphibians and that this method might be<br>really really useful for assaying<br>biodiversity<br>and we also have these existing<br>infrastructure networks of air quality<br>monitors all across uh different<br>countries so like all around Europe all<br>around North America we have these<br>systems in place that we can utilize for<br>dual purposes<br>but that doesn't really help us if we<br>work in areas which don't have these<br>really extensive networks how do we<br>survey our really diverse ecosystems<br>especially in corners of the world where<br>we don't have large Swiss that aren't<br>populated how do we go out and do Edna<br>monitoring here<br>and that's where I'm focusing my<br>research over the next few years is in<br>making air DNA technology accessible to<br>everyone I want to build a platform that<br>has large taxonomic breadth so it<br>doesn't matter what species that you're<br>working on I want something that<br>preserves DNA long term because we have<br>a lot of groups that go out they do<br>surveying once every three months<br>potentially once every six months and<br>they're not going to go back the next<br>day to do their survey to collect their<br>devices I want something that works for<br>them and that is also Compact and<br>lightweight so they're not charging<br>around big equipment something that's<br>affordable so almost anyone can do it<br>and something that's not non-technical<br>so it doesn't require understanding of<br>sterile technique or molecular DNA<br>analysis in order to deploy and devices<br>and understand the data that come out of<br>them<br>I'm testing a variety of different trap<br>designs both active ones that pump air<br>as well as things that collect DNA<br>passively and my preliminary results<br>show that both are effective they work<br>really really well so some big<br>contraption that pumps DNA is probably<br>just as good as something that<br>essentially you're waving a tissue<br>around in the air<br>foreign the work that I have in progress<br>right now is I'm pushing the boundaries<br>of where Edna can go and when it breaks<br>down so we'll be testing different<br>temperature regimes humidity longevity<br>how long does it last does wind matter<br>does it matter where you're putting the<br>device and which species can you detect<br>with it and which can't you detect with<br>it<br>I'm partnering with<br>organizations here in Australia so I've<br>got a project up and running with the<br>New South Wales government the<br>department of planning and environment<br>to put out traps in coordination with<br>their traditional monitoring methods to<br>look for different endangered species of<br>microbats as well as rodents and I'm<br>also working with the national territory<br>the Northern Territory National Resource<br>Management Group to put out our e-dna<br>devices alongside bioacoustic devices to<br>look for the white throated grass run as<br>well as the co-occurrence of domestic<br>cats so this will be a really cool<br>opportunity to test the platform in very<br>different environments as well as in<br>looking for different species and<br>whether or not it's going to work in the<br>hands of ecologists which I'm really<br>excited to see<br>and all this is going to be done in the<br>new state-of-the-art collections<br>building that is just up the road from<br>me and is still a big hole but as of<br>next year we're going to have a<br>state-of-the-art trace facility right<br>next to our collections so we're going<br>to be a hub for this sort of work going<br>forward<br>so that was a bit of a lot of stuff that<br>I've been doing over the last couple<br>years and it's involved a whole whole<br>lot of people both here at the csiro uq<br>Anu and that should say University of<br>Canberra they're not a Anu both of them<br>um but yeah there's been a joint effort<br>and it's been funded by bio platforms<br>Australia and I've got some of my<br>contact details on there if you want to<br>ask me questions by email or hit me up<br>on Twitter but otherwise you guys can<br>pester me with questions until your<br>awesome that was great<br>a clap on behalf of everybody I hope you<br>heard that yeah um cool so we'll go into<br>um you've stopped sharing so I'll go<br>into feedback and questions from the<br>audience<br>okay<br>um well I can start off<br>um I was really interested in lots of<br>different elements of that and I<br>wondered when you were sort of talking<br>in your really first part of the talk<br>what what would your advice be to people<br>that are um trying to get do whole<br>genome resequencing from old samples<br>samples that are like up to over over<br>that sort of 40-year barrier that you<br>mentioned because I think we can by now<br>get you know small quantities of DNA<br>even you know from these samples in fact<br>I've sort of worked on samples before<br>and haven't really been that limited in<br>concentration but it's the kind of<br>breadth of the genome that the samples<br>um represent in I guess so do you think<br>there's hope that we will get to a point<br>where we can get decent coverage out of<br>Museum samples at that whole genome<br>level yeah I do and I put the 40-year<br>marker in the paper because we had so<br>few individuals that we had sampled from<br>prior to that and so we just wanted to<br>mention that age might be a factor but<br>since publishing that we've done a bunch<br>more sampling in particular in the water<br>dragons and I don't think age matters<br>when it comes to formal and preserved<br>specimens because once the formal and<br>preserve preservation is done it just<br>kind of sits there which is different<br>than something like pinned insects which<br>are just going to be continually<br>degrading over time so it's a bit of a<br>different story and where I'm reporting<br>that we're getting you know 11x cover 3x<br>cover it's because these are our species<br>with like 1.5 gig genomes and it just<br>depends on how much money you want to<br>pump into sequencing them as to whether<br>or not you're going to get really high<br>cover and<br>you know just recently this year the<br>cost of short read sequencing has again<br>dropped because there's no sequencing<br>platforms that can pump out<br>tenfold more sequencing reads than they<br>were able to do previously so short read<br>technology still relevant and it just<br>kind of depends on how badly you want<br>that genome and the other thing is is<br>using a mapping approach that is going<br>to get as much information as you can so<br>we don't do any kind of trimming of our<br>um of our reads at all we just map it<br>using a program called K line and the<br>other thing we do is that we use a<br>library prep kit that suppresses any<br>primer dimer and also has unique<br>molecular identifiers so we're able to<br>in our initial paper molecular ecology<br>resources we were losing<br>60 of the data because it wasn't mapping<br>I think most of that was Primer dimer<br>and then in the second round a lot of it<br>was over amplification because we were<br>had so much so little to begin with that<br>we hadn't really honed that in but now<br>I'm getting upwards of like 70 of our<br>stuff mapping and retained after<br>deduplication so now we're we're getting<br>to the point where we can from almost<br>any specimen get the whole genome<br>sequencing cover at least in form one<br>preserve stuff<br>oh that's really exciting and I think<br>those sort of methods can be applied to<br>anything so long as you yeah yeah cool<br>um there's a question from Georgia<br>um since she typed it I'll read it for<br>you it says very cool work thank you for<br>sharing how long how long of amplicons<br>have you been able to get from formal<br>and fixed DNA<br>small it's<br>the average fragments are under 100<br>bases<br>you might have a random longer one in<br>there and so maybe you'd be lucky if<br>you're targeting you know fragments that<br>are 150 or 200 bases you might be lucky<br>but in our bucket seat approach<br>um I want to say they were 80 one of<br>them was 80 base pairs and one of them<br>was like 110 so you do really need to<br>Target really small regions but if you<br>have a region of interest you just tile<br>them and again it just depends on how<br>badly you want that that sequence data<br>um because it's not going to work if<br>you're targeting 300 and 400 base pair<br>amplicons that's something that we we<br>still struggle with where people will<br>try our method they'll try an amplicon<br>that's way too big and they'll tell us<br>it doesn't work and I'm like well you're<br>never going to get your 500 base pair<br>amplicon it's just it's just not going<br>to work<br>um and we are also trialling doing<br>reduced representation sequencing on the<br>output from Mr extraction so we've got<br>samples over at Dart here in Canberra to<br>see whether or not that will work there<br>is a lot of evidence to show that bait<br>capture works really well with these<br>samples and so that would become a more<br>affordable way of doing<br>um population genetics on these kind of<br>samples<br>foreign I have another question uh just<br>a quick one maybe to wrap us up<br>um so you mentioned at the beginning<br>that you were talking about how this<br>sort of uh there is starting to be<br>consideration in Australia and also<br>around the world about data sovereignty<br>um aspects of all of these different<br>types of data<br>um in the environmental DNA program that<br>you're developing at the moment and also<br>the museum work where people you know<br>have historically deposited samples you<br>know well before any consideration of<br>these sorts of issues<br>um how how are you how's your group kind<br>of working towards addressing these<br>issues or how do you foresee this<br>working well in the future so<br>two questions there the curation side of<br>it<br>it's honestly something that we are<br>struggling with it's something that we<br>are just now starting to more<br>comfortably even have conversations<br>within our own collection about it and<br>it's something that we are working with<br>other institutions internationally to<br>learn from and figure out how to do this<br>really well there's a workshop coming up<br>here in Canberra hosted at the Anu and<br>it's going to be something that's<br>involving researchers and curators<br>because researchers are doing you know<br>collections they have historical<br>collections that they've inherited from<br>different people<br>and just understanding the breadth of<br>the problem and then trying to figure<br>out how to do something with I'm not<br>saying we're doing it well<br>um and it's I think something that we<br>need to learn how to have uncomfortable<br>conversations about more and more so<br>because even I've had researchers who<br>want to ask me like oh I want to access<br>specimens in the collection what do we<br>do about data sovereignty and I'm like<br>I'm not sure thank you for asking let's<br>dig in this together because it might be<br>something very specific to those<br>specimens that we don't have a protocol<br>for<br>um so unfortunately I don't have<br>a really good response that I'm happy<br>with on the collections end of it on the<br>e-dna side of it<br>um something I'm working with from the<br>very beginning with Airborne Edna is<br>because these studies will be conducted<br>in collaboration with traditional owners<br>we're going to be doing<br>um having discussions on the first hand<br>are what it means what this information<br>is going to be so that we can very very<br>openly say the DNA that we're collecting<br>it might be yours it might be your<br>relatives and what does that mean for<br>you how does that fit into the<br>information that you want to get out of<br>it because we do have the ability to do<br>like blocking of human DNA do you not<br>want that are there particular species<br>in the area that you're concerned about<br>that you want information on in which<br>case will Target that or are there<br>species that you would rather us not<br>provide information about because that<br>would you know open you up to people<br>knowing where sensitive texts are so<br>from the very beginning we're Consulting<br>and also involving the traditional<br>owners to make sure that it's a<br>co-design process and we won't be<br>engaging in any kind of sampling that<br>isn't a code design from the very<br>beginning and that's something that I'm<br>trying to push really hard<br>um and it's something that I'm also<br>trying to kick start a bit of a thought<br>um like a thought experiment on what is<br>e-dna to uh instigate a conversation<br>around the world and so I met a<br>researcher Don Lewis at icg just a<br>couple weeks ago and she does<br>um research on Paleo genomics and she<br>herself is an indigenous woman and I<br>want to get her Insight on what it might<br>mean to translate that kind of work to<br>e-dna<br>um because I think there's a lot we can<br>learn from the ancient DNA space who's<br>grappled with these kind of questions<br>already that we can also translate to<br>e-dna and just not reinvent the wheel<br>and make sure that we're all<br>communicating really well<br>but a very active area of discussion and<br>research yeah cool<br>well I guess we've got one minute left<br>on our timeline so I'll just see if<br>anybody else has any quick last-minute<br>questions<br>okay all right well um thank you again<br>that was a really excellent seminar and<br>lots of things lots of interesting<br>things covered so you've given us lots<br>of food for thought<br>um I'll stop recording now</p></main><footer style="margin-top: 2rem; background: #0001; padding: 2rem; text-align: center;"><p>We Are The University</p><ul style="list-style-type: none; padding: 0; margin: 0;"><li><a href="/">Home</a></li><li><a href="/about">About</a></li><li><a href="/contact">Contact</a></li></ul></footer></body></html>